<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>Mirrorstoolkit: one-liner analysis tool • mirrorstoolkit</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.3.1/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.3.1/bootstrap.bundle.min.js"></script><link href="../deps/font-awesome-6.5.2/css/all.min.css" rel="stylesheet">
<link href="../deps/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet">
<script src="../deps/headroom-0.11.0/headroom.min.js"></script><script src="../deps/headroom-0.11.0/jQuery.headroom.min.js"></script><script src="../deps/bootstrap-toc-1.0.1/bootstrap-toc.min.js"></script><script src="../deps/clipboard.js-2.0.11/clipboard.min.js"></script><script src="../deps/search-1.0.0/autocomplete.jquery.min.js"></script><script src="../deps/search-1.0.0/fuse.min.js"></script><script src="../deps/search-1.0.0/mark.min.js"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Mirrorstoolkit: one-liner analysis tool">
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>


    <nav class="navbar navbar-expand-lg fixed-top bg-light" data-bs-theme="light" aria-label="Site navigation"><div class="container">

    <a class="navbar-brand me-2" href="../index.html">mirrorstoolkit</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.1.0</small>


    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item"><a class="nav-link" href="../reference/index.html">Reference</a></li>
<li class="active nav-item dropdown">
  <button class="nav-link dropdown-toggle" type="button" id="dropdown-articles" data-bs-toggle="dropdown" aria-expanded="false" aria-haspopup="true">Articles</button>
  <ul class="dropdown-menu" aria-labelledby="dropdown-articles">
<li><a class="dropdown-item" href="../articles/introduction.html">Mirrorstoolkit: one-liner analysis tool</a></li>
  </ul>
</li>
      </ul>
<ul class="navbar-nav">
<li class="nav-item"><form class="form-inline" role="search">
 <input class="form-control" type="search" name="search-input" id="search-input" autocomplete="off" aria-label="Search site" placeholder="Search for" data-search-index="../search.json">
</form></li>
      </ul>
</div>


  </div>
</nav><div class="container template-article">



<script src="introduction_files/htmlwidgets-1.6.4/htmlwidgets.js"></script><link href="introduction_files/vis-9.1.0/vis-network.min.css" rel="stylesheet">
<script src="introduction_files/vis-9.1.0/vis-network.min.js"></script><script src="introduction_files/visNetwork-binding-2.1.4/visNetwork.js"></script><link href="introduction_files/htmltools-fill-0.5.9/fill.css" rel="stylesheet">
<script src="introduction_files/plotly-binding-4.12.0/plotly.js"></script><script src="introduction_files/typedarray-0.1/typedarray.min.js"></script><link href="introduction_files/crosstalk-1.2.2/css/crosstalk.min.css" rel="stylesheet">
<script src="introduction_files/crosstalk-1.2.2/js/crosstalk.min.js"></script><link href="introduction_files/plotly-htmlwidgets-css-2.25.2/plotly-htmlwidgets.css" rel="stylesheet">
<script src="introduction_files/plotly-main-2.25.2/plotly-latest.min.js"></script><div class="row">
  <main id="main" class="col-md-9"><div class="page-header">

      <h1>Mirrorstoolkit: one-liner analysis tool</h1>
                        <h4 data-toc-skip class="author">Yihao Li</h4>
            <address class="author_afil">
      Reyes-Soffer Lab, Division of Preventive medicine, Department of
Medicine, Columbia University Irving Medical Center;Badri Vardarajan
Lab, Gertrude H. Sergievsky Center, Department of Neurology, Columbia
University Irving Medical Center<br><div class="d-none name"><code>introduction.Rmd</code></div>
    </address>
</div>

    
    
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://dplyr.tidyverse.org" class="external-link">dplyr</a></span><span class="op">)</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; Attaching package: 'dplyr'</span></span>
<span><span class="co">#&gt; The following objects are masked from 'package:stats':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     filter, lag</span></span>
<span><span class="co">#&gt; The following objects are masked from 'package:base':</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt;     intersect, setdiff, setequal, union</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://readxl.tidyverse.org" class="external-link">readxl</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va">mirrorstoolkit</span><span class="op">)</span></span></code></pre></div>
<div class="section level2">
<h2 id="data-preparation">Data Preparation<a class="anchor" aria-label="anchor" href="#data-preparation"></a>
</h2>
<div class="section level3">
<h3 id="compute-nih-equation-2-for-ldl-calc-estimation">Compute NIH equation 2 for LDL (calc) estimation<a class="anchor" aria-label="anchor" href="#compute-nih-equation-2-for-ldl-calc-estimation"></a>
</h3>
<p>compute_nih_ldl functions allows either value or vectorized
operation</p>
<div class="section level4">
<h4 id="by-value">By value<a class="anchor" aria-label="anchor" href="#by-value"></a>
</h4>
<p>Simply input the value</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/compute_nih_ldl.html">compute_nih_ldl</a></span><span class="op">(</span>tchol <span class="op">=</span> <span class="fl">112.9</span>,hdl <span class="op">=</span> <span class="fl">39.9</span>, trig <span class="op">=</span> <span class="fl">151</span><span class="op">)</span></span>
<span><span class="co">#&gt; [1] 47.18626</span></span></code></pre></div>
</div>
<div class="section level4">
<h4 id="vectorized">Vectorized<a class="anchor" aria-label="anchor" href="#vectorized"></a>
</h4>
<p>Use the column name, it uses basic dplyr naming flow. If your column
name is not one character string surround it by “`”</p>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">df</span> <span class="op">=</span> <span class="va">df</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span></span>
<span>  `LDL (calc) mg/dL` <span class="op">=</span> <span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/compute_nih_ldl.html">compute_nih_ldl</a></span><span class="op">(</span></span>
<span>    tchol <span class="op">=</span> <span class="va">`Chol mg/dL`</span>,</span>
<span>    hdl <span class="op">=</span> <span class="va">`HDL mg/dL`</span>,</span>
<span>    trig <span class="op">=</span> <span class="va">`Trig mg/dL`</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="va">df</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> </span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span></span>
<span>    <span class="fu">tidyr</span><span class="fu">::</span><span class="fu"><a href="https://tidyselect.r-lib.org/reference/all_of.html" class="external-link">all_of</a></span><span class="op">(</span></span>
<span>      <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Chol mg/dL"</span>, </span>
<span>        <span class="st">"HDL mg/dL"</span>,</span>
<span>        <span class="st">"Trig mg/dL"</span>, </span>
<span>        <span class="st">"LDL (calc) mg/dL"</span><span class="op">)</span></span>
<span>      <span class="op">)</span><span class="op">)</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 6 × 4</span></span></span>
<span><span class="co">#&gt;   `Chol mg/dL` `HDL mg/dL` `Trig mg/dL` `LDL (calc) mg/dL`</span></span>
<span><span class="co">#&gt;          <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>       <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>        <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>              <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">1</span>         113.        39.9          151               47.2</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">2</span>         189.        76.5          146               87.8</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">3</span>          92         49.2           82               26.1</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">4</span>         153.        29.4          315               73.0</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">5</span>         134.        58.8           83               59.7</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">6</span>         105.        41.3          145               38.7</span></span></code></pre></div>
</div>
</div>
</div>
<div class="section level2">
<h2 id="exploratory-data-analysis">Exploratory Data Analysis<a class="anchor" aria-label="anchor" href="#exploratory-data-analysis"></a>
</h2>
<div class="section level3">
<h3 id="single-group-analysis">Single Group analysis<a class="anchor" aria-label="anchor" href="#single-group-analysis"></a>
</h3>
<div class="section level4">
<h4 id="normality-inspection">Normality Inspection<a class="anchor" aria-label="anchor" href="#normality-inspection"></a>
</h4>
<p>This is a part which I feel is not emphasized enough in our lab
practice - we need to determine if a lab evaluation or result result is
normally distributed. <strong>inspect_distribution()</strong> can give a
basic visualization about the distribution by</p>
<ul>
<li>The histogram</li>
<li>The density plot</li>
<li>
<a href="https://en.wikipedia.org/wiki/Shapiro%E2%80%93Wilk_test" class="external-link">Shapiro-Wilk
test</a> result - please be aware that the null hypothesis is it’s
normally distributed, thus if it is significant, it is not normal.</li>
</ul>
<p>It usually gives a lot of warnings (depends on the completeness and
quality of your data for most of the times), I suppressed them here for
tidiness, but expose the warnings (omit the last function) are
recommended, it will give you useful informations.</p>
<p>Please always be aware that the normality it’s a very subjective
decision, always add a huge grab of your own salts from your reading and
experience, especially when you have a small sample size. See below,
from my perspective:</p>
<ul>
<li>Albumin, from PMID: 18405401, is generally normal overall.</li>
<li>Calcium is quite normal, which is a bit out of my expectation.</li>
<li>Smoking is actually boolean (binary, 0/1).</li>
<li>LDL, it is on you, you can either treat it as normal or not normal,
my experience is it’s generally normal.</li>
<li>wIS often treated as normally distributed on my side.</li>
<li>Lp(a), skewed right for sure</li>
</ul>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/inspect_distribution.html">inspect_distribution</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">df</span>, </span>
<span>  variable <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>    <span class="st">"Albumin"</span>, </span>
<span>    <span class="st">"Calcium"</span>,</span>
<span>    <span class="st">"Smoking"</span>,</span>
<span>    <span class="st">"LDL (calc) mg/dL"</span>,</span>
<span>    <span class="st">"wIS"</span>,</span>
<span>    <span class="st">"Lp(a) (nmol/L)"</span><span class="op">)</span></span>
<span>  <span class="op">)</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> <span class="fu">base</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/base/warning.html" class="external-link">suppressWarnings</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; $visualization</span></span>
<span><span class="co">#&gt; $`1`</span></span></code></pre></div>
<p><img src="introduction_files/figure-html/unnamed-chunk-5-1.png" class="r-plt" alt="" width="672"></p>
<pre><code><span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; $`2`</span></span></code></pre>
<p><img src="introduction_files/figure-html/unnamed-chunk-5-2.png" class="r-plt" alt="" width="672"></p>
<pre><code><span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; $`3`</span></span></code></pre>
<p><img src="introduction_files/figure-html/unnamed-chunk-5-3.png" class="r-plt" alt="" width="672"></p>
<pre><code><span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; $`4`</span></span></code></pre>
<p><img src="introduction_files/figure-html/unnamed-chunk-5-4.png" class="r-plt" alt="" width="672"></p>
<pre><code><span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; $`5`</span></span></code></pre>
<p><img src="introduction_files/figure-html/unnamed-chunk-5-5.png" class="r-plt" alt="" width="672"></p>
<pre><code><span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; $`6`</span></span></code></pre>
<p><img src="introduction_files/figure-html/unnamed-chunk-5-6.png" class="r-plt" alt="" width="672"></p>
<pre><code><span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; attr(,"class")</span></span>
<span><span class="co">#&gt; [1] "list"      "ggarrange"</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; $`shapiro-wilk_test`</span></span>
<span><span class="co">#&gt;           variable      p_value normality</span></span>
<span><span class="co">#&gt; 1          Albumin 2.016173e-03     FALSE</span></span>
<span><span class="co">#&gt; 2          Calcium 6.699764e-02      TRUE</span></span>
<span><span class="co">#&gt; 3 LDL (calc) mg/dL 1.457113e-02     FALSE</span></span>
<span><span class="co">#&gt; 4   Lp(a) (nmol/L) 5.255467e-08     FALSE</span></span>
<span><span class="co">#&gt; 5          Smoking 1.278103e-10     FALSE</span></span>
<span><span class="co">#&gt; 6              wIS 7.879851e-03     FALSE</span></span></code></pre>
</div>
<div class="section level4">
<h4 id="characteristic-table">Characteristic Table<a class="anchor" aria-label="anchor" href="#characteristic-table"></a>
</h4>
<p>With the result above, you can make your decision and generate a
characteristic table.</p>
<ul>
<li>Boolean (1/0) data should be in boolean_list, the ratio of 1 will be
computed</li>
<li>Normally distributed continuous data should be in sd_list, the
mean(sd) will be computed</li>
<li>Non-normally distributed continuous data should be in IQR_list, the
median(IQR) will be computed</li>
</ul>
<p>These three lists can be skipped if you don’t have one variable falls
into this category.</p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/characteristic_table_generator.html">characteristic_table_generator</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">df</span>,</span>
<span>  boolean_list <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Smoking"</span><span class="op">)</span>,</span>
<span>  sd_list <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>    <span class="st">"Calcium"</span>, </span>
<span>    <span class="st">"wIS"</span>, </span>
<span>    <span class="st">"LDL (calc) mg/dL"</span>,</span>
<span>    <span class="st">"Albumin"</span></span>
<span>    <span class="op">)</span>,</span>
<span>  IQR_list <span class="op">=</span> <span class="st">"Lp(a) (nmol/L)"</span>,</span>
<span>  order <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>    <span class="st">"Smoking"</span>, </span>
<span>    <span class="st">"Albumin"</span>,</span>
<span>    <span class="st">"Calcium"</span>, </span>
<span>    <span class="st">"LDL (calc) mg/dL"</span>,</span>
<span>    <span class="st">"wIS"</span>,</span>
<span>    <span class="st">"Lp(a) (nmol/L)"</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt;                              overall</span></span>
<span><span class="co">#&gt; Smoking                29/57(50.88%)</span></span>
<span><span class="co">#&gt; Albumin                   3.4 ± 0.63</span></span>
<span><span class="co">#&gt; Calcium                  8.83 ± 0.73</span></span>
<span><span class="co">#&gt; LDL (calc) mg/dL       58.34 ± 24.31</span></span>
<span><span class="co">#&gt; wIS                     22.46 ± 4.45</span></span>
<span><span class="co">#&gt; Lp(a) (nmol/L)   46.76(19.66-118.88)</span></span></code></pre></div>
<p>Another implementation can be used here (although I prefer the one
above), I don’t know why R original output is not a table for
multi-column summary.</p>
<p>Please be aware that this one will run summary() for all columns
provided in this table.</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/multi_line_summary.html">multi_line_summary</a></span><span class="op">(</span></span>
<span>  <span class="va">df</span><span class="op">[</span>,<span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>    <span class="st">"Smoking"</span>, </span>
<span>    <span class="st">"Albumin"</span>,</span>
<span>    <span class="st">"Calcium"</span>, </span>
<span>    <span class="st">"LDL (calc) mg/dL"</span>,</span>
<span>    <span class="st">"wIS"</span>,</span>
<span>    <span class="st">"Lp(a) (nmol/L)"</span><span class="op">)</span><span class="op">]</span><span class="op">)</span></span>
<span><span class="co">#&gt; Warning in (function (..., deparse.level = 1) : number of rows of result is not</span></span>
<span><span class="co">#&gt; a multiple of vector length (arg 1)</span></span>
<span><span class="co">#&gt;                      Min.  1st Qu.   Median       Mean  3rd Qu.     Max.</span></span>
<span><span class="co">#&gt; Smoking           0.00000  0.00000  1.00000  0.5087719   1.0000   1.0000</span></span>
<span><span class="co">#&gt; Albumin           2.10000  3.00000  3.60000  3.3981132   3.9000   4.3000</span></span>
<span><span class="co">#&gt; Calcium           7.00000  8.30000  8.90000  8.8339623   9.3000  11.3000</span></span>
<span><span class="co">#&gt; LDL (calc) mg/dL 19.95264 36.36183 57.88097 58.3391133  72.3185 132.4144</span></span>
<span><span class="co">#&gt; wIS              13.76000 18.37000 21.80000 22.4583636  26.3950  29.3200</span></span>
<span><span class="co">#&gt; Lp(a) (nmol/L)    2.36000 19.65500 46.76000 82.5434545 118.8850 495.4500</span></span>
<span><span class="co">#&gt;                      NA's</span></span>
<span><span class="co">#&gt; Smoking           0.00000</span></span>
<span><span class="co">#&gt; Albumin           4.00000</span></span>
<span><span class="co">#&gt; Calcium           4.00000</span></span>
<span><span class="co">#&gt; LDL (calc) mg/dL 19.95264</span></span>
<span><span class="co">#&gt; wIS               2.00000</span></span>
<span><span class="co">#&gt; Lp(a) (nmol/L)    2.00000</span></span></code></pre></div>
</div>
</div>
<div class="section level3">
<h3 id="for-two-groups-result">For two groups result<a class="anchor" aria-label="anchor" href="#for-two-groups-result"></a>
</h3>
<p>Most of our analysis is based on two groups, e.g. African American vs
Caucasians, Study vs control, etc.</p>
<div class="section level4">
<h4 id="normality-inspection-1">Normality Inspection<a class="anchor" aria-label="anchor" href="#normality-inspection-1"></a>
</h4>
<p>Add a “by” variable will classify the data. However, I’d suggest keep
watching it overall for the distribution. This is simply for
completeness in quick inspection.</p>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/inspect_distribution.html">inspect_distribution</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">df</span>, </span>
<span>  by <span class="op">=</span> <span class="st">"Race"</span>,</span>
<span>  variable <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>    <span class="st">"Albumin"</span>, </span>
<span>    <span class="st">"Calcium"</span>,</span>
<span>    <span class="st">"Smoking"</span>,</span>
<span>    <span class="st">"LDL (calc) mg/dL"</span>,</span>
<span>    <span class="st">"wIS"</span>,</span>
<span>    <span class="st">"Lp(a) (nmol/L)"</span><span class="op">)</span></span>
<span>  <span class="op">)</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> <span class="fu">base</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/base/warning.html" class="external-link">suppressWarnings</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; $visualization</span></span>
<span><span class="co">#&gt; $`1`</span></span></code></pre></div>
<p><img src="introduction_files/figure-html/unnamed-chunk-8-1.png" class="r-plt" alt="" width="672"></p>
<pre><code><span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; $`2`</span></span></code></pre>
<p><img src="introduction_files/figure-html/unnamed-chunk-8-2.png" class="r-plt" alt="" width="672"></p>
<pre><code><span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; $`3`</span></span></code></pre>
<p><img src="introduction_files/figure-html/unnamed-chunk-8-3.png" class="r-plt" alt="" width="672"></p>
<pre><code><span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; $`4`</span></span></code></pre>
<p><img src="introduction_files/figure-html/unnamed-chunk-8-4.png" class="r-plt" alt="" width="672"></p>
<pre><code><span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; $`5`</span></span></code></pre>
<p><img src="introduction_files/figure-html/unnamed-chunk-8-5.png" class="r-plt" alt="" width="672"></p>
<pre><code><span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; $`6`</span></span></code></pre>
<p><img src="introduction_files/figure-html/unnamed-chunk-8-6.png" class="r-plt" alt="" width="672"></p>
<pre><code><span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; attr(,"class")</span></span>
<span><span class="co">#&gt; [1] "list"      "ggarrange"</span></span>
<span><span class="co">#&gt; </span></span>
<span><span class="co">#&gt; $`shapiro-wilk_test`</span></span>
<span><span class="co">#&gt;           variable      p_value normality             AF             EU</span></span>
<span><span class="co">#&gt; 1          Albumin 2.016173e-03     FALSE FALSE(3.3e-02) FALSE(1.9e-02)</span></span>
<span><span class="co">#&gt; 2          Calcium 6.699764e-02      TRUE  TRUE(4.9e-01) FALSE(3.0e-02)</span></span>
<span><span class="co">#&gt; 3 LDL (calc) mg/dL 1.457113e-02     FALSE  TRUE(9.2e-01) FALSE(3.6e-03)</span></span>
<span><span class="co">#&gt; 4   Lp(a) (nmol/L) 5.255467e-08     FALSE  TRUE(1.9e-01) FALSE(1.7e-08)</span></span>
<span><span class="co">#&gt; 5          Smoking 1.278103e-10     FALSE FALSE(1.7e-04) FALSE(3.4e-09)</span></span>
<span><span class="co">#&gt; 6              wIS 7.879851e-03     FALSE FALSE(4.5e-02) FALSE(6.8e-03)</span></span></code></pre>
</div>
<div class="section level4">
<h4 id="characteristic-table-1">Characteristic Table<a class="anchor" aria-label="anchor" href="#characteristic-table-1"></a>
</h4>
<p>With a “by” provided, it will generate statistics for sub-class,
meanwhile a p_value column is provided.</p>
<ul>
<li>For variables in boolean_list, the p-value is from chi-square test
(stats::chisq.test())</li>
<li>For variables in IQR_list, the p-value is from Mann-Whitney U test
(Wilcoxon rank-sum test, stats::wilcox.test())</li>
<li>For varaibles in sd_list, the p-value is depending on variable
<em>normal_policy</em>
<ul>
<li>if normal_policy is “wilcox”(<em>default</em>): it will run
Mann-Whitney U test, for the following two reason:
<ul>
<li>Our normal data is often not “that normal”.</li>
<li>A sample size of 30 or more is often cited as a general guideline to
ensure a normal distribution is a reasonable approximation.</li>
</ul>
</li>
<li>if normal_policy is “t-test”: it will run <a href="https://en.wikipedia.org/wiki/Welch%27s_t-test" class="external-link">Welch’s t-test</a>
(stats::t.test(var.equal = FALSE)), which is a more reliable adaptation
of Student’s t-test when the two samples have unequal variances and
possibly unequal sample sizes</li>
</ul>
</li>
</ul>
<div class="sourceCode" id="cb20"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/characteristic_table_generator.html">characteristic_table_generator</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">df</span>,</span>
<span>  by <span class="op">=</span> <span class="st">"Race"</span>,</span>
<span>  boolean_list <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Smoking"</span><span class="op">)</span>,</span>
<span>  sd_list <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>    <span class="st">"Calcium"</span>, </span>
<span>    <span class="st">"wIS"</span>, </span>
<span>    <span class="st">"LDL (calc) mg/dL"</span>,</span>
<span>    <span class="st">"Albumin"</span></span>
<span>    <span class="op">)</span>,</span>
<span>  IQR_list <span class="op">=</span> <span class="st">"Lp(a) (nmol/L)"</span>,</span>
<span>  order <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>    <span class="st">"Smoking"</span>, </span>
<span>    <span class="st">"Albumin"</span>,</span>
<span>    <span class="st">"Calcium"</span>, </span>
<span>    <span class="st">"LDL (calc) mg/dL"</span>,</span>
<span>    <span class="st">"wIS"</span>,</span>
<span>    <span class="st">"Lp(a) (nmol/L)"</span><span class="op">)</span>,</span>
<span>  normal_policy <span class="op">=</span> <span class="st">"t-test"</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt;                              overall                   AF                 EU</span></span>
<span><span class="co">#&gt; Smoking                29/57(50.88%)         6/13(46.15%)      23/44(52.27%)</span></span>
<span><span class="co">#&gt; Albumin                   3.4 ± 0.63           3.1 ± 0.88        3.46 ± 0.56</span></span>
<span><span class="co">#&gt; Calcium                  8.83 ± 0.73          8.41 ± 0.79         8.92 ± 0.7</span></span>
<span><span class="co">#&gt; LDL (calc) mg/dL       58.34 ± 24.31        60.24 ± 24.85      57.78 ± 24.41</span></span>
<span><span class="co">#&gt; wIS                     22.46 ± 4.45         21.31 ± 3.52       22.81 ± 4.68</span></span>
<span><span class="co">#&gt; Lp(a) (nmol/L)   46.76(19.66-118.88) 127.64(46.92-197.37) 41.08(15.27-79.61)</span></span>
<span><span class="co">#&gt;                     p_value</span></span>
<span><span class="co">#&gt; Smoking          7.5912e-01</span></span>
<span><span class="co">#&gt; Albumin          2.6747e-01</span></span>
<span><span class="co">#&gt; Calcium          9.9941e-02</span></span>
<span><span class="co">#&gt; LDL (calc) mg/dL 7.5591e-01</span></span>
<span><span class="co">#&gt; wIS              2.2530e-01</span></span>
<span><span class="co">#&gt; Lp(a) (nmol/L)   4.8697e-02</span></span></code></pre></div>
</div>
</div>
<div class="section level3">
<h3 id="for-3-groups-result">For 3+ groups result<a class="anchor" aria-label="anchor" href="#for-3-groups-result"></a>
</h3>
<div class="section level4">
<h4 id="normality-inspection-2">Normality Inspection<a class="anchor" aria-label="anchor" href="#normality-inspection-2"></a>
</h4>
<p>Normality inspection is same as the 2 groups result</p>
</div>
<div class="section level4">
<h4 id="characteristic-table-2">Characteristic Table<a class="anchor" aria-label="anchor" href="#characteristic-table-2"></a>
</h4>
<p>With a “by” provided, it will generate statistics for sub-class,
meanwhile a p_value column is provided.</p>
<ul>
<li>For variables in boolean_list, the p-value is from chi-square test
(stats::chisq.test())</li>
<li>For variables in IQR_list, the p-value is from Kruskal–Wallis test
(stats::kruskal.test())</li>
<li>For varaibles in sd_list, the p-value is depending on variable
<em>normal_policy</em>
<ul>
<li>if normal_policy is “wilcox”(<em>default</em>, I inherited the name
from above thus won’t make things too complex): it will run
stats::kruskal.test(), for the following two reason:
<ul>
<li>Our normal data is often not “that normal”.</li>
<li>A sample size of 30 or more is often cited as a general guideline to
ensure a normal distribution is a reasonable approximation.</li>
</ul>
</li>
<li>if normal_policy is “t-test”: it will run ANOVA(stats::aov())</li>
</ul>
</li>
</ul>
<div class="sourceCode" id="cb21"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">df</span> <span class="op">=</span> <span class="va">df</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span></span>
<span>  Calciphylaxis_str <span class="op">=</span> <span class="va">Calciphylaxis</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> <span class="fu">plyr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/plyr/man/mapvalues.html" class="external-link">mapvalues</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="fl">0</span>,<span class="fl">1</span><span class="op">)</span>, <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"control"</span>, <span class="st">"disease"</span><span class="op">)</span><span class="op">)</span>,</span>
<span>  group <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste</a></span><span class="op">(</span><span class="va">`Race`</span>, <span class="va">`Calciphylaxis_str`</span>, sep <span class="op">=</span> <span class="st">"_"</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="va">df</span><span class="op">$</span><span class="va">group</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> <span class="fu"><a href="https://rdrr.io/r/base/table.html" class="external-link">table</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="co">#&gt; .</span></span>
<span><span class="co">#&gt; AF_control AF_disease EU_control EU_disease </span></span>
<span><span class="co">#&gt;          8          5         22         22</span></span></code></pre></div>
<div class="sourceCode" id="cb22"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">table_1_by_group</span> <span class="op">=</span> <span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/characteristic_table_generator.html">characteristic_table_generator</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">df</span>,</span>
<span>  by <span class="op">=</span> <span class="st">"group"</span>,</span>
<span>  boolean_list <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Smoking"</span><span class="op">)</span>,</span>
<span>  sd_list <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>    <span class="st">"Calcium"</span>, </span>
<span>    <span class="st">"wIS"</span>, </span>
<span>    <span class="st">"LDL (calc) mg/dL"</span>,</span>
<span>    <span class="st">"Albumin"</span></span>
<span>    <span class="op">)</span>,</span>
<span>  IQR_list <span class="op">=</span> <span class="st">"Lp(a) (nmol/L)"</span>,</span>
<span>  order <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>    <span class="st">"Smoking"</span>, </span>
<span>    <span class="st">"Albumin"</span>,</span>
<span>    <span class="st">"Calcium"</span>, </span>
<span>    <span class="st">"LDL (calc) mg/dL"</span>,</span>
<span>    <span class="st">"wIS"</span>,</span>
<span>    <span class="st">"Lp(a) (nmol/L)"</span><span class="op">)</span>,</span>
<span>  normal_policy <span class="op">=</span> <span class="st">"t-test"</span></span>
<span><span class="op">)</span></span>
<span><span class="va">table_1_by_group</span></span>
<span><span class="co">#&gt;                              overall          AF_control          AF_disease</span></span>
<span><span class="co">#&gt; Smoking                29/57(50.88%)            4/8(50%)            2/5(40%)</span></span>
<span><span class="co">#&gt; Albumin                   3.4 ± 0.63         3.52 ± 0.83         2.76 ± 0.83</span></span>
<span><span class="co">#&gt; Calcium                  8.83 ± 0.73          8.7 ± 0.45         8.18 ± 0.97</span></span>
<span><span class="co">#&gt; LDL (calc) mg/dL       58.34 ± 24.31       64.04 ± 28.44       54.16 ± 19.04</span></span>
<span><span class="co">#&gt; wIS                     22.46 ± 4.45        22.22 ± 3.48         19.85 ± 3.4</span></span>
<span><span class="co">#&gt; Lp(a) (nmol/L)   46.76(19.66-118.88) 137.46(43.7-206.76) 55.59(51.67-129.45)</span></span>
<span><span class="co">#&gt;                          EU_control        EU_disease    p_value</span></span>
<span><span class="co">#&gt; Smoking               12/22(54.55%)        11/22(50%) 9.8051e-01</span></span>
<span><span class="co">#&gt; Albumin                  3.82 ± 0.3        3.1 ± 0.54 2.8352e-05</span></span>
<span><span class="co">#&gt; Calcium                 8.79 ± 0.54       9.05 ± 0.82 1.0060e-01</span></span>
<span><span class="co">#&gt; LDL (calc) mg/dL      56.18 ± 17.64     59.38 ± 30.07 8.5607e-01</span></span>
<span><span class="co">#&gt; wIS                    22.98 ± 4.51      22.65 ± 4.96 5.7002e-01</span></span>
<span><span class="co">#&gt; Lp(a) (nmol/L)   33.85(18.76-87.43) 43.4(14.11-75.14) 2.0291e-01</span></span></code></pre></div>
<p>The output illutrated the p-value as literal string for I need to
keep the format for paper use… To filter the p-value you can do this</p>
<div class="sourceCode" id="cb23"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">table_1_by_group</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> </span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span>p_value <span class="op">=</span> <span class="va">p_value</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> <span class="fu"><a href="https://rdrr.io/r/base/numeric.html" class="external-link">as.numeric</a></span><span class="op">(</span><span class="op">)</span><span class="op">)</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> </span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">p_value</span> <span class="op">&lt;</span> <span class="fl">0.05</span><span class="op">)</span></span>
<span><span class="co">#&gt;            overall  AF_control  AF_disease EU_control EU_disease    p_value</span></span>
<span><span class="co">#&gt; Albumin 3.4 ± 0.63 3.52 ± 0.83 2.76 ± 0.83 3.82 ± 0.3 3.1 ± 0.54 2.8352e-05</span></span></code></pre></div>
</div>
<div class="section level4">
<h4 id="pairwise-test">Pairwise test<a class="anchor" aria-label="anchor" href="#pairwise-test"></a>
</h4>
<p>Please always be aware that for 3+ group test, one p-value is not
sufficient to claim too much stuff. Multi-class
chi-square/Kruskal-Wallis/ANOVA only told you “some group is different
from the other”</p>
<div class="section level5">
<h5 id="boolean-variables">Boolean Variables<a class="anchor" aria-label="anchor" href="#boolean-variables"></a>
</h5>
<p>Pairwise comparisons between pairs of proportions
(stats::pairwise.prop.test) used here, a filled barchart used in the
visualization.</p>
<p>When the pairwise test is throwing “Warning: Chi-squared
approximation may be incorrect” means that one or more of your table’s
expected cell counts are too small for the Chi-squared test to be
reliable, typically when they are less than 5. This violates the test’s
assumptions, and you should address it by either increasing sample size,
combining categories, or using an alternative like Fisher’s exact test
or a simulated p-value.</p>
<ul>
<li>If it’s labelled significant, it can be a good support to your
claim.</li>
<li>If it’s not labelled as significant, it doesn’t always mean that
they are not different, it’s simply stats::pairwise.prop.test cannot
handle it. Be bold to make your choice.</li>
</ul>
<p>Some settings, I grabbed a setting from the air for the
illustration:</p>
<ul>
<li>
<em>adjustment_method</em> The choice of this variable is from <a href="https://www.rdocumentation.org/packages/stats/versions/3.6.2/topics/p.adjust" class="external-link">here</a>.
“fdr” (Benjamini-Hochberg FDR correction) by default thus you can often
skip it.</li>
<li>
<em>p_threshold</em> will affect the filtering from first to second
table, the significant result will be labelled on the top of the figure,
0.05 by default</li>
<li>
<em>variable</em> can take one or more variables.</li>
</ul>
<div class="sourceCode" id="cb24"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">pairwise_boolean</span> <span class="op">=</span> <span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/pairwise_boolean_report.html">pairwise_boolean_report</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">df</span>, </span>
<span>  variable <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"antiplatelet_group"</span>, <span class="st">"statin_group"</span><span class="op">)</span>, </span>
<span>  by <span class="op">=</span> <span class="st">"group"</span>,</span>
<span>  adjustment_method <span class="op">=</span> <span class="st">"none"</span>,</span>
<span>  p_threshold <span class="op">=</span> <span class="fl">0.3</span><span class="op">)</span></span>
<span><span class="co">#&gt; Warning in prop.test(x[c(i, j)], n[c(i, j)], ...): Chi-squared approximation</span></span>
<span><span class="co">#&gt; may be incorrect</span></span>
<span><span class="co">#&gt; Warning in prop.test(x[c(i, j)], n[c(i, j)], ...): Chi-squared approximation</span></span>
<span><span class="co">#&gt; may be incorrect</span></span>
<span><span class="co">#&gt; Warning in prop.test(x[c(i, j)], n[c(i, j)], ...): Chi-squared approximation</span></span>
<span><span class="co">#&gt; may be incorrect</span></span>
<span><span class="co">#&gt; Warning in prop.test(x[c(i, j)], n[c(i, j)], ...): Chi-squared approximation</span></span>
<span><span class="co">#&gt; may be incorrect</span></span>
<span><span class="co">#&gt; Warning in prop.test(x[c(i, j)], n[c(i, j)], ...): Chi-squared approximation</span></span>
<span><span class="co">#&gt; may be incorrect</span></span>
<span><span class="co">#&gt; Warning in prop.test(x[c(i, j)], n[c(i, j)], ...): Chi-squared approximation</span></span>
<span><span class="co">#&gt; may be incorrect</span></span>
<span><span class="co">#&gt; Warning in prop.test(x[c(i, j)], n[c(i, j)], ...): Chi-squared approximation</span></span>
<span><span class="co">#&gt; may be incorrect</span></span>
<span><span class="co">#&gt; Warning in prop.test(x[c(i, j)], n[c(i, j)], ...): Chi-squared approximation</span></span>
<span><span class="co">#&gt; may be incorrect</span></span>
<span><span class="co">#&gt; Warning in prop.test(x[c(i, j)], n[c(i, j)], ...): Chi-squared approximation</span></span>
<span><span class="co">#&gt; may be incorrect</span></span>
<span><span class="co">#&gt; Warning in prop.test(x[c(i, j)], n[c(i, j)], ...): Chi-squared approximation</span></span>
<span><span class="co">#&gt; may be incorrect</span></span>
<span><span class="co">#&gt; Warning in ggsignif::geom_signif(data = annotation_significant,</span></span>
<span><span class="co">#&gt; ggplot2::aes(xmin = group1, : Ignoring unknown aesthetics: <span style="color: #00BB00;">xmin</span>,</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">xmax</span>, <span style="color: #00BB00;">annotations</span>, and <span style="color: #00BB00;">y_position</span></span></span></code></pre></div>
<div class="sourceCode" id="cb25"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">pairwise_boolean</span><span class="op">$</span><span class="va">result</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 12 × 4</span></span></span>
<span><span class="co">#&gt;    variable_name      group1     group2     p_value</span></span>
<span><span class="co">#&gt;    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>              <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>      <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>        <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 1</span> antiplatelet_group AF_disease AF_control   0.396</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 2</span> antiplatelet_group EU_control AF_control   1.00 </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 3</span> antiplatelet_group EU_control AF_disease   0.357</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 4</span> antiplatelet_group EU_disease AF_control   1    </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 5</span> antiplatelet_group EU_disease AF_disease   0.281</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 6</span> antiplatelet_group EU_disease EU_control   0.931</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 7</span> statin_group       AF_disease AF_control   1    </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 8</span> statin_group       EU_control AF_control   1    </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 9</span> statin_group       EU_control AF_disease   1    </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">10</span> statin_group       EU_disease AF_control   0.907</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">11</span> statin_group       EU_disease AF_disease   1    </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">12</span> statin_group       EU_disease EU_control   0.803</span></span></code></pre></div>
<div class="sourceCode" id="cb26"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">pairwise_boolean</span><span class="op">$</span><span class="va">significant_result</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 1 × 4</span></span></span>
<span><span class="co">#&gt;   variable_name      group1     group2     p_value</span></span>
<span><span class="co">#&gt;   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>              <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>      <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>        <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">1</span> antiplatelet_group EU_disease AF_disease    0.28</span></span></code></pre></div>
<div class="sourceCode" id="cb27"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">pairwise_boolean</span><span class="op">$</span><span class="va">plot</span></span></code></pre></div>
<p><img src="introduction_files/figure-html/unnamed-chunk-16-1.png" class="r-plt" alt="" width="672"></p>
</div>
<div class="section level5">
<h5 id="not-normally-distributed-continuous-variables">Not Normally Distributed Continuous Variables<a class="anchor" aria-label="anchor" href="#not-normally-distributed-continuous-variables"></a>
</h5>
<p>Pairwise Mann-Whiteney U-test (stats::pairwise.wilcox.test) used
here, boxplot for illustration.</p>
<p>If you are not confident with the normality or the sample size, use
this one.</p>
<p>Please be aware of a toxic chemical reaction here:</p>
<ul>
<li>Mann-Whiteney U test is based on the rank of the sample</li>
<li>Boxplot can drop some outliers out.</li>
</ul>
<p>Thus this report is very visually misleading with extremely small
sample size, e.g. AF_disease group only have 5 samples and 1 outliers
makes it not significantly different from the other</p>
<div class="sourceCode" id="cb28"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">df</span><span class="op">$</span><span class="va">`Lp(a) (nmol/L)`</span></span>
<span><span class="co">#&gt;  [1] 204.60   7.77  30.60     NA  18.76  86.02  33.85   6.33  31.30  33.85</span></span>
<span><span class="co">#&gt; [11]  38.89  58.27   9.88  29.37 252.60  34.02  10.87  46.92 147.29 127.64</span></span>
<span><span class="co">#&gt; [21] 234.93  13.59 197.37  87.43 495.45   3.97 125.88 156.62  74.34 195.71</span></span>
<span><span class="co">#&gt; [31]  20.55  78.89  43.26  75.14 129.45   3.37  55.59   4.65   8.23  79.85</span></span>
<span><span class="co">#&gt; [41] 111.89  43.40  46.76  51.67  33.39   4.65  20.83  44.61  14.11  55.33</span></span>
<span><span class="co">#&gt; [51] 326.37  74.65   2.36 273.65     NA 139.47   3.65</span></span></code></pre></div>
<div class="sourceCode" id="cb29"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">pairwise_wilcox</span> <span class="op">=</span> <span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/pairwise_wilcox_report.html">pairwise_wilcox_report</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">df</span>, </span>
<span>  variable <span class="op">=</span> <span class="st">"Lp(a) (nmol/L)"</span>, </span>
<span>  by <span class="op">=</span> <span class="st">"group"</span>,</span>
<span>  p_threshold <span class="op">=</span> <span class="fl">0.30</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt; Warning in ggsignif::geom_signif(data = annotation_significant,</span></span>
<span><span class="co">#&gt; ggplot2::aes(xmin = group1, : Ignoring unknown aesthetics: <span style="color: #00BB00;">xmin</span>,</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">xmax</span>, <span style="color: #00BB00;">annotations</span>, and <span style="color: #00BB00;">y_position</span></span></span></code></pre></div>
<div class="sourceCode" id="cb30"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">pairwise_wilcox</span><span class="op">$</span><span class="va">result</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 6 × 4</span></span></span>
<span><span class="co">#&gt;   variable_name  group1     group2     p_value</span></span>
<span><span class="co">#&gt;   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>          <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>      <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>        <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">1</span> Lp(a) (nmol/L) AF_disease AF_control   0.826</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">2</span> Lp(a) (nmol/L) EU_control AF_control   0.250</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">3</span> Lp(a) (nmol/L) EU_control AF_disease   0.738</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">4</span> Lp(a) (nmol/L) EU_disease AF_control   0.250</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">5</span> Lp(a) (nmol/L) EU_disease AF_disease   0.738</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">6</span> Lp(a) (nmol/L) EU_disease EU_control   0.738</span></span></code></pre></div>
<div class="sourceCode" id="cb31"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">pairwise_wilcox</span><span class="op">$</span><span class="va">significant_result</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 2 × 4</span></span></span>
<span><span class="co">#&gt;   variable_name  group1     group2     p_value</span></span>
<span><span class="co">#&gt;   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>          <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>      <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>        <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">1</span> Lp(a) (nmol/L) EU_control AF_control    0.25</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">2</span> Lp(a) (nmol/L) EU_disease AF_control    0.25</span></span></code></pre></div>
<div class="sourceCode" id="cb32"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">pairwise_wilcox</span><span class="op">$</span><span class="va">plot</span></span>
<span><span class="co">#&gt; Warning: Removed 2 rows containing non-finite outside the scale range</span></span>
<span><span class="co">#&gt; (`stat_boxplot()`).</span></span></code></pre></div>
<p><img src="introduction_files/figure-html/unnamed-chunk-21-1.png" class="r-plt" alt="" width="672"></p>
</div>
<div class="section level5">
<h5 id="normally-distributed-continuous-variables">Normally Distributed Continuous Variables<a class="anchor" aria-label="anchor" href="#normally-distributed-continuous-variables"></a>
</h5>
<p>Pairwise t-test (stats::pairwise.t.test) used here, barchart for mean
with errorbar (1 sd, not 95% CI) for illustration.</p>
<p>Still, if you are not confident with the normality or the sample
size, use the one for not normally distributed variable above.</p>
<div class="sourceCode" id="cb33"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">pairwise_t_test</span> <span class="op">=</span> <span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/pairwise_t_test_report.html">pairwise_t_test_report</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">df</span>, </span>
<span>  variable <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Calcium"</span>,<span class="st">"Albumin"</span><span class="op">)</span>, </span>
<span>  by <span class="op">=</span> <span class="st">"group"</span>,</span>
<span>  p_threshold <span class="op">=</span> <span class="fl">0.15</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt; Warning in ggsignif::geom_signif(data = annotation_significant,</span></span>
<span><span class="co">#&gt; ggplot2::aes(xmin = group1, : Ignoring unknown aesthetics: <span style="color: #00BB00;">xmin</span>,</span></span>
<span><span class="co">#&gt; <span style="color: #00BB00;">xmax</span>, <span style="color: #00BB00;">annotations</span>, and <span style="color: #00BB00;">y_position</span></span></span></code></pre></div>
<div class="sourceCode" id="cb34"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">pairwise_t_test</span><span class="op">$</span><span class="va">result</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 12 × 4</span></span></span>
<span><span class="co">#&gt;    variable_name group1     group2      p_value</span></span>
<span><span class="co">#&gt;    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>         <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>      <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>         <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 1</span> Albumin       AF_disease AF_control 0.061<span style="text-decoration: underline;">7</span>  </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 2</span> Albumin       EU_control AF_control 0.298   </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 3</span> Albumin       EU_control AF_disease 0.000<span style="text-decoration: underline;">379</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 4</span> Albumin       EU_disease AF_control 0.201   </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 5</span> Albumin       EU_disease AF_disease 0.225   </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 6</span> Albumin       EU_disease EU_control 0.000<span style="text-decoration: underline;">156</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 7</span> Calcium       AF_disease AF_control 0.416   </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 8</span> Calcium       EU_control AF_control 0.814   </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;"> 9</span> Calcium       EU_control AF_disease 0.261   </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">10</span> Calcium       EU_disease AF_control 0.439   </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">11</span> Calcium       EU_disease AF_disease 0.097<span style="text-decoration: underline;">7</span>  </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">12</span> Calcium       EU_disease EU_control 0.416</span></span></code></pre></div>
<div class="sourceCode" id="cb35"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">pairwise_t_test</span><span class="op">$</span><span class="va">significant_result</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 4 × 4</span></span></span>
<span><span class="co">#&gt;   variable_name group1     group2     p_value</span></span>
<span><span class="co">#&gt;   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>         <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>      <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>        <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">1</span> Albumin       AF_disease AF_control 0.062  </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">2</span> Albumin       EU_control AF_disease 0.000<span style="text-decoration: underline;">38</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">3</span> Albumin       EU_disease EU_control 0.000<span style="text-decoration: underline;">16</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">4</span> Calcium       EU_disease AF_disease 0.098</span></span></code></pre></div>
<div class="sourceCode" id="cb36"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">pairwise_t_test</span><span class="op">$</span><span class="va">plot</span></span></code></pre></div>
<p><img src="introduction_files/figure-html/unnamed-chunk-25-1.png" class="r-plt" alt="" width="672"></p>
<p>If you only want the first table in supplemental data, you can run
the one below as well. Replace “report” to “table” in the function name
and remove the p_threshold if you have. This function is internally
called in mirrorstoolkit::pairwise_t_test_report, so do the other
two.</p>
<div class="sourceCode" id="cb37"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/pairwise_t_test_table.html">pairwise_t_test_table</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">df</span>, </span>
<span>  variable <span class="op">=</span> <span class="st">"Calcium"</span>, </span>
<span>  by <span class="op">=</span> <span class="st">"group"</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 6 × 4</span></span></span>
<span><span class="co">#&gt;   variable_name group1     group2     p_value</span></span>
<span><span class="co">#&gt;   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>         <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>      <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>        <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">1</span> Calcium       AF_disease AF_control  0.416 </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">2</span> Calcium       EU_control AF_control  0.814 </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">3</span> Calcium       EU_control AF_disease  0.261 </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">4</span> Calcium       EU_disease AF_control  0.439 </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">5</span> Calcium       EU_disease AF_disease  0.097<span style="text-decoration: underline;">7</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">6</span> Calcium       EU_disease EU_control  0.416</span></span></code></pre></div>
</div>
</div>
</div>
<div class="section level3">
<h3 id="correlation-matrix">Correlation matrix<a class="anchor" aria-label="anchor" href="#correlation-matrix"></a>
</h3>
<p>We provide two functions for correlation matrix analysis.</p>
<div class="section level4">
<h4 id="to-visualize-the-correlation-matrix">To visualize the correlation matrix<a class="anchor" aria-label="anchor" href="#to-visualize-the-correlation-matrix"></a>
</h4>
<p>mirrorstoolkit::correlation_matrix_plot will return a ggplot2
instance, sharing a similar argument setting:</p>
<ul>
<li>variable is optional, if not provided, will use all the columns
provided in data. However, as a fool-proofing operation, providing the
variable column is a good sanity check as the function will internally
enforces all the value in your data table to numeric.</li>
<li>font_size is 5 by default for all labels.</li>
<li>please be aware that the order is reordered by hierarchical
clustering for better cluster illustration</li>
</ul>
<div class="sourceCode" id="cb38"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/correlation_matrix_plot.html">correlation_matrix_plot</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">df</span>,</span>
<span>  variable <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"BMI"</span>, <span class="st">"Albumin"</span>, <span class="st">"Calcium"</span>, <span class="st">"CRP"</span>, <span class="st">"apoB (nmol/L)"</span>,<span class="st">"%Lp(a)-ApoB100"</span><span class="op">)</span>,</span>
<span>  font_size <span class="op">=</span> <span class="fl">8</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt; Warning: `aes_string()` was deprecated in ggplot2 3.0.0.</span></span>
<span><span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> Please use tidy evaluation idioms with `aes()`.</span></span>
<span><span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> See also `vignette("ggplot2-in-packages")` for more information.</span></span>
<span><span class="co">#&gt; <span style="color: #00BBBB;">ℹ</span> The deprecated feature was likely used in the <span style="color: #0000BB;">ggcorrplot</span> package.</span></span>
<span><span class="co">#&gt;   Please report the issue at <span style="color: #0000BB; font-style: italic;">&lt;https://github.com/kassambara/ggcorrplot/issues&gt;</span>.</span></span>
<span><span class="co">#&gt; <span style="color: #555555;">This warning is displayed once per session.</span></span></span>
<span><span class="co">#&gt; <span style="color: #555555;">Call `lifecycle::last_lifecycle_warnings()` to see where this warning was</span></span></span>
<span><span class="co">#&gt; <span style="color: #555555;">generated.</span></span></span></code></pre></div>
<p><img src="introduction_files/figure-html/unnamed-chunk-27-1.png" class="r-plt" alt="" width="672"></p>
<p>This function also provide a filter operation by the p-value as long
as the p_threshold is specified. see below (I changed the way inputting
column here for illustration purpose, it doesn’t matter):</p>
<div class="sourceCode" id="cb39"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/correlation_matrix_plot.html">correlation_matrix_plot</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">df</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/select.html" class="external-link">select</a></span><span class="op">(</span></span>
<span>    <span class="fu">tidyr</span><span class="fu">::</span><span class="fu"><a href="https://tidyselect.r-lib.org/reference/all_of.html" class="external-link">all_of</a></span><span class="op">(</span><span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"BMI"</span>, <span class="st">"Albumin"</span>, <span class="st">"Calcium"</span>, <span class="st">"CRP"</span>, <span class="st">"apoB (nmol/L)"</span>,<span class="st">"%Lp(a)-ApoB100"</span><span class="op">)</span><span class="op">)</span></span>
<span>    <span class="op">)</span>,</span>
<span>  font_size <span class="op">=</span> <span class="fl">8</span>,</span>
<span>  p_threshold <span class="op">=</span> <span class="fl">0.05</span></span>
<span><span class="op">)</span></span></code></pre></div>
<p><img src="introduction_files/figure-html/unnamed-chunk-28-1.png" class="r-plt" alt="" width="672"></p>
</div>
<div class="section level4">
<h4 id="to-further-deal-with-the-correlation-matrix">To further deal with the correlation matrix<a class="anchor" aria-label="anchor" href="#to-further-deal-with-the-correlation-matrix"></a>
</h4>
<p>For potential multi-omics studies in the future, I left an function
here outputting the matrix as data.frame format, which removed all the
redundant columns. It has a p_threshold option same as above but I don’t
think it’s necessary here.</p>
<div class="sourceCode" id="cb40"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/correlation_matrix_dataframe.html">correlation_matrix_dataframe</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">df</span>,</span>
<span>  variable <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"BMI"</span>, <span class="st">"Albumin"</span>, <span class="st">"Calcium"</span>, <span class="st">"CRP"</span>, <span class="st">"apoB (nmol/L)"</span>,<span class="st">"%Lp(a)-ApoB100"</span><span class="op">)</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt;              row         column        corr      p_value</span></span>
<span><span class="co">#&gt; 1            BMI        Albumin  0.07246750 6.096735e-01</span></span>
<span><span class="co">#&gt; 2            BMI        Calcium -0.09116465 5.203815e-01</span></span>
<span><span class="co">#&gt; 3        Albumin        Calcium  0.33290055 1.486389e-02</span></span>
<span><span class="co">#&gt; 4            BMI            CRP  0.06434398 7.309315e-01</span></span>
<span><span class="co">#&gt; 5        Albumin            CRP -0.66165518 3.729085e-05</span></span>
<span><span class="co">#&gt; 6        Calcium            CRP -0.48625030 4.776512e-03</span></span>
<span><span class="co">#&gt; 7            BMI  apoB (nmol/L) -0.00519316 9.696990e-01</span></span>
<span><span class="co">#&gt; 8        Albumin  apoB (nmol/L)  0.25441832 6.600239e-02</span></span>
<span><span class="co">#&gt; 9        Calcium  apoB (nmol/L) -0.10436060 4.570736e-01</span></span>
<span><span class="co">#&gt; 10           CRP  apoB (nmol/L)  0.15435436 3.989530e-01</span></span>
<span><span class="co">#&gt; 11           BMI %Lp(a)-ApoB100  0.27960245 4.059818e-02</span></span>
<span><span class="co">#&gt; 12       Albumin %Lp(a)-ApoB100  0.17946329 2.076293e-01</span></span>
<span><span class="co">#&gt; 13       Calcium %Lp(a)-ApoB100  0.14327436 3.158588e-01</span></span>
<span><span class="co">#&gt; 14           CRP %Lp(a)-ApoB100 -0.11916085 5.231636e-01</span></span>
<span><span class="co">#&gt; 15 apoB (nmol/L) %Lp(a)-ApoB100 -0.12428106 3.659771e-01</span></span></code></pre></div>
</div>
</div>
<div class="section level3">
<h3 id="composition-component-trends">Composition Component Trends<a class="anchor" aria-label="anchor" href="#composition-component-trends"></a>
</h3>
<p>For the illustration of trends of a composition with a set of
components</p>
<div class="sourceCode" id="cb41"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">lipidomic_samples</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="fl">3</span><span class="op">)</span></span>
<span><span class="co">#&gt;        WES_ID Lp(a) (nmol/L) DG 30:0/14:0 DG 30:1/14:0 DG 32:0/16:0</span></span>
<span><span class="co">#&gt; 1 washei26416          157.0 0.0008071535 0.0001372855  0.003098725</span></span>
<span><span class="co">#&gt; 2 washei27458           37.1 0.0009333268 0.0002105400  0.004237114</span></span>
<span><span class="co">#&gt; 3 washei28197           15.0 0.0032868436 0.0010452749  0.013764988</span></span>
<span><span class="co">#&gt;   DG 32:1/16:0 DG 32:2/16:1 DG 34:0/16:0 DG 34:1/16:0 DG 34:2/16:0 DG 34:2/16:1</span></span>
<span><span class="co">#&gt; 1 0.0006956733 0.0002478549  0.001698103  0.004516096 0.0013578084  0.001842253</span></span>
<span><span class="co">#&gt; 2 0.0007368318 0.0007199269  0.002174789  0.002236245 0.0005861636  0.001207711</span></span>
<span><span class="co">#&gt; 3 0.0036933192 0.0031672078  0.005407289  0.004072219 0.0021591445  0.003816118</span></span>
<span><span class="co">#&gt;   DG 36:0/18:0 DG 36:1/18:0 DG 36:2/18:0 DG 36:2/18:1 DG 36:3/18:1 DG 36:4/18:0</span></span>
<span><span class="co">#&gt; 1  0.001415583  0.003683802 0.0007110398  0.009479445 0.0020066608 9.143163e-06</span></span>
<span><span class="co">#&gt; 2  0.004551087  0.002515179 0.0005592532  0.005935862 0.0009731812 2.037506e-05</span></span>
<span><span class="co">#&gt; 3  0.009499735  0.005311344 0.0016281908  0.012059293 0.0022217465 0.000000e+00</span></span>
<span><span class="co">#&gt;   DG 38:1/18:0 DG 38:2/18:0 DG 38:2/18:1 DG 38:3/18:0 DG 38:3/18:1 DG 38:4/18:0</span></span>
<span><span class="co">#&gt; 1 0.0001244584 0.0025175506 0.0009816070 0.0007872274 0.0019228689 4.408206e-05</span></span>
<span><span class="co">#&gt; 2 0.0000000000 0.0010531250 0.0011219876 0.0013524833 0.0004459673 4.037663e-05</span></span>
<span><span class="co">#&gt; 3 0.0002654990 0.0006729207 0.0008393582 0.0053460809 0.0004614999 1.576657e-04</span></span>
<span><span class="co">#&gt;   DG 38:4/18:1 DG 40:4/18:0 DG 40:4/18:1 DG 40:5/18:0 DG 40:5/18:1 DG 40:6/18:0</span></span>
<span><span class="co">#&gt; 1 0.0003366250 1.112712e-05 1.190158e-05 4.217323e-05 1.682002e-05 9.500192e-05</span></span>
<span><span class="co">#&gt; 2 0.0003986452 2.709289e-05 1.349291e-05 5.475041e-06 1.950699e-05 1.778880e-04</span></span>
<span><span class="co">#&gt; 3 0.0018140866 8.260741e-05 9.626816e-05 2.173361e-04 6.999751e-05 5.937550e-04</span></span>
<span><span class="co">#&gt;   DG 40:6/18:1</span></span>
<span><span class="co">#&gt; 1 7.053647e-05</span></span>
<span><span class="co">#&gt; 2 5.693585e-05</span></span>
<span><span class="co">#&gt; 3 1.035781e-04</span></span></code></pre></div>
<p>You can either specify the column you want by variable = c(“col1”,
“col2”, “col3”) if you need, but most of the times it will be quite
lengthy. If it is not specified, it will use all the columns not
mentioned in the table.</p>
<div class="sourceCode" id="cb42"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/trend_bar_chart.html">trend_bar_chart</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">lipidomic_samples</span>,</span>
<span>  id <span class="op">=</span> <span class="st">"WES_ID"</span>,</span>
<span>  by <span class="op">=</span> <span class="st">"Lp(a) (nmol/L)"</span>,</span>
<span>  legend_title <span class="op">=</span> <span class="st">"DG"</span></span>
<span><span class="op">)</span></span></code></pre></div>
<p><img src="introduction_files/figure-html/unnamed-chunk-31-1.png" class="r-plt" alt="" width="672"></p>
</div>
</div>
<div class="section level2">
<h2 id="association-studies">Association Studies<a class="anchor" aria-label="anchor" href="#association-studies"></a>
</h2>
<p>For those you are interested, you might want to log them if you think
they are not normally distributed:</p>
<div class="sourceCode" id="cb43"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">df</span> <span class="op">=</span> <span class="va">df</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span></span>
<span>  LDL <span class="op">=</span> <span class="va">`LDL (calc) mg/dL`</span>,</span>
<span>  lpa_logged <span class="op">=</span> <span class="va">`Lp(a) (nmol/L)`</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> <span class="fu">base</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/r/base/Log.html" class="external-link">log2</a></span><span class="op">(</span><span class="op">)</span>,</span>
<span><span class="op">)</span></span></code></pre></div>
<p>OLS_wrapper and logistic_wrapper() can deal with most of the
association analysis we need. In most of our tasks, we can specify the
follow 2 or 3 lists as global settings used repetitively.</p>
<div class="sourceCode" id="cb44"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># The adjustment variable you need</span></span>
<span><span class="va">adjustments</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>  <span class="st">"Age"</span>, </span>
<span>  <span class="st">"Gender"</span>, </span>
<span>  <span class="st">"AF"</span>, </span>
<span>  <span class="st">"BMI"</span>,</span>
<span>  <span class="st">"Smoking"</span>,</span>
<span>  <span class="st">"antiplatelet_group"</span>, </span>
<span>  <span class="st">"anticoagulant_group"</span>, </span>
<span>  <span class="st">"statin_group"</span></span>
<span>  <span class="op">)</span></span>
<span><span class="co"># The variable you are interested in</span></span>
<span><span class="va">variable_list</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>  <span class="st">"Albumin"</span>,</span>
<span>  <span class="st">"Calcium"</span>,</span>
<span>  <span class="st">"LDL"</span>,</span>
<span>  <span class="st">"wIS"</span>, </span>
<span>  <span class="st">"lpa_logged"</span><span class="op">)</span></span>
<span><span class="co"># In case the column name is not what you want in the output </span></span>
<span><span class="va">variable_list_formal</span> <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span></span>
<span>  <span class="st">"Albumin"</span>,</span>
<span>  <span class="st">"Calcium"</span>,</span>
<span>  <span class="st">"LDL(calc) mg/dL"</span>,</span>
<span>  <span class="st">"wIS"</span>,</span>
<span>  <span class="st">"Lp(a) nmol/L, 2-logged"</span></span>
<span>  <span class="op">)</span></span></code></pre></div>
<div class="section level3">
<h3 id="logistics-regression">Logistics regression<a class="anchor" aria-label="anchor" href="#logistics-regression"></a>
</h3>
<p>The wrapper is runing in the following way:</p>
<ul>
<li>For each column name mentioned in <em>variable_list</em>, let’s call
it <em>variable_i</em>
<ul>
<li>Grab the columns mentioned in <em>variable_i</em>,
<em>adjustments</em>, and <em>response</em> from <em>data</em>. If
<em>variable_i</em> is in <em>adjustments</em>, will remove the
duplicates</li>
<li>Logistic regression <em>response</em>
<math display="inline" xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mo>∼</mo><annotation encoding="application/x-tex">\sim</annotation></semantics></math><em>variable_i</em> + <em>adjustment</em> + 1</li>
<li>Seek the effect size and coefficient of <em>variable_i</em>, compute
the odd ratio by exp(effect size)</li>
</ul>
</li>
<li>Collect all the data</li>
</ul>
<p>What you need to specify is:</p>
<ul>
<li>data, your dataframe</li>
<li>response, which the column name of your target (y)</li>
<li>adjustments, you adjustment list defined above</li>
<li>by, if you want to categorize the data</li>
<li>variable_of_interest, your variable list</li>
<li>variable_of_interest_formal_name, the variable name list in the
data</li>
</ul>
<div class="sourceCode" id="cb45"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/logistic_wrapper.html">logistic_wrapper</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">df</span>, </span>
<span>  response <span class="op">=</span> <span class="st">"Calciphylaxis"</span>,</span>
<span>  adjustments <span class="op">=</span> <span class="va">adjustments</span>,</span>
<span>  variable_of_interest <span class="op">=</span> <span class="va">variable_list</span>,</span>
<span>  variable_of_interest_formal_name <span class="op">=</span> <span class="va">variable_list_formal</span></span>
<span>  <span class="op">)</span></span>
<span><span class="co">#&gt;                         Effect Size   Odd Ratio    P-value</span></span>
<span><span class="co">#&gt; Albumin                -5.170826684 0.005679871 0.00327561</span></span>
<span><span class="co">#&gt; Calcium                 0.833885757 2.302247357 0.11224877</span></span>
<span><span class="co">#&gt; LDL(calc) mg/dL         0.006642994 1.006665108 0.61218217</span></span>
<span><span class="co">#&gt; wIS                     0.003948747 1.003956554 0.95723792</span></span>
<span><span class="co">#&gt; Lp(a) nmol/L, 2-logged  0.014135081 1.014235454 0.94323132</span></span></code></pre></div>
</div>
<div class="section level3">
<h3 id="ordinary-linear-regression">Ordinary Linear Regression<a class="anchor" aria-label="anchor" href="#ordinary-linear-regression"></a>
</h3>
<p>The setting is similar, let’s generate a continuous target
variable:</p>
<div class="sourceCode" id="cb46"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">df_disease</span> <span class="op">=</span><span class="va">df</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> </span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/filter.html" class="external-link">filter</a></span><span class="op">(</span><span class="va">Calciphylaxis</span> <span class="op">==</span> <span class="fl">1</span><span class="op">)</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> </span>
<span>  <span class="fu">dplyr</span><span class="fu">::</span><span class="fu"><a href="https://dplyr.tidyverse.org/reference/mutate.html" class="external-link">mutate</a></span><span class="op">(</span></span>
<span>    duration_before_diagnosis <span class="op">=</span> <span class="op">(</span><span class="va">Calciphylaxis_diagnosis_date</span> <span class="op">-</span> <span class="va">HD_start_date</span><span class="op">)</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span></span>
<span>      <span class="fu">lubridate</span><span class="fu">::</span><span class="fu"><a href="https://lubridate.tidyverse.org/reference/as.duration.html" class="external-link">as.duration</a></span><span class="op">(</span><span class="op">)</span><span class="op">/</span> <span class="fu">lubridate</span><span class="fu">::</span><span class="fu"><a href="https://lubridate.tidyverse.org/reference/duration.html" class="external-link">dmonths</a></span><span class="op">(</span><span class="fl">1</span><span class="op">)</span>,</span>
<span>    duration_before_diagnosis <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/Extremes.html" class="external-link">pmax</a></span><span class="op">(</span><span class="va">duration_before_diagnosis</span>,<span class="fl">0</span><span class="op">)</span><span class="op">)</span></span>
<span><span class="va">df_disease</span><span class="op">$</span><span class="va">duration_before_diagnosis</span></span>
<span><span class="co">#&gt;  [1] 78.7186858 47.6714579 92.9117043 39.3593429  0.0000000  2.2340862</span></span>
<span><span class="co">#&gt;  [7] 23.7535934  3.9425051 26.0205339  2.9240246 23.1950719  2.9568789</span></span>
<span><span class="co">#&gt; [13] 28.3860370  0.7227926 65.5770021  9.3963039  3.7453799 83.8439425</span></span>
<span><span class="co">#&gt; [19] 27.2032854  0.0000000 24.7392197 12.4517454 10.8418891 28.4517454</span></span>
<span><span class="co">#&gt; [25]  1.9712526 50.9240246  0.9527721</span></span></code></pre></div>
<p>I called it OLS because machine learning community call it so…
Forgive me. Ordinary linear regression doesn’t have OR defined thus only
beta and p-value here.</p>
<div class="sourceCode" id="cb47"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/OLS_wrapper.html">OLS_wrapper</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">df_disease</span>,</span>
<span>  response <span class="op">=</span> <span class="st">"duration_before_diagnosis"</span>,</span>
<span>  adjustments <span class="op">=</span> <span class="va">adjustments</span>,</span>
<span>  variable_of_interest <span class="op">=</span> <span class="va">variable_list</span>,</span>
<span>  variable_of_interest_formal_name <span class="op">=</span> <span class="va">variable_list_formal</span></span>
<span><span class="op">)</span></span>
<span><span class="co">#&gt;                        Effect Size     P-value</span></span>
<span><span class="co">#&gt; Albumin                 9.38512237 0.419285024</span></span>
<span><span class="co">#&gt; Calcium                23.95816412 0.004014825</span></span>
<span><span class="co">#&gt; LDL(calc) mg/dL         0.08337316 0.749217682</span></span>
<span><span class="co">#&gt; wIS                    -1.01676960 0.442564703</span></span>
<span><span class="co">#&gt; Lp(a) nmol/L, 2-logged  0.51243742 0.906371748</span></span></code></pre></div>
</div>
</div>
<div class="section level2">
<h2 id="summarize-association-studies">Summarize association studies<a class="anchor" aria-label="anchor" href="#summarize-association-studies"></a>
</h2>
<div class="section level3">
<h3 id="manhattan-plot">Manhattan Plot<a class="anchor" aria-label="anchor" href="#manhattan-plot"></a>
</h3>
<p>A quick Manhattan plot API is provided. A minimal data input can be
the following</p>
<div class="sourceCode" id="cb48"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">association_result</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="fl">3</span><span class="op">)</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 3 × 4</span></span></span>
<span><span class="co">#&gt;   SNP        pos phenotype    `FDR_adjusted_p-value`</span></span>
<span><span class="co">#&gt;   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>    <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>                         <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">1</span> 412-C/T    412 Glucose                      0.022<span style="text-decoration: underline;">0</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">2</span> 4785-C/A  <span style="text-decoration: underline;">4</span>785 Hypertension                 0.031<span style="text-decoration: underline;">6</span></span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">3</span> 387-G/C    387 Hypertension                 0.031<span style="text-decoration: underline;">6</span></span></span></code></pre></div>
<p>Simply provide the column names to the API.</p>
<ul>
<li>pos: the position of SNV</li>
<li>phenotype: the phenotype it links to</li>
<li>p_value: the p_value of the association</li>
<li>formal_name: give name to each SNV</li>
<li>phenotype_list: the list of phenotype you want to work on.</li>
<li>p_value_threshold: default 0.05, which will affect the significance
level (red horizontal line), all points above this line should be
labelled unless there are too many overlapping</li>
</ul>
<div class="sourceCode" id="cb49"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/manhattan_plot.html">manhattan_plot</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">association_result</span>,</span>
<span>  pos <span class="op">=</span> <span class="st">"pos"</span>,</span>
<span>  phenotype <span class="op">=</span> <span class="st">"phenotype"</span>,</span>
<span>  p_value <span class="op">=</span> <span class="st">"FDR_adjusted_p-value"</span>,</span>
<span>  formal_name <span class="op">=</span> <span class="st">"SNP"</span>,</span>
<span>  phenotype_list <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"Hypertension"</span>, <span class="st">"HDL"</span>, <span class="st">"Glucose"</span><span class="op">)</span></span>
<span><span class="op">)</span></span></code></pre></div>
<p><img src="introduction_files/figure-html/unnamed-chunk-38-1.png" class="r-plt" alt="" width="672">
It’s also okay for single phenotype, with another p-value threshold</p>
<div class="sourceCode" id="cb50"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/manhattan_plot.html">manhattan_plot</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">association_result</span>,</span>
<span>  pos <span class="op">=</span> <span class="st">"pos"</span>,</span>
<span>  phenotype <span class="op">=</span> <span class="st">"phenotype"</span>,</span>
<span>  p_value <span class="op">=</span> <span class="st">"FDR_adjusted_p-value"</span>,</span>
<span>  formal_name <span class="op">=</span> <span class="st">"SNP"</span>,</span>
<span>  phenotype_list <span class="op">=</span> <span class="st">"Hypertension"</span>,</span>
<span>  p_value_threshold <span class="op">=</span> <span class="fl">0.10</span></span>
<span><span class="op">)</span></span></code></pre></div>
<p><img src="introduction_files/figure-html/unnamed-chunk-39-1.png" class="r-plt" alt="" width="672"></p>
</div>
<div class="section level3">
<h3 id="directed-acyclic-graph-for-multi-omics-data">Directed Acyclic Graph for multi-omics data<a class="anchor" aria-label="anchor" href="#directed-acyclic-graph-for-multi-omics-data"></a>
</h3>
<p>For data like these…</p>
<div class="sourceCode" id="cb51"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">multi_omics_association_result</span> <span class="op"><a href="../reference/pipe.html">%&gt;%</a></span> <span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="fl">5</span><span class="op">)</span></span>
<span><span class="co">#&gt; <span style="color: #949494;"># A tibble: 5 × 6</span></span></span>
<span><span class="co">#&gt;   from     to     beta `FDR_adjusted_p-value` from_type    to_type  </span></span>
<span><span class="co">#&gt;   <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>    <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span> <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span>                  <span style="color: #949494; font-style: italic;">&lt;dbl&gt;</span> <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>        <span style="color: #949494; font-style: italic;">&lt;chr&gt;</span>    </span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">1</span> 4894-G/A lpa    1.36                 0.021<span style="text-decoration: underline;">1</span> genotype_SNP phenotype</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">2</span> 620-A/G  lpa    1.43                 0.021<span style="text-decoration: underline;">1</span> genotype_SNP phenotype</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">3</span> 621-T/A  lpa    1.43                 0.021<span style="text-decoration: underline;">1</span> genotype_SNP phenotype</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">4</span> 4922-C/G lpa    1.49                 0.021<span style="text-decoration: underline;">1</span> genotype_SNP phenotype</span></span>
<span><span class="co">#&gt; <span style="color: #BCBCBC;">5</span> 4970-T/A lpa    1.35                 0.021<span style="text-decoration: underline;">1</span> genotype_SNP phenotype</span></span></code></pre></div>
<p>You can quickly generate these by networkX, it can be saved as an
interactive html files if you need.</p>
<div class="sourceCode" id="cb52"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">dag</span> <span class="op">=</span> <span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/multi_omics_directed_acyclic_graph.html">multi_omics_directed_acyclic_graph</a></span><span class="op">(</span></span>
<span>  data <span class="op">=</span> <span class="va">multi_omics_association_result</span>,</span>
<span>  from <span class="op">=</span> <span class="st">"from"</span>,</span>
<span>  to <span class="op">=</span> <span class="st">"to"</span>,</span>
<span>  beta <span class="op">=</span> <span class="st">"beta"</span>,</span>
<span>  p_value <span class="op">=</span> <span class="st">"FDR_adjusted_p-value"</span>,</span>
<span>  from_class <span class="op">=</span> <span class="st">"from_type"</span>,</span>
<span>  to_class <span class="op">=</span> <span class="st">"to_type"</span></span>
<span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb53"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">dag</span><span class="op">$</span><span class="va">illustration_network</span></span></code></pre></div>
<div id="htmlwidget-ac96cb3ee4656e2e9ec3" style="width:100%;height:432px;" class="visNetwork html-widget"></div>
<script type="application/json" data-for="htmlwidget-ac96cb3ee4656e2e9ec3">{"x":{"nodes":{"id":["4894-G/A","620-A/G","621-T/A","4922-C/G","4970-T/A","649-T/C","4908-A/G","943-T/A","632-A/G","654-C/T","4605-G/C","476-C/T","426-T/C","4967-G/A","4888-T/C","5084-C/G","4951-A/C","937-T/C","970-G/A","698-C/T","4966-G/A","4733-G/A","4245-C/T","661-G/A","4551-A/C","4910-G/C","4963-G/A","4989-C/T","586-A/C","663-C/T","4754-A/T","566-G/C","4740-C/T","4816-T/A","4755-T/C","4942-C/T","4774-A/G","568-G/C","4933-G/A","588-T/G","4735-A/G","4943-A/G","690-A/C","600-T/C","647-C/T","4785-C/A","386-A/G","4807-G/A","650-A/C","826-A/G","4923-A/C","4783-G/C","940-T/G","4862-G/T","4846-A/G","954-G/C","595-A/G","4858-A/T","184-A/G","AC","PA","NSer","Cer","dhCer","LPI","LPEp","LPS","Sulf","PEp","PE","LPE","PS","BMP","DG","NAPS","MG","PG","MhCer","LPCe","AcylPG","LPC","PC","TG","PCe","NAPE","Apolipoprotein B-100","Complement C1s subcomponent","Apolipoprotein(a)","Apolipoprotein D","Complement factor H","Complement C1q subcomponent subunit B","Apolipoprotein E","Serum amyloid A-4 protein","Immunoglobulin heavy constant mu","Complement C1q subcomponent subunit C","Immunoglobulin heavy variable 1-2","412-C/T","166-G/A","387-G/C","727-T/C","402-G/A","4527-A/T","4653-A/T","723-A/G","4541-G/A","Complement_C1s_subcomponent","Apolipoprotein_B-100","Complement_factor_B","Apolipoprotein_E","Properdin","Serum_paraoxonase_arylesterase_1","Complement_C3","Apolipoprotein_C-I","Transthyretin","Serum_amyloid_A-4_protein","Clusterin","Sulfhydryl_oxidase_1","Prenylcysteine_oxidase_1","Apolipoprotein_C-IV","Complement_factor_H","Apolipoprotein_A-II","Immunoglobulin_heavy_variable_1-2","Inter-alpha-trypsin_inhibitor_heavy_chain_H4","Heparin_cofactor_2","VNTR_estimate","5055-A/G","5052-A/T","5069-C/T","5037-T/G","4599-T/A","640-C/T","4604-G/C","2470-C/T","4734-G/T","612-G/C","4577-G/A","4503-A/G","1028-C/G","421-T/A","681-A/G","751-G/C","520-G/T","4612-T/A","5040-G/C","4889-A/T","494-A/C","387-G/T","199-C/T","728-A/G","653-C/A","1046-G/A","4644-G/A","4696-T/A","591-G/T","707-C/G","959-C/G","4976-G/T","606-G/T","578-C/T","546-G/T","1005-G/A","917-C/T","4473-C/T","1144-A/G","781-G/T","260-G/A","748-T/G","5004-C/A","602-G/A","4572-G/A","464-A/G","5092-G/T","4671-C/A","4674-A/C","4704-G/C","4685-T/C","1053-G/A","1044-A/G","4672-A/G","4441-A/G","4919-G/A","651-C/G","4683-T/C","4842-C/T","5078-G/A","4266-G/T","950-A/G","372-C/T","4260-A/C","1323-T/C","4755-T/G","4228-G/A","5013-G/T","4755-T/A","690-A/G","4583-T/A","4598-T/C","4623-T/C","717-G/A","824-C/A","907-A/G","835-T/C","4738-T/G","746-T/C","4716-G/T","1091-C/A","1010-G/A","4461-G/C","4462-T/C","572-T/G","185-A/G","1205-T/A","150-G/T","1070-A/G","1269-A/G","1217-A/T","708-G/A","357-C/T","4639-A/T","4640-G/C","4426-G/T","410-T/A","406-C/T","837-T/C","398-G/T","4436-A/G","4433-C/T","4355-T/C","411-G/T","4893-C/A","314-C/T","393-G/A","820-T/G","818-G/C","4279-C/A","342-G/T","353-G/C","4788-C/G","lpa","wIS","Glucose (mg/dL)","Hypertension","FC","CE","LacCer","PI","GM3","Immunoglobulin heavy variable 1-3","Complement C4-A","Alpha-1-acid glycoprotein 1","Albumin","Immunoglobulin heavy variable 4-34","Keratin, type I cytoskeletal 9","Ceruloplasmin","Complement component C7","Immunoglobulin heavy variable 3-15","Complement C3","Complement C1q subcomponent subunit A","Vitamin D-binding protein","Probable non-functional immunoglobulin kappa variable 3-7","Immunoglobulin heavy variable 5-51","Complement C4-B","Kallistatin","Platelet_basic_protein","Basement_membrane-specific_heparan_sulfate_proteoglycan_core_protein","Complement_C1q_subcomponent_subunit_B","Plasminogen","Immunoglobulin_heavy_variable_5-51","Complement_C1q_subcomponent_subunit_A","Immunoglobulin_heavy_variable_1-24","Isoform_1_of_Fibronectin","Keratin__type_I_cytoskeletal_9","Isoform_1_of_Immunoglobulin_heavy_constant_alpha_1","Complement_C1q_subcomponent_subunit_C","Immunoglobulin_heavy_constant_alpha_2","Alpha-2-HS-glycoprotein","Immunoglobulin_heavy_variable_1-46","Keratin__type_I_cytoskeletal_13","Alpha-1-antitrypsin","BPI_fold-containing_family_B_member_1","Extracellular_matrix_protein_1","N-acetylmuramoyl-L-alanine_amidase","Keratin__type_II_cytoskeletal_1b","Intelectin-1","Immunoglobulin_kappa_variable_1-8","Immunoglobulin_heavy_variable_3-72","Immunoglobulin_heavy_variable_3-7","Complement_component_C8_beta_chain","Tsukushi","Cathelicidin_antimicrobial_peptide","Lipopolysaccharide-binding_protein","Complement_C1r_subcomponent","Immunoglobulin_heavy_variable_1-18","Fibrinogen_alpha_chain","Complement_C4-A","Complement_component_C8_gamma_chain","Alpha-1B-glycoprotein","Apolipoprotein_A-I","Heat_shock_70_kDa_protein_1B","Fructose-bisphosphate_aldolase_A","Inter-alpha-trypsin_inhibitor_heavy_chain_H1","L-lactate_dehydrogenase_A_chain","Serpin_B4","Tubulin_alpha-1A_chain","Serpin_B3","Protein-glutamine_gamma-glutamyltransferase_K","Annexin_A2","14-3-3_protein_sigma","Cellular_retinoic_acid-binding_protein_2","Annexin_A1","Prelamin-A_C","Endoplasmic_reticulum_chaperone_BiP","Heat_shock_protein_HSP_90-beta","Histone_H4","Elongation_factor_1-alpha_2","Alpha-enolase","Triosephosphate_isomerase","Pyruvate_kinase_PKM","ATP_synthase_subunit_alpha__mitochondrial","Heat_shock_protein_beta-1","Vimentin","Galectin-7","Hemopexin","Zinc-alpha-2-glycoprotein","Protein_S100-A9","Glyceraldehyde-3-phosphate_dehydrogenase","14-3-3_protein_zeta_delta","Protein_S100-A8","Desmoplakin","Serum_amyloid_P-component","Cystatin-A","Sex_hormone-binding_globulin","Sushi__von_Willebrand_factor_type_A__EGF_and_pentraxin_domain-containing_protein_1","Keratin__type_I_cytoskeletal_17","Plasma_protease_C1_inhibitor","Haptoglobin","Junction_plakoglobin","Vitamin_D-binding_protein","Keratin__type_II_cytoskeletal_5","Angiotensinogen","Protein_S100-A7","Monocyte_differentiation_antigen_CD14"],"group":["genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","genotype_VNTR","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","genotype_SNP","phenotype","phenotype","phenotype","phenotype","lipidomics","lipidomics","lipidomics","lipidomics","lipidomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics","proteomics"],"level":[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,3,3,3,3,3,3,3,3,3,3,3,1,1,1,1,1,1,1,1,1,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,4,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,5,5,5,5,2,2,2,2,2,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3,3],"label":["4894-G/A","620-A/G","621-T/A","4922-C/G","4970-T/A","649-T/C","4908-A/G","943-T/A","632-A/G","654-C/T","4605-G/C","476-C/T","426-T/C","4967-G/A","4888-T/C","5084-C/G","4951-A/C","937-T/C","970-G/A","698-C/T","4966-G/A","4733-G/A","4245-C/T","661-G/A","4551-A/C","4910-G/C","4963-G/A","4989-C/T","586-A/C","663-C/T","4754-A/T","566-G/C","4740-C/T","4816-T/A","4755-T/C","4942-C/T","4774-A/G","568-G/C","4933-G/A","588-T/G","4735-A/G","4943-A/G","690-A/C","600-T/C","647-C/T","4785-C/A","386-A/G","4807-G/A","650-A/C","826-A/G","4923-A/C","4783-G/C","940-T/G","4862-G/T","4846-A/G","954-G/C","595-A/G","4858-A/T","184-A/G","AC","PA","NSer","Cer","dhCer","LPI","LPEp","LPS","Sulf","PEp","PE","LPE","PS","BMP","DG","NAPS","MG","PG","MhCer","LPCe","AcylPG","LPC","PC","TG","PCe","NAPE","Apolipoprotein B-100","Complement C1s subcomponent","Apolipoprotein(a)","Apolipoprotein D","Complement factor H","Complement C1q subcomponent subunit B","Apolipoprotein E","Serum amyloid A-4 protein","Immunoglobulin heavy constant mu","Complement C1q subcomponent subunit C","Immunoglobulin heavy variable 1-2","412-C/T","166-G/A","387-G/C","727-T/C","402-G/A","4527-A/T","4653-A/T","723-A/G","4541-G/A","Complement_C1s_subcomponent","Apolipoprotein_B-100","Complement_factor_B","Apolipoprotein_E","Properdin","Serum_paraoxonase_arylesterase_1","Complement_C3","Apolipoprotein_C-I","Transthyretin","Serum_amyloid_A-4_protein","Clusterin","Sulfhydryl_oxidase_1","Prenylcysteine_oxidase_1","Apolipoprotein_C-IV","Complement_factor_H","Apolipoprotein_A-II","Immunoglobulin_heavy_variable_1-2","Inter-alpha-trypsin_inhibitor_heavy_chain_H4","Heparin_cofactor_2","VNTR_estimate","5055-A/G","5052-A/T","5069-C/T","5037-T/G","4599-T/A","640-C/T","4604-G/C","2470-C/T","4734-G/T","612-G/C","4577-G/A","4503-A/G","1028-C/G","421-T/A","681-A/G","751-G/C","520-G/T","4612-T/A","5040-G/C","4889-A/T","494-A/C","387-G/T","199-C/T","728-A/G","653-C/A","1046-G/A","4644-G/A","4696-T/A","591-G/T","707-C/G","959-C/G","4976-G/T","606-G/T","578-C/T","546-G/T","1005-G/A","917-C/T","4473-C/T","1144-A/G","781-G/T","260-G/A","748-T/G","5004-C/A","602-G/A","4572-G/A","464-A/G","5092-G/T","4671-C/A","4674-A/C","4704-G/C","4685-T/C","1053-G/A","1044-A/G","4672-A/G","4441-A/G","4919-G/A","651-C/G","4683-T/C","4842-C/T","5078-G/A","4266-G/T","950-A/G","372-C/T","4260-A/C","1323-T/C","4755-T/G","4228-G/A","5013-G/T","4755-T/A","690-A/G","4583-T/A","4598-T/C","4623-T/C","717-G/A","824-C/A","907-A/G","835-T/C","4738-T/G","746-T/C","4716-G/T","1091-C/A","1010-G/A","4461-G/C","4462-T/C","572-T/G","185-A/G","1205-T/A","150-G/T","1070-A/G","1269-A/G","1217-A/T","708-G/A","357-C/T","4639-A/T","4640-G/C","4426-G/T","410-T/A","406-C/T","837-T/C","398-G/T","4436-A/G","4433-C/T","4355-T/C","411-G/T","4893-C/A","314-C/T","393-G/A","820-T/G","818-G/C","4279-C/A","342-G/T","353-G/C","4788-C/G","lpa","wIS","Glucose (mg/dL)","Hypertension","FC","CE","LacCer","PI","GM3","Immunoglobulin heavy variable 1-3","Complement C4-A","Alpha-1-acid glycoprotein 1","Albumin","Immunoglobulin heavy variable 4-34","Keratin, type I cytoskeletal 9","Ceruloplasmin","Complement component C7","Immunoglobulin heavy variable 3-15","Complement C3","Complement C1q subcomponent subunit A","Vitamin D-binding protein","Probable non-functional immunoglobulin kappa variable 3-7","Immunoglobulin heavy variable 5-51","Complement C4-B","Kallistatin","Platelet_basic_protein","Basement_membrane-specific_heparan_sulfate_proteoglycan_core_protein","Complement_C1q_subcomponent_subunit_B","Plasminogen","Immunoglobulin_heavy_variable_5-51","Complement_C1q_subcomponent_subunit_A","Immunoglobulin_heavy_variable_1-24","Isoform_1_of_Fibronectin","Keratin__type_I_cytoskeletal_9","Isoform_1_of_Immunoglobulin_heavy_constant_alpha_1","Complement_C1q_subcomponent_subunit_C","Immunoglobulin_heavy_constant_alpha_2","Alpha-2-HS-glycoprotein","Immunoglobulin_heavy_variable_1-46","Keratin__type_I_cytoskeletal_13","Alpha-1-antitrypsin","BPI_fold-containing_family_B_member_1","Extracellular_matrix_protein_1","N-acetylmuramoyl-L-alanine_amidase","Keratin__type_II_cytoskeletal_1b","Intelectin-1","Immunoglobulin_kappa_variable_1-8","Immunoglobulin_heavy_variable_3-72","Immunoglobulin_heavy_variable_3-7","Complement_component_C8_beta_chain","Tsukushi","Cathelicidin_antimicrobial_peptide","Lipopolysaccharide-binding_protein","Complement_C1r_subcomponent","Immunoglobulin_heavy_variable_1-18","Fibrinogen_alpha_chain","Complement_C4-A","Complement_component_C8_gamma_chain","Alpha-1B-glycoprotein","Apolipoprotein_A-I","Heat_shock_70_kDa_protein_1B","Fructose-bisphosphate_aldolase_A","Inter-alpha-trypsin_inhibitor_heavy_chain_H1","L-lactate_dehydrogenase_A_chain","Serpin_B4","Tubulin_alpha-1A_chain","Serpin_B3","Protein-glutamine_gamma-glutamyltransferase_K","Annexin_A2","14-3-3_protein_sigma","Cellular_retinoic_acid-binding_protein_2","Annexin_A1","Prelamin-A_C","Endoplasmic_reticulum_chaperone_BiP","Heat_shock_protein_HSP_90-beta","Histone_H4","Elongation_factor_1-alpha_2","Alpha-enolase","Triosephosphate_isomerase","Pyruvate_kinase_PKM","ATP_synthase_subunit_alpha__mitochondrial","Heat_shock_protein_beta-1","Vimentin","Galectin-7","Hemopexin","Zinc-alpha-2-glycoprotein","Protein_S100-A9","Glyceraldehyde-3-phosphate_dehydrogenase","14-3-3_protein_zeta_delta","Protein_S100-A8","Desmoplakin","Serum_amyloid_P-component","Cystatin-A","Sex_hormone-binding_globulin","Sushi__von_Willebrand_factor_type_A__EGF_and_pentraxin_domain-containing_protein_1","Keratin__type_I_cytoskeletal_17","Plasma_protease_C1_inhibitor","Haptoglobin","Junction_plakoglobin","Vitamin_D-binding_protein","Keratin__type_II_cytoskeletal_5","Angiotensinogen","Protein_S100-A7","Monocyte_differentiation_antigen_CD14"],"title":["<p><b>genotype_SNP<\/b><br>4894-G/A<\/p>","<p><b>genotype_SNP<\/b><br>620-A/G<\/p>","<p><b>genotype_SNP<\/b><br>621-T/A<\/p>","<p><b>genotype_SNP<\/b><br>4922-C/G<\/p>","<p><b>genotype_SNP<\/b><br>4970-T/A<\/p>","<p><b>genotype_SNP<\/b><br>649-T/C<\/p>","<p><b>genotype_SNP<\/b><br>4908-A/G<\/p>","<p><b>genotype_SNP<\/b><br>943-T/A<\/p>","<p><b>genotype_SNP<\/b><br>632-A/G<\/p>","<p><b>genotype_SNP<\/b><br>654-C/T<\/p>","<p><b>genotype_SNP<\/b><br>4605-G/C<\/p>","<p><b>genotype_SNP<\/b><br>476-C/T<\/p>","<p><b>genotype_SNP<\/b><br>426-T/C<\/p>","<p><b>genotype_SNP<\/b><br>4967-G/A<\/p>","<p><b>genotype_SNP<\/b><br>4888-T/C<\/p>","<p><b>genotype_SNP<\/b><br>5084-C/G<\/p>","<p><b>genotype_SNP<\/b><br>4951-A/C<\/p>","<p><b>genotype_SNP<\/b><br>937-T/C<\/p>","<p><b>genotype_SNP<\/b><br>970-G/A<\/p>","<p><b>genotype_SNP<\/b><br>698-C/T<\/p>","<p><b>genotype_SNP<\/b><br>4966-G/A<\/p>","<p><b>genotype_SNP<\/b><br>4733-G/A<\/p>","<p><b>genotype_SNP<\/b><br>4245-C/T<\/p>","<p><b>genotype_SNP<\/b><br>661-G/A<\/p>","<p><b>genotype_SNP<\/b><br>4551-A/C<\/p>","<p><b>genotype_SNP<\/b><br>4910-G/C<\/p>","<p><b>genotype_SNP<\/b><br>4963-G/A<\/p>","<p><b>genotype_SNP<\/b><br>4989-C/T<\/p>","<p><b>genotype_SNP<\/b><br>586-A/C<\/p>","<p><b>genotype_SNP<\/b><br>663-C/T<\/p>","<p><b>genotype_SNP<\/b><br>4754-A/T<\/p>","<p><b>genotype_SNP<\/b><br>566-G/C<\/p>","<p><b>genotype_SNP<\/b><br>4740-C/T<\/p>","<p><b>genotype_SNP<\/b><br>4816-T/A<\/p>","<p><b>genotype_SNP<\/b><br>4755-T/C<\/p>","<p><b>genotype_SNP<\/b><br>4942-C/T<\/p>","<p><b>genotype_SNP<\/b><br>4774-A/G<\/p>","<p><b>genotype_SNP<\/b><br>568-G/C<\/p>","<p><b>genotype_SNP<\/b><br>4933-G/A<\/p>","<p><b>genotype_SNP<\/b><br>588-T/G<\/p>","<p><b>genotype_SNP<\/b><br>4735-A/G<\/p>","<p><b>genotype_SNP<\/b><br>4943-A/G<\/p>","<p><b>genotype_SNP<\/b><br>690-A/C<\/p>","<p><b>genotype_SNP<\/b><br>600-T/C<\/p>","<p><b>genotype_SNP<\/b><br>647-C/T<\/p>","<p><b>genotype_SNP<\/b><br>4785-C/A<\/p>","<p><b>genotype_SNP<\/b><br>386-A/G<\/p>","<p><b>genotype_SNP<\/b><br>4807-G/A<\/p>","<p><b>genotype_SNP<\/b><br>650-A/C<\/p>","<p><b>genotype_SNP<\/b><br>826-A/G<\/p>","<p><b>genotype_SNP<\/b><br>4923-A/C<\/p>","<p><b>genotype_SNP<\/b><br>4783-G/C<\/p>","<p><b>genotype_SNP<\/b><br>940-T/G<\/p>","<p><b>genotype_SNP<\/b><br>4862-G/T<\/p>","<p><b>genotype_SNP<\/b><br>4846-A/G<\/p>","<p><b>genotype_SNP<\/b><br>954-G/C<\/p>","<p><b>genotype_SNP<\/b><br>595-A/G<\/p>","<p><b>genotype_SNP<\/b><br>4858-A/T<\/p>","<p><b>genotype_SNP<\/b><br>184-A/G<\/p>","<p><b>lipidomics<\/b><br>AC<\/p>","<p><b>lipidomics<\/b><br>PA<\/p>","<p><b>lipidomics<\/b><br>NSer<\/p>","<p><b>lipidomics<\/b><br>Cer<\/p>","<p><b>lipidomics<\/b><br>dhCer<\/p>","<p><b>lipidomics<\/b><br>LPI<\/p>","<p><b>lipidomics<\/b><br>LPEp<\/p>","<p><b>lipidomics<\/b><br>LPS<\/p>","<p><b>lipidomics<\/b><br>Sulf<\/p>","<p><b>lipidomics<\/b><br>PEp<\/p>","<p><b>lipidomics<\/b><br>PE<\/p>","<p><b>lipidomics<\/b><br>LPE<\/p>","<p><b>lipidomics<\/b><br>PS<\/p>","<p><b>lipidomics<\/b><br>BMP<\/p>","<p><b>lipidomics<\/b><br>DG<\/p>","<p><b>lipidomics<\/b><br>NAPS<\/p>","<p><b>lipidomics<\/b><br>MG<\/p>","<p><b>lipidomics<\/b><br>PG<\/p>","<p><b>lipidomics<\/b><br>MhCer<\/p>","<p><b>lipidomics<\/b><br>LPCe<\/p>","<p><b>lipidomics<\/b><br>AcylPG<\/p>","<p><b>lipidomics<\/b><br>LPC<\/p>","<p><b>lipidomics<\/b><br>PC<\/p>","<p><b>lipidomics<\/b><br>TG<\/p>","<p><b>lipidomics<\/b><br>PCe<\/p>","<p><b>lipidomics<\/b><br>NAPE<\/p>","<p><b>proteomics<\/b><br>Apolipoprotein B-100<\/p>","<p><b>proteomics<\/b><br>Complement C1s subcomponent<\/p>","<p><b>proteomics<\/b><br>Apolipoprotein(a)<\/p>","<p><b>proteomics<\/b><br>Apolipoprotein D<\/p>","<p><b>proteomics<\/b><br>Complement factor H<\/p>","<p><b>proteomics<\/b><br>Complement C1q subcomponent subunit B<\/p>","<p><b>proteomics<\/b><br>Apolipoprotein E<\/p>","<p><b>proteomics<\/b><br>Serum amyloid A-4 protein<\/p>","<p><b>proteomics<\/b><br>Immunoglobulin heavy constant mu<\/p>","<p><b>proteomics<\/b><br>Complement C1q subcomponent subunit C<\/p>","<p><b>proteomics<\/b><br>Immunoglobulin heavy variable 1-2<\/p>","<p><b>genotype_SNP<\/b><br>412-C/T<\/p>","<p><b>genotype_SNP<\/b><br>166-G/A<\/p>","<p><b>genotype_SNP<\/b><br>387-G/C<\/p>","<p><b>genotype_SNP<\/b><br>727-T/C<\/p>","<p><b>genotype_SNP<\/b><br>402-G/A<\/p>","<p><b>genotype_SNP<\/b><br>4527-A/T<\/p>","<p><b>genotype_SNP<\/b><br>4653-A/T<\/p>","<p><b>genotype_SNP<\/b><br>723-A/G<\/p>","<p><b>genotype_SNP<\/b><br>4541-G/A<\/p>","<p><b>proteomics<\/b><br>Complement_C1s_subcomponent<\/p>","<p><b>proteomics<\/b><br>Apolipoprotein_B-100<\/p>","<p><b>proteomics<\/b><br>Complement_factor_B<\/p>","<p><b>proteomics<\/b><br>Apolipoprotein_E<\/p>","<p><b>proteomics<\/b><br>Properdin<\/p>","<p><b>proteomics<\/b><br>Serum_paraoxonase_arylesterase_1<\/p>","<p><b>proteomics<\/b><br>Complement_C3<\/p>","<p><b>proteomics<\/b><br>Apolipoprotein_C-I<\/p>","<p><b>proteomics<\/b><br>Transthyretin<\/p>","<p><b>proteomics<\/b><br>Serum_amyloid_A-4_protein<\/p>","<p><b>proteomics<\/b><br>Clusterin<\/p>","<p><b>proteomics<\/b><br>Sulfhydryl_oxidase_1<\/p>","<p><b>proteomics<\/b><br>Prenylcysteine_oxidase_1<\/p>","<p><b>proteomics<\/b><br>Apolipoprotein_C-IV<\/p>","<p><b>proteomics<\/b><br>Complement_factor_H<\/p>","<p><b>proteomics<\/b><br>Apolipoprotein_A-II<\/p>","<p><b>proteomics<\/b><br>Immunoglobulin_heavy_variable_1-2<\/p>","<p><b>proteomics<\/b><br>Inter-alpha-trypsin_inhibitor_heavy_chain_H4<\/p>","<p><b>proteomics<\/b><br>Heparin_cofactor_2<\/p>","<p><b>genotype_VNTR<\/b><br>VNTR_estimate<\/p>","<p><b>genotype_SNP<\/b><br>5055-A/G<\/p>","<p><b>genotype_SNP<\/b><br>5052-A/T<\/p>","<p><b>genotype_SNP<\/b><br>5069-C/T<\/p>","<p><b>genotype_SNP<\/b><br>5037-T/G<\/p>","<p><b>genotype_SNP<\/b><br>4599-T/A<\/p>","<p><b>genotype_SNP<\/b><br>640-C/T<\/p>","<p><b>genotype_SNP<\/b><br>4604-G/C<\/p>","<p><b>genotype_SNP<\/b><br>2470-C/T<\/p>","<p><b>genotype_SNP<\/b><br>4734-G/T<\/p>","<p><b>genotype_SNP<\/b><br>612-G/C<\/p>","<p><b>genotype_SNP<\/b><br>4577-G/A<\/p>","<p><b>genotype_SNP<\/b><br>4503-A/G<\/p>","<p><b>genotype_SNP<\/b><br>1028-C/G<\/p>","<p><b>genotype_SNP<\/b><br>421-T/A<\/p>","<p><b>genotype_SNP<\/b><br>681-A/G<\/p>","<p><b>genotype_SNP<\/b><br>751-G/C<\/p>","<p><b>genotype_SNP<\/b><br>520-G/T<\/p>","<p><b>genotype_SNP<\/b><br>4612-T/A<\/p>","<p><b>genotype_SNP<\/b><br>5040-G/C<\/p>","<p><b>genotype_SNP<\/b><br>4889-A/T<\/p>","<p><b>genotype_SNP<\/b><br>494-A/C<\/p>","<p><b>genotype_SNP<\/b><br>387-G/T<\/p>","<p><b>genotype_SNP<\/b><br>199-C/T<\/p>","<p><b>genotype_SNP<\/b><br>728-A/G<\/p>","<p><b>genotype_SNP<\/b><br>653-C/A<\/p>","<p><b>genotype_SNP<\/b><br>1046-G/A<\/p>","<p><b>genotype_SNP<\/b><br>4644-G/A<\/p>","<p><b>genotype_SNP<\/b><br>4696-T/A<\/p>","<p><b>genotype_SNP<\/b><br>591-G/T<\/p>","<p><b>genotype_SNP<\/b><br>707-C/G<\/p>","<p><b>genotype_SNP<\/b><br>959-C/G<\/p>","<p><b>genotype_SNP<\/b><br>4976-G/T<\/p>","<p><b>genotype_SNP<\/b><br>606-G/T<\/p>","<p><b>genotype_SNP<\/b><br>578-C/T<\/p>","<p><b>genotype_SNP<\/b><br>546-G/T<\/p>","<p><b>genotype_SNP<\/b><br>1005-G/A<\/p>","<p><b>genotype_SNP<\/b><br>917-C/T<\/p>","<p><b>genotype_SNP<\/b><br>4473-C/T<\/p>","<p><b>genotype_SNP<\/b><br>1144-A/G<\/p>","<p><b>genotype_SNP<\/b><br>781-G/T<\/p>","<p><b>genotype_SNP<\/b><br>260-G/A<\/p>","<p><b>genotype_SNP<\/b><br>748-T/G<\/p>","<p><b>genotype_SNP<\/b><br>5004-C/A<\/p>","<p><b>genotype_SNP<\/b><br>602-G/A<\/p>","<p><b>genotype_SNP<\/b><br>4572-G/A<\/p>","<p><b>genotype_SNP<\/b><br>464-A/G<\/p>","<p><b>genotype_SNP<\/b><br>5092-G/T<\/p>","<p><b>genotype_SNP<\/b><br>4671-C/A<\/p>","<p><b>genotype_SNP<\/b><br>4674-A/C<\/p>","<p><b>genotype_SNP<\/b><br>4704-G/C<\/p>","<p><b>genotype_SNP<\/b><br>4685-T/C<\/p>","<p><b>genotype_SNP<\/b><br>1053-G/A<\/p>","<p><b>genotype_SNP<\/b><br>1044-A/G<\/p>","<p><b>genotype_SNP<\/b><br>4672-A/G<\/p>","<p><b>genotype_SNP<\/b><br>4441-A/G<\/p>","<p><b>genotype_SNP<\/b><br>4919-G/A<\/p>","<p><b>genotype_SNP<\/b><br>651-C/G<\/p>","<p><b>genotype_SNP<\/b><br>4683-T/C<\/p>","<p><b>genotype_SNP<\/b><br>4842-C/T<\/p>","<p><b>genotype_SNP<\/b><br>5078-G/A<\/p>","<p><b>genotype_SNP<\/b><br>4266-G/T<\/p>","<p><b>genotype_SNP<\/b><br>950-A/G<\/p>","<p><b>genotype_SNP<\/b><br>372-C/T<\/p>","<p><b>genotype_SNP<\/b><br>4260-A/C<\/p>","<p><b>genotype_SNP<\/b><br>1323-T/C<\/p>","<p><b>genotype_SNP<\/b><br>4755-T/G<\/p>","<p><b>genotype_SNP<\/b><br>4228-G/A<\/p>","<p><b>genotype_SNP<\/b><br>5013-G/T<\/p>","<p><b>genotype_SNP<\/b><br>4755-T/A<\/p>","<p><b>genotype_SNP<\/b><br>690-A/G<\/p>","<p><b>genotype_SNP<\/b><br>4583-T/A<\/p>","<p><b>genotype_SNP<\/b><br>4598-T/C<\/p>","<p><b>genotype_SNP<\/b><br>4623-T/C<\/p>","<p><b>genotype_SNP<\/b><br>717-G/A<\/p>","<p><b>genotype_SNP<\/b><br>824-C/A<\/p>","<p><b>genotype_SNP<\/b><br>907-A/G<\/p>","<p><b>genotype_SNP<\/b><br>835-T/C<\/p>","<p><b>genotype_SNP<\/b><br>4738-T/G<\/p>","<p><b>genotype_SNP<\/b><br>746-T/C<\/p>","<p><b>genotype_SNP<\/b><br>4716-G/T<\/p>","<p><b>genotype_SNP<\/b><br>1091-C/A<\/p>","<p><b>genotype_SNP<\/b><br>1010-G/A<\/p>","<p><b>genotype_SNP<\/b><br>4461-G/C<\/p>","<p><b>genotype_SNP<\/b><br>4462-T/C<\/p>","<p><b>genotype_SNP<\/b><br>572-T/G<\/p>","<p><b>genotype_SNP<\/b><br>185-A/G<\/p>","<p><b>genotype_SNP<\/b><br>1205-T/A<\/p>","<p><b>genotype_SNP<\/b><br>150-G/T<\/p>","<p><b>genotype_SNP<\/b><br>1070-A/G<\/p>","<p><b>genotype_SNP<\/b><br>1269-A/G<\/p>","<p><b>genotype_SNP<\/b><br>1217-A/T<\/p>","<p><b>genotype_SNP<\/b><br>708-G/A<\/p>","<p><b>genotype_SNP<\/b><br>357-C/T<\/p>","<p><b>genotype_SNP<\/b><br>4639-A/T<\/p>","<p><b>genotype_SNP<\/b><br>4640-G/C<\/p>","<p><b>genotype_SNP<\/b><br>4426-G/T<\/p>","<p><b>genotype_SNP<\/b><br>410-T/A<\/p>","<p><b>genotype_SNP<\/b><br>406-C/T<\/p>","<p><b>genotype_SNP<\/b><br>837-T/C<\/p>","<p><b>genotype_SNP<\/b><br>398-G/T<\/p>","<p><b>genotype_SNP<\/b><br>4436-A/G<\/p>","<p><b>genotype_SNP<\/b><br>4433-C/T<\/p>","<p><b>genotype_SNP<\/b><br>4355-T/C<\/p>","<p><b>genotype_SNP<\/b><br>411-G/T<\/p>","<p><b>genotype_SNP<\/b><br>4893-C/A<\/p>","<p><b>genotype_SNP<\/b><br>314-C/T<\/p>","<p><b>genotype_SNP<\/b><br>393-G/A<\/p>","<p><b>genotype_SNP<\/b><br>820-T/G<\/p>","<p><b>genotype_SNP<\/b><br>818-G/C<\/p>","<p><b>genotype_SNP<\/b><br>4279-C/A<\/p>","<p><b>genotype_SNP<\/b><br>342-G/T<\/p>","<p><b>genotype_SNP<\/b><br>353-G/C<\/p>","<p><b>genotype_SNP<\/b><br>4788-C/G<\/p>","<p><b>phenotype<\/b><br>lpa<\/p>","<p><b>phenotype<\/b><br>wIS<\/p>","<p><b>phenotype<\/b><br>Glucose (mg/dL)<\/p>","<p><b>phenotype<\/b><br>Hypertension<\/p>","<p><b>lipidomics<\/b><br>FC<\/p>","<p><b>lipidomics<\/b><br>CE<\/p>","<p><b>lipidomics<\/b><br>LacCer<\/p>","<p><b>lipidomics<\/b><br>PI<\/p>","<p><b>lipidomics<\/b><br>GM3<\/p>","<p><b>proteomics<\/b><br>Immunoglobulin heavy variable 1-3<\/p>","<p><b>proteomics<\/b><br>Complement C4-A<\/p>","<p><b>proteomics<\/b><br>Alpha-1-acid glycoprotein 1<\/p>","<p><b>proteomics<\/b><br>Albumin<\/p>","<p><b>proteomics<\/b><br>Immunoglobulin heavy variable 4-34<\/p>","<p><b>proteomics<\/b><br>Keratin, type I cytoskeletal 9<\/p>","<p><b>proteomics<\/b><br>Ceruloplasmin<\/p>","<p><b>proteomics<\/b><br>Complement component C7<\/p>","<p><b>proteomics<\/b><br>Immunoglobulin heavy variable 3-15<\/p>","<p><b>proteomics<\/b><br>Complement C3<\/p>","<p><b>proteomics<\/b><br>Complement C1q subcomponent subunit A<\/p>","<p><b>proteomics<\/b><br>Vitamin D-binding protein<\/p>","<p><b>proteomics<\/b><br>Probable non-functional immunoglobulin kappa variable 3-7<\/p>","<p><b>proteomics<\/b><br>Immunoglobulin heavy variable 5-51<\/p>","<p><b>proteomics<\/b><br>Complement C4-B<\/p>","<p><b>proteomics<\/b><br>Kallistatin<\/p>","<p><b>proteomics<\/b><br>Platelet_basic_protein<\/p>","<p><b>proteomics<\/b><br>Basement_membrane-specific_heparan_sulfate_proteoglycan_core_protein<\/p>","<p><b>proteomics<\/b><br>Complement_C1q_subcomponent_subunit_B<\/p>","<p><b>proteomics<\/b><br>Plasminogen<\/p>","<p><b>proteomics<\/b><br>Immunoglobulin_heavy_variable_5-51<\/p>","<p><b>proteomics<\/b><br>Complement_C1q_subcomponent_subunit_A<\/p>","<p><b>proteomics<\/b><br>Immunoglobulin_heavy_variable_1-24<\/p>","<p><b>proteomics<\/b><br>Isoform_1_of_Fibronectin<\/p>","<p><b>proteomics<\/b><br>Keratin__type_I_cytoskeletal_9<\/p>","<p><b>proteomics<\/b><br>Isoform_1_of_Immunoglobulin_heavy_constant_alpha_1<\/p>","<p><b>proteomics<\/b><br>Complement_C1q_subcomponent_subunit_C<\/p>","<p><b>proteomics<\/b><br>Immunoglobulin_heavy_constant_alpha_2<\/p>","<p><b>proteomics<\/b><br>Alpha-2-HS-glycoprotein<\/p>","<p><b>proteomics<\/b><br>Immunoglobulin_heavy_variable_1-46<\/p>","<p><b>proteomics<\/b><br>Keratin__type_I_cytoskeletal_13<\/p>","<p><b>proteomics<\/b><br>Alpha-1-antitrypsin<\/p>","<p><b>proteomics<\/b><br>BPI_fold-containing_family_B_member_1<\/p>","<p><b>proteomics<\/b><br>Extracellular_matrix_protein_1<\/p>","<p><b>proteomics<\/b><br>N-acetylmuramoyl-L-alanine_amidase<\/p>","<p><b>proteomics<\/b><br>Keratin__type_II_cytoskeletal_1b<\/p>","<p><b>proteomics<\/b><br>Intelectin-1<\/p>","<p><b>proteomics<\/b><br>Immunoglobulin_kappa_variable_1-8<\/p>","<p><b>proteomics<\/b><br>Immunoglobulin_heavy_variable_3-72<\/p>","<p><b>proteomics<\/b><br>Immunoglobulin_heavy_variable_3-7<\/p>","<p><b>proteomics<\/b><br>Complement_component_C8_beta_chain<\/p>","<p><b>proteomics<\/b><br>Tsukushi<\/p>","<p><b>proteomics<\/b><br>Cathelicidin_antimicrobial_peptide<\/p>","<p><b>proteomics<\/b><br>Lipopolysaccharide-binding_protein<\/p>","<p><b>proteomics<\/b><br>Complement_C1r_subcomponent<\/p>","<p><b>proteomics<\/b><br>Immunoglobulin_heavy_variable_1-18<\/p>","<p><b>proteomics<\/b><br>Fibrinogen_alpha_chain<\/p>","<p><b>proteomics<\/b><br>Complement_C4-A<\/p>","<p><b>proteomics<\/b><br>Complement_component_C8_gamma_chain<\/p>","<p><b>proteomics<\/b><br>Alpha-1B-glycoprotein<\/p>","<p><b>proteomics<\/b><br>Apolipoprotein_A-I<\/p>","<p><b>proteomics<\/b><br>Heat_shock_70_kDa_protein_1B<\/p>","<p><b>proteomics<\/b><br>Fructose-bisphosphate_aldolase_A<\/p>","<p><b>proteomics<\/b><br>Inter-alpha-trypsin_inhibitor_heavy_chain_H1<\/p>","<p><b>proteomics<\/b><br>L-lactate_dehydrogenase_A_chain<\/p>","<p><b>proteomics<\/b><br>Serpin_B4<\/p>","<p><b>proteomics<\/b><br>Tubulin_alpha-1A_chain<\/p>","<p><b>proteomics<\/b><br>Serpin_B3<\/p>","<p><b>proteomics<\/b><br>Protein-glutamine_gamma-glutamyltransferase_K<\/p>","<p><b>proteomics<\/b><br>Annexin_A2<\/p>","<p><b>proteomics<\/b><br>14-3-3_protein_sigma<\/p>","<p><b>proteomics<\/b><br>Cellular_retinoic_acid-binding_protein_2<\/p>","<p><b>proteomics<\/b><br>Annexin_A1<\/p>","<p><b>proteomics<\/b><br>Prelamin-A_C<\/p>","<p><b>proteomics<\/b><br>Endoplasmic_reticulum_chaperone_BiP<\/p>","<p><b>proteomics<\/b><br>Heat_shock_protein_HSP_90-beta<\/p>","<p><b>proteomics<\/b><br>Histone_H4<\/p>","<p><b>proteomics<\/b><br>Elongation_factor_1-alpha_2<\/p>","<p><b>proteomics<\/b><br>Alpha-enolase<\/p>","<p><b>proteomics<\/b><br>Triosephosphate_isomerase<\/p>","<p><b>proteomics<\/b><br>Pyruvate_kinase_PKM<\/p>","<p><b>proteomics<\/b><br>ATP_synthase_subunit_alpha__mitochondrial<\/p>","<p><b>proteomics<\/b><br>Heat_shock_protein_beta-1<\/p>","<p><b>proteomics<\/b><br>Vimentin<\/p>","<p><b>proteomics<\/b><br>Galectin-7<\/p>","<p><b>proteomics<\/b><br>Hemopexin<\/p>","<p><b>proteomics<\/b><br>Zinc-alpha-2-glycoprotein<\/p>","<p><b>proteomics<\/b><br>Protein_S100-A9<\/p>","<p><b>proteomics<\/b><br>Glyceraldehyde-3-phosphate_dehydrogenase<\/p>","<p><b>proteomics<\/b><br>14-3-3_protein_zeta_delta<\/p>","<p><b>proteomics<\/b><br>Protein_S100-A8<\/p>","<p><b>proteomics<\/b><br>Desmoplakin<\/p>","<p><b>proteomics<\/b><br>Serum_amyloid_P-component<\/p>","<p><b>proteomics<\/b><br>Cystatin-A<\/p>","<p><b>proteomics<\/b><br>Sex_hormone-binding_globulin<\/p>","<p><b>proteomics<\/b><br>Sushi__von_Willebrand_factor_type_A__EGF_and_pentraxin_domain-containing_protein_1<\/p>","<p><b>proteomics<\/b><br>Keratin__type_I_cytoskeletal_17<\/p>","<p><b>proteomics<\/b><br>Plasma_protease_C1_inhibitor<\/p>","<p><b>proteomics<\/b><br>Haptoglobin<\/p>","<p><b>proteomics<\/b><br>Junction_plakoglobin<\/p>","<p><b>proteomics<\/b><br>Vitamin_D-binding_protein<\/p>","<p><b>proteomics<\/b><br>Keratin__type_II_cytoskeletal_5<\/p>","<p><b>proteomics<\/b><br>Angiotensinogen<\/p>","<p><b>proteomics<\/b><br>Protein_S100-A7<\/p>","<p><b>proteomics<\/b><br>Monocyte_differentiation_antigen_CD14<\/p>"]},"edges":{"from":["4894-G/A","620-A/G","621-T/A","4922-C/G","4970-T/A","649-T/C","4908-A/G","943-T/A","632-A/G","654-C/T","4605-G/C","476-C/T","426-T/C","4967-G/A","4888-T/C","5084-C/G","4951-A/C","937-T/C","970-G/A","698-C/T","4966-G/A","4733-G/A","4245-C/T","661-G/A","4551-A/C","4910-G/C","4963-G/A","4989-C/T","586-A/C","620-A/G","621-T/A","663-C/T","4970-T/A","4754-A/T","4894-G/A","4967-G/A","654-C/T","698-C/T","566-G/C","4740-C/T","586-A/C","4816-T/A","4755-T/C","4942-C/T","4774-A/G","632-A/G","4966-G/A","568-G/C","4888-T/C","4908-A/G","4933-G/A","661-G/A","4963-G/A","588-T/G","649-T/C","4951-A/C","4735-A/G","4943-A/G","937-T/C","690-A/C","600-T/C","647-C/T","943-T/A","4785-C/A","386-A/G","4922-C/G","4989-C/T","4807-G/A","650-A/C","826-A/G","4923-A/C","4783-G/C","940-T/G","4862-G/T","4846-A/G","954-G/C","595-A/G","4910-G/C","4858-A/T","184-A/G","AC","PA","NSer","Cer","dhCer","LPI","AC","dhCer","LPEp","LPI","Cer","LPS","Sulf","PEp","PA","PE","LPEp","Sulf","LPE","LPS","PS","BMP","PS","NSer","BMP","LPE","DG","NAPS","PEp","PE","MG","NAPS","NSer","PA","AC","DG","PG","dhCer","Cer","PG","LPI","LPS","LPEp","Sulf","MhCer","BMP","PS","PEp","MhCer","MG","PE","LPE","DG","PG","LPCe","AcylPG","NAPS","LPC","MG","MhCer","PC","LPCe","AcylPG","TG","PCe","NAPE","TG","Apolipoprotein B-100","Apolipoprotein B-100","Apolipoprotein B-100","Apolipoprotein B-100","Complement C1s subcomponent","Apolipoprotein(a)","Complement C1s subcomponent","Apolipoprotein(a)","Apolipoprotein D","Apolipoprotein B-100","Complement factor H","Complement C1q subcomponent subunit B","Apolipoprotein E","Serum amyloid A-4 protein","Apolipoprotein E","Immunoglobulin heavy constant mu","Serum amyloid A-4 protein","Complement C1q subcomponent subunit C","Apolipoprotein E","Apolipoprotein E","Immunoglobulin heavy constant mu","Serum amyloid A-4 protein","Serum amyloid A-4 protein","Immunoglobulin heavy constant mu","Immunoglobulin heavy variable 1-2","Complement factor H","Immunoglobulin heavy constant mu","Apolipoprotein D","412-C/T","166-G/A","387-G/C","727-T/C","4785-C/A","402-G/A","4527-A/T","4653-A/T","723-A/G","4541-G/A","Complement_C1s_subcomponent","Complement_C1s_subcomponent","Complement_C1s_subcomponent","Complement_C1s_subcomponent","Complement_C1s_subcomponent","Complement_C1s_subcomponent","Complement_C1s_subcomponent","Complement_C1s_subcomponent","Apolipoprotein_B-100","Complement_C1s_subcomponent","Complement_C1s_subcomponent","Apolipoprotein_B-100","Complement_C1s_subcomponent","Complement_C1s_subcomponent","Complement_factor_B","Apolipoprotein_B-100","Complement_C1s_subcomponent","Apolipoprotein_B-100","Complement_C1s_subcomponent","Apolipoprotein_B-100","Complement_C1s_subcomponent","Apolipoprotein_B-100","Apolipoprotein_E","Apolipoprotein_E","Apolipoprotein_E","Apolipoprotein_E","Apolipoprotein_E","Apolipoprotein_B-100","Apolipoprotein_E","Apolipoprotein_E","Complement_factor_B","Complement_C1s_subcomponent","Complement_C1s_subcomponent","Complement_factor_B","Complement_factor_B","Complement_C1s_subcomponent","Apolipoprotein_B-100","Properdin","Apolipoprotein_E","Apolipoprotein_E","Apolipoprotein_E","Complement_C1s_subcomponent","Complement_factor_B","Complement_factor_B","Complement_factor_B","Apolipoprotein_B-100","Properdin","Complement_factor_B","Properdin","Apolipoprotein_E","Apolipoprotein_E","Apolipoprotein_B-100","Properdin","Complement_factor_B","Serum_paraoxonase_arylesterase_1","Complement_factor_B","Complement_factor_B","Properdin","Apolipoprotein_B-100","Apolipoprotein_B-100","Properdin","Apolipoprotein_E","Properdin","Complement_factor_B","Properdin","Complement_factor_B","Complement_factor_B","Properdin","Serum_paraoxonase_arylesterase_1","Complement_C3","Apolipoprotein(a)","Apolipoprotein(a)","Properdin","Complement_factor_B","Serum_paraoxonase_arylesterase_1","Serum_paraoxonase_arylesterase_1","Properdin","Apolipoprotein_B-100","Complement_factor_B","Apolipoprotein(a)","Properdin","Apolipoprotein_B-100","Serum_paraoxonase_arylesterase_1","Apolipoprotein_C-I","Apolipoprotein(a)","Serum_paraoxonase_arylesterase_1","Apolipoprotein_C-I","Apolipoprotein(a)","Apolipoprotein_C-I","Complement_factor_B","Serum_paraoxonase_arylesterase_1","Serum_paraoxonase_arylesterase_1","Apolipoprotein_B-100","Apolipoprotein_E","Apolipoprotein(a)","Apolipoprotein(a)","Properdin","Apolipoprotein_E","Serum_paraoxonase_arylesterase_1","Complement_C1s_subcomponent","Complement_factor_B","Properdin","Serum_paraoxonase_arylesterase_1","Apolipoprotein_C-I","Transthyretin","Apolipoprotein_E","Apolipoprotein_B-100","Apolipoprotein_E","Apolipoprotein_C-I","Properdin","Complement_factor_B","Apolipoprotein_B-100","Complement_factor_B","Apolipoprotein_B-100","Serum_amyloid_A-4_protein","Serum_paraoxonase_arylesterase_1","Properdin","Clusterin","Clusterin","Apolipoprotein_E","Serum_paraoxonase_arylesterase_1","Serum_paraoxonase_arylesterase_1","Complement_factor_B","Serum_paraoxonase_arylesterase_1","Properdin","Complement_factor_B","Complement_C1s_subcomponent","Apolipoprotein_E","Apolipoprotein_C-I","Apolipoprotein_C-I","Serum_amyloid_A-4_protein","Transthyretin","Sulfhydryl_oxidase_1","Complement_factor_B","Apolipoprotein(a)","Apolipoprotein_C-I","Properdin","Apolipoprotein(a)","Apolipoprotein_C-I","Transthyretin","Clusterin","Prenylcysteine_oxidase_1","Apolipoprotein_C-IV","Apolipoprotein_C-I","Transthyretin","Serum_amyloid_A-4_protein","Apolipoprotein(a)","Clusterin","Transthyretin","Apolipoprotein(a)","Properdin","Complement_factor_B","Complement_C3","Clusterin","Clusterin","Serum_amyloid_A-4_protein","Serum_amyloid_A-4_protein","Clusterin","Clusterin","Transthyretin","Clusterin","Serum_amyloid_A-4_protein","Transthyretin","Serum_amyloid_A-4_protein","Transthyretin","Apolipoprotein_C-I","Transthyretin","Apolipoprotein_E","Transthyretin","Properdin","Serum_paraoxonase_arylesterase_1","Apolipoprotein_C-I","Apolipoprotein_B-100","Clusterin","Complement_factor_H","Sulfhydryl_oxidase_1","Complement_factor_H","Serum_amyloid_A-4_protein","Transthyretin","Apolipoprotein_A-II","Apolipoprotein(a)","Complement_factor_H","Clusterin","Transthyretin","Complement_factor_B","Sulfhydryl_oxidase_1","Properdin","Serum_amyloid_A-4_protein","Complement_C3","Apolipoprotein_C-I","Serum_paraoxonase_arylesterase_1","Prenylcysteine_oxidase_1","Apolipoprotein_C-IV","Complement_factor_H","Immunoglobulin_heavy_variable_1-2","Apolipoprotein_C-IV","Complement_factor_H","Serum_paraoxonase_arylesterase_1","Serum_amyloid_A-4_protein","Apolipoprotein_C-I","Immunoglobulin_heavy_variable_1-2","Inter-alpha-trypsin_inhibitor_heavy_chain_H4","Immunoglobulin_heavy_variable_1-2","Apolipoprotein(a)","Complement_factor_B","Sulfhydryl_oxidase_1","Transthyretin","Complement_factor_H","Transthyretin","Apolipoprotein_C-IV","Complement_factor_H","Clusterin","Transthyretin","Complement_factor_B","Serum_paraoxonase_arylesterase_1","Immunoglobulin_heavy_variable_1-2","Transthyretin","Properdin","Transthyretin","Apolipoprotein(a)","Complement_C3","Heparin_cofactor_2","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","4970-T/A","4970-T/A","4970-T/A","4970-T/A","4967-G/A","4970-T/A","4970-T/A","4967-G/A","4970-T/A","4970-T/A","4970-T/A","4970-T/A","4967-G/A","4970-T/A","4967-G/A","4966-G/A","4967-G/A","4970-T/A","4970-T/A","4970-T/A","4967-G/A","4966-G/A","4966-G/A","4970-T/A","4967-G/A","4966-G/A","4970-T/A","4966-G/A","4966-G/A","4951-A/C","4966-G/A","4967-G/A","4966-G/A","4951-A/C","4951-A/C","4908-A/G","4951-A/C","4951-A/C","4967-G/A","4970-T/A","4908-A/G","4951-A/C","4970-T/A","4967-G/A","4967-G/A","4951-A/C","4967-G/A","4966-G/A","4951-A/C","4951-A/C","4967-G/A","4966-G/A","4967-G/A","4966-G/A","4967-G/A","4967-G/A","4966-G/A","4966-G/A","4966-G/A","4951-A/C","4966-G/A","4967-G/A","4908-A/G","4966-G/A","4908-A/G","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","VNTR_estimate","5055-A/G","5052-A/T","5069-C/T","5037-T/G","4599-T/A","640-C/T","640-C/T","4604-G/C","2470-C/T","4734-G/T","612-G/C","4577-G/A","4503-A/G","1028-C/G","421-T/A","681-A/G","4604-G/C","2470-C/T","4604-G/C","2470-C/T","751-G/C","751-G/C","520-G/T","4612-T/A","4612-T/A","520-G/T","640-C/T","5040-G/C","4889-A/T","494-A/C","520-G/T","751-G/C","4612-T/A","2470-C/T","4604-G/C","494-A/C","5040-G/C","4889-A/T","387-G/T","199-C/T","728-A/G","653-C/A","1046-G/A","4644-G/A","4696-T/A","591-G/T","707-C/G","959-C/G","4976-G/T","5037-T/G","4599-T/A","5055-A/G","5052-A/T","5069-C/T","5052-A/T","5037-T/G","5055-A/G","5069-C/T","4599-T/A","606-G/T","578-C/T","546-G/T","1005-G/A","917-C/T","4473-C/T","640-C/T","1144-A/G","781-G/T","260-G/A","748-T/G","5004-C/A","199-C/T","387-G/T","602-G/A","4572-G/A","464-A/G","5092-G/T","4671-C/A","4674-A/C","4704-G/C","4685-T/C","1053-G/A","1044-A/G","4503-A/G","681-A/G","1028-C/G","612-G/C","421-T/A","4734-G/T","4577-G/A","520-G/T","4976-G/T","4672-A/G","4612-T/A","751-G/C","520-G/T","4612-T/A","751-G/C","4441-A/G","4919-G/A","4441-A/G","4919-G/A","748-T/G","5004-C/A","199-C/T","387-G/T","651-C/G","4683-T/C","4842-C/T","5078-G/A","5052-A/T","4599-T/A","5037-T/G","5069-C/T","5055-A/G","4266-G/T","950-A/G","372-C/T","748-T/G","5004-C/A","4260-A/C","1323-T/C","4755-T/G","520-G/T","751-G/C","4612-T/A","4228-G/A","5013-G/T","4755-T/A","690-A/G","4755-T/A","4583-T/A","4755-T/G","5004-C/A","748-T/G","4734-G/T","4503-A/G","612-G/C","1028-C/G","4577-G/A","681-A/G","421-T/A","4598-T/C","4623-T/C","199-C/T","387-G/T","387-G/T","199-C/T","717-G/A","824-C/A","826-A/G","5037-T/G","5069-C/T","5052-A/T","4599-T/A","5055-A/G","907-A/G","640-C/T","835-T/C","835-T/C","4734-G/T","1028-C/G","612-G/C","4503-A/G","4577-G/A","421-T/A","681-A/G","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","835-T/C","4738-T/G","746-T/C","4716-G/T","421-T/A","4577-G/A","681-A/G","612-G/C","4503-A/G","4734-G/T","1028-C/G","4653-A/T","199-C/T","387-G/T","835-T/C","835-T/C","835-T/C","1091-C/A","835-T/C","4583-T/A","835-T/C","1010-G/A","4755-T/G","4461-G/C","4462-T/C","572-T/G","185-A/G","4228-G/A","1205-T/A","4260-A/C","1323-T/C","4842-C/T","5078-G/A","4683-T/C","651-C/G","4266-G/T","150-G/T","1070-A/G","1269-A/G","4922-C/G","1269-A/G","1144-A/G","260-G/A","781-G/T","1217-A/T","4976-G/T","835-T/C","708-G/A","708-G/A","717-G/A","824-C/A","357-C/T","4734-G/T","612-G/C","1028-C/G","4503-A/G","421-T/A","4577-G/A","681-A/G","4639-A/T","4640-G/C","4527-A/T","835-T/C","4426-G/T","5013-G/T","835-T/C","421-T/A","4503-A/G","681-A/G","612-G/C","4734-G/T","1028-C/G","4577-G/A","835-T/C","184-A/G","410-T/A","406-C/T","781-G/T","837-T/C","398-G/T","4436-A/G","4433-C/T","4355-T/C","1144-A/G","411-G/T","4893-C/A","260-G/A","314-C/T","393-G/A","820-T/G","818-G/C","835-T/C","4279-C/A","342-G/T","353-G/C","708-G/A","387-G/C","4788-C/G","4755-T/A"],"to":["lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","lpa","lpa","lpa","lpa","lpa","lpa","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","wIS","lpa","lpa","wIS","lpa","lpa","lpa","wIS","wIS","wIS","lpa","lpa","lpa","lpa","lpa","wIS","wIS","lpa","lpa","lpa","wIS","wIS","lpa","lpa","lpa","lpa","lpa","lpa","lpa","wIS","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","wIS","wIS","lpa","wIS","lpa","lpa","wIS","lpa","lpa","wIS","wIS","wIS","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","wIS","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","lpa","Glucose (mg/dL)","Hypertension","Hypertension","Hypertension","Hypertension","Hypertension","Hypertension","Hypertension","Hypertension","Hypertension","PA","AC","Cer","dhCer","LPS","PS","Sulf","NAPS","NSer","NSer","LPI","AC","PEp","LPEp","FC","PA","BMP","LPI","PE","dhCer","MhCer","Cer","dhCer","AC","PA","Cer","LPS","LPS","NSer","PS","TG","LPE","DG","MhCer","PG","PG","PS","PS","LPI","Sulf","NAPS","MG","PEp","Cer","dhCer","Sulf","LPS","MG","PEp","PEp","LPEp","LPEp","dhCer","PA","PS","PS","LPS","Cer","PEp","NAPS","NAPS","PE","PE","PE","LPI","LPI","NAPS","PG","NAPS","FC","LPI","AC","AC","AC","AC","dhCer","PA","PG","Sulf","NSer","MG","PE","PEp","PS","PA","LPS","AC","dhCer","dhCer","CE","PE","Cer","LPE","LPE","Cer","LPS","MhCer","MhCer","Sulf","AcylPG","AcylPG","Sulf","PA","Cer","MhCer","MG","MG","PG","LPS","DG","DG","DG","BMP","BMP","PS","MhCer","TG","PS","NAPS","DG","DG","BMP","LPEp","LPEp","NSer","NSer","TG","BMP","PE","PA","PA","PA","LacCer","LPE","LPEp","LPEp","LPEp","Sulf","Sulf","Sulf","Sulf","Sulf","PS","PEp","Cer","NSer","PS","dhCer","dhCer","NAPS","BMP","PI","TG","NSer","AC","AC","dhCer","LPS","LPEp","PEp","Cer","Cer","LPS","LPS","PS","NSer","PE","AcylPG","AcylPG","AcylPG","AcylPG","MhCer","MhCer","PA","AC","PI","PA","NAPS","LPE","PS","PEp","DG","PEp","NAPE","NAPE","LPE","LPE","Sulf","PEp","LPE","LPE","LPS","LPS","LPEp","Sulf","NSer","LPI","LPI","LPI","LPI","LPI","NSer","LPE","LPE","GM3","GM3","NSer","NSer","AC","AC","LPS","PE","LacCer","LacCer","NSer","NSer","TG","FC","FC","BMP","LPS","FC","LPI","dhCer","NSer","LPS","AC","Cer","LPEp","PEp","PA","Sulf","PS","PE","LPE","MG","PG","NAPS","BMP","DG","MhCer","LPC","LPEp","LPS","Sulf","Cer","LPS","LPE","dhCer","LPEp","PEp","BMP","AC","MhCer","Sulf","MG","AC","LPS","Cer","PE","PA","AcylPG","dhCer","LPEp","AC","PS","LPE","LPE","LPI","Cer","Sulf","LPE","dhCer","BMP","BMP","LPEp","AC","AC","Sulf","NSer","NSer","NSer","NSer","Cer","PG","PEp","LPI","LPI","PE","PEp","dhCer","LPS","MhCer","PE","AcylPG","AcylPG","PA","MG","MG","PA","LPI","PA","MhCer","PG","Cer","NSer","dhCer","Immunoglobulin heavy variable 1-3","Complement C4-A","Alpha-1-acid glycoprotein 1","Albumin","Apolipoprotein D","Immunoglobulin heavy variable 4-34","Keratin, type I cytoskeletal 9","Ceruloplasmin","Apolipoprotein B-100","Complement component C7","Immunoglobulin heavy variable 3-15","Complement C3","Complement C1q subcomponent subunit A","Vitamin D-binding protein","Probable non-functional immunoglobulin kappa variable 3-7","Immunoglobulin heavy variable 5-51","Properdin","Complement C4-B","Kallistatin","Kallistatin","Kallistatin","Kallistatin","Kallistatin","Platelet_basic_protein","Basement_membrane-specific_heparan_sulfate_proteoglycan_core_protein","Complement_C1q_subcomponent_subunit_B","Complement_C1q_subcomponent_subunit_B","Plasminogen","Plasminogen","Plasminogen","Plasminogen","Plasminogen","Plasminogen","Plasminogen","Immunoglobulin_heavy_variable_5-51","Immunoglobulin_heavy_variable_5-51","Complement_C1q_subcomponent_subunit_A","Complement_C1q_subcomponent_subunit_A","Immunoglobulin_heavy_variable_1-24","Immunoglobulin_heavy_variable_1-2","Immunoglobulin_heavy_variable_1-24","Immunoglobulin_heavy_variable_1-24","Immunoglobulin_heavy_variable_1-2","Immunoglobulin_heavy_variable_1-2","Isoform_1_of_Fibronectin","Keratin__type_I_cytoskeletal_9","Keratin__type_I_cytoskeletal_9","Keratin__type_I_cytoskeletal_9","Isoform_1_of_Immunoglobulin_heavy_constant_alpha_1","Isoform_1_of_Immunoglobulin_heavy_constant_alpha_1","Isoform_1_of_Immunoglobulin_heavy_constant_alpha_1","Complement_C1q_subcomponent_subunit_C","Complement_C1q_subcomponent_subunit_C","Immunoglobulin_heavy_constant_alpha_2","Immunoglobulin_heavy_constant_alpha_2","Immunoglobulin_heavy_constant_alpha_2","Serum_paraoxonase_arylesterase_1","Serum_paraoxonase_arylesterase_1","Alpha-2-HS-glycoprotein","Alpha-2-HS-glycoprotein","Alpha-2-HS-glycoprotein","Alpha-2-HS-glycoprotein","Alpha-2-HS-glycoprotein","Alpha-2-HS-glycoprotein","Alpha-2-HS-glycoprotein","Alpha-2-HS-glycoprotein","Immunoglobulin_heavy_variable_1-46","Keratin__type_I_cytoskeletal_13","Keratin__type_I_cytoskeletal_13","Keratin__type_I_cytoskeletal_13","Keratin__type_I_cytoskeletal_13","Keratin__type_I_cytoskeletal_13","Alpha-1-antitrypsin","Alpha-1-antitrypsin","Alpha-1-antitrypsin","Alpha-1-antitrypsin","Alpha-1-antitrypsin","Immunoglobulin_heavy_constant_alpha_2","Immunoglobulin_heavy_constant_alpha_2","Immunoglobulin_heavy_constant_alpha_2","BPI_fold-containing_family_B_member_1","BPI_fold-containing_family_B_member_1","BPI_fold-containing_family_B_member_1","Extracellular_matrix_protein_1","N-acetylmuramoyl-L-alanine_amidase","N-acetylmuramoyl-L-alanine_amidase","N-acetylmuramoyl-L-alanine_amidase","Serum_amyloid_A-4_protein","Serum_amyloid_A-4_protein","BPI_fold-containing_family_B_member_1","BPI_fold-containing_family_B_member_1","Keratin__type_II_cytoskeletal_1b","Keratin__type_II_cytoskeletal_1b","Keratin__type_II_cytoskeletal_1b","Keratin__type_II_cytoskeletal_1b","Plasminogen","Plasminogen","Plasminogen","Plasminogen","Keratin__type_I_cytoskeletal_13","Keratin__type_I_cytoskeletal_13","Albumin","Albumin","Albumin","Albumin","Albumin","Albumin","Albumin","Intelectin-1","Intelectin-1","Intelectin-1","Intelectin-1","Intelectin-1","Immunoglobulin_kappa_variable_1-8","Immunoglobulin_kappa_variable_1-8","Immunoglobulin_kappa_variable_1-8","Immunoglobulin_heavy_variable_3-72","Immunoglobulin_heavy_variable_3-7","Immunoglobulin_heavy_variable_3-7","Immunoglobulin_heavy_variable_3-72","Complement_component_C8_beta_chain","Complement_component_C8_beta_chain","Tsukushi","Tsukushi","Cathelicidin_antimicrobial_peptide","Cathelicidin_antimicrobial_peptide","Cathelicidin_antimicrobial_peptide","Cathelicidin_antimicrobial_peptide","Lipopolysaccharide-binding_protein","Lipopolysaccharide-binding_protein","Lipopolysaccharide-binding_protein","Lipopolysaccharide-binding_protein","Lipopolysaccharide-binding_protein","Cathelicidin_antimicrobial_peptide","Plasminogen","Plasminogen","Complement_C1r_subcomponent","Complement_C1r_subcomponent","Cathelicidin_antimicrobial_peptide","Cathelicidin_antimicrobial_peptide","Ceruloplasmin","Immunoglobulin_heavy_variable_1-18","Immunoglobulin_heavy_variable_1-18","Immunoglobulin_heavy_variable_1-18","Cathelicidin_antimicrobial_peptide","Isoform_1_of_Fibronectin","Fibrinogen_alpha_chain","Immunoglobulin_heavy_constant_alpha_2","Isoform_1_of_Fibronectin","Keratin__type_I_cytoskeletal_9","Complement_C4-A","Complement_component_C8_gamma_chain","Complement_component_C8_gamma_chain","Alpha-1B-glycoprotein","Alpha-1B-glycoprotein","Alpha-1B-glycoprotein","Alpha-1B-glycoprotein","Alpha-1B-glycoprotein","Alpha-1B-glycoprotein","Alpha-1B-glycoprotein","Kallistatin","Kallistatin","Apolipoprotein_A-I","Apolipoprotein_A-I","Apolipoprotein_A-II","Apolipoprotein_A-II","Plasminogen","Plasminogen","Plasminogen","Inter-alpha-trypsin_inhibitor_heavy_chain_H4","Inter-alpha-trypsin_inhibitor_heavy_chain_H4","Inter-alpha-trypsin_inhibitor_heavy_chain_H4","Inter-alpha-trypsin_inhibitor_heavy_chain_H4","Inter-alpha-trypsin_inhibitor_heavy_chain_H4","Lipopolysaccharide-binding_protein","Fibrinogen_alpha_chain","Heat_shock_70_kDa_protein_1B","Fructose-bisphosphate_aldolase_A","Inter-alpha-trypsin_inhibitor_heavy_chain_H1","Inter-alpha-trypsin_inhibitor_heavy_chain_H1","Inter-alpha-trypsin_inhibitor_heavy_chain_H1","Inter-alpha-trypsin_inhibitor_heavy_chain_H1","Inter-alpha-trypsin_inhibitor_heavy_chain_H1","Inter-alpha-trypsin_inhibitor_heavy_chain_H1","Inter-alpha-trypsin_inhibitor_heavy_chain_H1","L-lactate_dehydrogenase_A_chain","Serpin_B4","Tubulin_alpha-1A_chain","Serpin_B3","Protein-glutamine_gamma-glutamyltransferase_K","Annexin_A2","14-3-3_protein_sigma","Cellular_retinoic_acid-binding_protein_2","Annexin_A1","Prelamin-A_C","Endoplasmic_reticulum_chaperone_BiP","Heat_shock_protein_HSP_90-beta","Histone_H4","Elongation_factor_1-alpha_2","Alpha-enolase","Triosephosphate_isomerase","Pyruvate_kinase_PKM","ATP_synthase_subunit_alpha__mitochondrial","Heat_shock_protein_beta-1","Vimentin","Galectin-7","Keratin__type_I_cytoskeletal_9","Keratin__type_I_cytoskeletal_9","Keratin__type_I_cytoskeletal_9","Hemopexin","Hemopexin","Hemopexin","Hemopexin","Hemopexin","Hemopexin","Hemopexin","Zinc-alpha-2-glycoprotein","Prenylcysteine_oxidase_1","Prenylcysteine_oxidase_1","Protein_S100-A9","Glyceraldehyde-3-phosphate_dehydrogenase","14-3-3_protein_zeta_delta","Keratin__type_I_cytoskeletal_13","Protein_S100-A8","Immunoglobulin_heavy_constant_alpha_2","Desmoplakin","Plasminogen","Plasminogen","Complement_C1r_subcomponent","Complement_C1r_subcomponent","Immunoglobulin_heavy_constant_alpha_2","Immunoglobulin_heavy_constant_alpha_2","Prenylcysteine_oxidase_1","Prenylcysteine_oxidase_1","Prenylcysteine_oxidase_1","Prenylcysteine_oxidase_1","Prenylcysteine_oxidase_1","Prenylcysteine_oxidase_1","Prenylcysteine_oxidase_1","Prenylcysteine_oxidase_1","Prenylcysteine_oxidase_1","Serum_amyloid_P-component","Serum_amyloid_P-component","Complement_C1r_subcomponent","Cystatin-A","Cystatin-A","Sex_hormone-binding_globulin","Sex_hormone-binding_globulin","Sex_hormone-binding_globulin","Prenylcysteine_oxidase_1","Sushi__von_Willebrand_factor_type_A__EGF_and_pentraxin_domain-containing_protein_1","Keratin__type_I_cytoskeletal_17","Plasminogen","Plasma_protease_C1_inhibitor","Plasma_protease_C1_inhibitor","Plasma_protease_C1_inhibitor","BPI_fold-containing_family_B_member_1","Haptoglobin","Haptoglobin","Haptoglobin","Haptoglobin","Haptoglobin","Haptoglobin","Haptoglobin","Zinc-alpha-2-glycoprotein","Zinc-alpha-2-glycoprotein","Zinc-alpha-2-glycoprotein","Zinc-alpha-2-glycoprotein","Serum_amyloid_A-4_protein","Fibrinogen_alpha_chain","Junction_plakoglobin","Vitamin_D-binding_protein","Vitamin_D-binding_protein","Vitamin_D-binding_protein","Vitamin_D-binding_protein","Vitamin_D-binding_protein","Vitamin_D-binding_protein","Vitamin_D-binding_protein","Keratin__type_II_cytoskeletal_5","Immunoglobulin_heavy_constant_alpha_2","Angiotensinogen","Angiotensinogen","Angiotensinogen","Angiotensinogen","Angiotensinogen","Angiotensinogen","Angiotensinogen","Angiotensinogen","Angiotensinogen","Angiotensinogen","Angiotensinogen","Angiotensinogen","Angiotensinogen","Angiotensinogen","Plasminogen","Plasminogen","Protein_S100-A7","Angiotensinogen","Monocyte_differentiation_antigen_CD14","Monocyte_differentiation_antigen_CD14","Immunoglobulin_heavy_variable_5-51","Alpha-2-HS-glycoprotein","Complement_C4-A","Extracellular_matrix_protein_1"],"arrows":["to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to","to"],"color":["lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightpink","lightpink","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightblue","lightpink","lightblue","lightblue","lightblue","lightpink","lightpink","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightpink","lightblue","lightblue","lightblue","lightpink","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightpink","lightblue","lightpink","lightblue","lightblue","lightpink","lightblue","lightblue","lightpink","lightpink","lightpink","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightpink","lightblue","lightpink","lightpink","lightpink","lightblue","lightpink","lightblue","lightpink","lightpink","lightblue","lightpink","lightpink","lightblue","lightblue","lightpink","lightblue","lightpink","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightblue","lightblue","lightpink","lightpink","lightblue","lightblue","lightpink","lightblue","lightblue","lightblue","lightblue","lightpink","lightpink","lightpink","lightblue","lightpink","lightpink","lightpink","lightblue","lightblue","lightblue","lightpink","lightpink","lightblue","lightpink","lightpink","lightpink","lightblue","lightblue","lightpink","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightblue","lightblue","lightblue","lightpink","lightpink","lightblue","lightblue","lightpink","lightblue","lightpink","lightblue","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightblue","lightblue","lightblue","lightpink","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightpink","lightblue","lightpink","lightblue","lightblue","lightblue","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightblue","lightpink","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightpink","lightblue","lightblue","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightpink","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightpink","lightpink","lightblue","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightblue","lightpink","lightblue","lightpink","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightblue","lightpink","lightblue","lightblue","lightpink","lightblue","lightpink","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightpink","lightpink","lightpink","lightblue","lightpink","lightblue","lightpink","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightblue","lightblue","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightblue","lightpink","lightpink","lightblue","lightblue","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightblue","lightpink","lightblue","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightpink","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightpink","lightblue","lightblue","lightblue","lightpink","lightblue","lightblue","lightpink","lightpink","lightpink","lightpink","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightpink","lightblue","lightpink","lightblue","lightpink","lightpink","lightblue","lightpink","lightblue","lightblue","lightpink","lightblue","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightblue","lightblue","lightblue","lightblue","lightblue","lightpink","lightblue","lightpink","lightpink","lightpink","lightpink","lightblue","lightblue","lightblue","lightpink","lightpink","lightblue","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightpink","lightblue","lightblue","lightpink","lightblue","lightblue","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightpink","lightpink","lightpink","lightblue","lightblue","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightpink","lightblue","lightpink","lightpink","lightpink","lightblue","lightblue","lightblue","lightblue","lightpink","lightblue","lightblue","lightblue","lightpink","lightblue","lightpink","lightblue","lightblue","lightblue","lightblue","lightpink","lightpink","lightpink","lightpink","lightblue","lightpink","lightblue","lightblue","lightblue","lightblue","lightpink","lightpink","lightpink","lightblue","lightpink","lightblue","lightpink","lightpink","lightblue","lightblue","lightpink","lightpink","lightpink","lightblue","lightpink","lightpink","lightpink","lightblue","lightblue","lightpink","lightpink","lightpink","lightpink","lightblue","lightpink","lightblue","lightblue"],"title":["<p><b>from 4894-G/A(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.3573e+00 FDR-adjusted p-value = 2.1148e-02<\/p>","<p><b>from 620-A/G(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.4324e+00 FDR-adjusted p-value = 2.1148e-02<\/p>","<p><b>from 621-T/A(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.4324e+00 FDR-adjusted p-value = 2.1148e-02<\/p>","<p><b>from 4922-C/G(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.4925e+00 FDR-adjusted p-value = 2.1148e-02<\/p>","<p><b>from 4970-T/A(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.3545e+00 FDR-adjusted p-value = 2.1148e-02<\/p>","<p><b>from 649-T/C(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.2193e+00 FDR-adjusted p-value = 2.2281e-02<\/p>","<p><b>from 4908-A/G(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.2211e+00 FDR-adjusted p-value = 2.2281e-02<\/p>","<p><b>from 943-T/A(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.2048e+00 FDR-adjusted p-value = 2.2281e-02<\/p>","<p><b>from 632-A/G(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.2592e+00 FDR-adjusted p-value = 2.7047e-02<\/p>","<p><b>from 654-C/T(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.2341e+00 FDR-adjusted p-value = 2.7047e-02<\/p>","<p><b>from 4605-G/C(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.6262e+00 FDR-adjusted p-value = 2.7047e-02<\/p>","<p><b>from 476-C/T(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.6262e+00 FDR-adjusted p-value = 2.7047e-02<\/p>","<p><b>from 426-T/C(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.6262e+00 FDR-adjusted p-value = 2.7047e-02<\/p>","<p><b>from 4967-G/A(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.1663e+00 FDR-adjusted p-value = 2.8047e-02<\/p>","<p><b>from 4888-T/C(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.2702e+00 FDR-adjusted p-value = 3.0952e-02<\/p>","<p><b>from 5084-C/G(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.7423e+00 FDR-adjusted p-value = 3.2009e-02<\/p>","<p><b>from 4951-A/C(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.2445e+00 FDR-adjusted p-value = 3.2009e-02<\/p>","<p><b>from 937-T/C(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.0874e+00 FDR-adjusted p-value = 3.7176e-02<\/p>","<p><b>from 970-G/A(genotype_SNP) to lpa(phenotype)<\/b><br> beta = -1.8074e+00 FDR-adjusted p-value = 3.8654e-02<\/p>","<p><b>from 698-C/T(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.0932e+00 FDR-adjusted p-value = 3.8654e-02<\/p>","<p><b>from 4966-G/A(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.0847e+00 FDR-adjusted p-value = 3.8654e-02<\/p>","<p><b>from 4733-G/A(genotype_SNP) to lpa(phenotype)<\/b><br> beta = -1.8074e+00 FDR-adjusted p-value = 3.8654e-02<\/p>","<p><b>from 4245-C/T(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.2671e+00 FDR-adjusted p-value = 3.8654e-02<\/p>","<p><b>from 661-G/A(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.0879e+00 FDR-adjusted p-value = 3.8654e-02<\/p>","<p><b>from 4551-A/C(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.5311e+00 FDR-adjusted p-value = 4.0234e-02<\/p>","<p><b>from 4910-G/C(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.0477e+00 FDR-adjusted p-value = 4.0722e-02<\/p>","<p><b>from 4963-G/A(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.0546e+00 FDR-adjusted p-value = 4.5489e-02<\/p>","<p><b>from 4989-C/T(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.0691e+00 FDR-adjusted p-value = 4.5489e-02<\/p>","<p><b>from 586-A/C(genotype_SNP) to lpa(phenotype)<\/b><br> beta = 1.1833e+00 FDR-adjusted p-value = 4.5489e-02<\/p>","<p><b>from 620-A/G(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -6.9633e+00 FDR-adjusted p-value = 7.6766e-05<\/p>","<p><b>from 621-T/A(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -6.9633e+00 FDR-adjusted p-value = 7.6766e-05<\/p>","<p><b>from 663-C/T(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -6.8884e+00 FDR-adjusted p-value = 7.6766e-05<\/p>","<p><b>from 4970-T/A(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -6.4146e+00 FDR-adjusted p-value = 7.6766e-05<\/p>","<p><b>from 4754-A/T(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -6.3909e+00 FDR-adjusted p-value = 7.6967e-05<\/p>","<p><b>from 4894-G/A(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -6.3015e+00 FDR-adjusted p-value = 7.6967e-05<\/p>","<p><b>from 4967-G/A(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.8849e+00 FDR-adjusted p-value = 1.9329e-04<\/p>","<p><b>from 654-C/T(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -6.1817e+00 FDR-adjusted p-value = 1.9329e-04<\/p>","<p><b>from 698-C/T(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.8812e+00 FDR-adjusted p-value = 1.9329e-04<\/p>","<p><b>from 566-G/C(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.4990e+00 FDR-adjusted p-value = 3.1718e-04<\/p>","<p><b>from 4740-C/T(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -6.2686e+00 FDR-adjusted p-value = 3.1718e-04<\/p>","<p><b>from 586-A/C(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -6.4559e+00 FDR-adjusted p-value = 3.5363e-04<\/p>","<p><b>from 4816-T/A(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.1568e+00 FDR-adjusted p-value = 4.2994e-04<\/p>","<p><b>from 4755-T/C(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -6.0411e+00 FDR-adjusted p-value = 4.2994e-04<\/p>","<p><b>from 4942-C/T(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.5433e+00 FDR-adjusted p-value = 4.2994e-04<\/p>","<p><b>from 4774-A/G(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -6.8123e+00 FDR-adjusted p-value = 4.2994e-04<\/p>","<p><b>from 632-A/G(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.8226e+00 FDR-adjusted p-value = 4.5496e-04<\/p>","<p><b>from 4966-G/A(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.4898e+00 FDR-adjusted p-value = 5.4525e-04<\/p>","<p><b>from 568-G/C(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.4770e+00 FDR-adjusted p-value = 5.8219e-04<\/p>","<p><b>from 4888-T/C(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.9314e+00 FDR-adjusted p-value = 6.7132e-04<\/p>","<p><b>from 4908-A/G(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.2202e+00 FDR-adjusted p-value = 8.0197e-04<\/p>","<p><b>from 4933-G/A(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.2740e+00 FDR-adjusted p-value = 1.0951e-03<\/p>","<p><b>from 661-G/A(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.1611e+00 FDR-adjusted p-value = 1.4537e-03<\/p>","<p><b>from 4963-G/A(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.1228e+00 FDR-adjusted p-value = 1.6523e-03<\/p>","<p><b>from 588-T/G(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.2356e+00 FDR-adjusted p-value = 1.6523e-03<\/p>","<p><b>from 649-T/C(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -4.9343e+00 FDR-adjusted p-value = 1.6595e-03<\/p>","<p><b>from 4951-A/C(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.4834e+00 FDR-adjusted p-value = 1.8606e-03<\/p>","<p><b>from 4735-A/G(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.9570e+00 FDR-adjusted p-value = 2.0241e-03<\/p>","<p><b>from 4943-A/G(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -4.8806e+00 FDR-adjusted p-value = 2.0241e-03<\/p>","<p><b>from 937-T/C(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -4.8246e+00 FDR-adjusted p-value = 2.0241e-03<\/p>","<p><b>from 690-A/C(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -4.8685e+00 FDR-adjusted p-value = 2.0241e-03<\/p>","<p><b>from 600-T/C(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -4.8393e+00 FDR-adjusted p-value = 2.5012e-03<\/p>","<p><b>from 647-C/T(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.6968e+00 FDR-adjusted p-value = 2.5012e-03<\/p>","<p><b>from 943-T/A(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -4.6039e+00 FDR-adjusted p-value = 3.1589e-03<\/p>","<p><b>from 4785-C/A(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -4.5485e+00 FDR-adjusted p-value = 4.3161e-03<\/p>","<p><b>from 386-A/G(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.2782e+00 FDR-adjusted p-value = 8.5298e-03<\/p>","<p><b>from 4922-C/G(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -4.7875e+00 FDR-adjusted p-value = 1.4851e-02<\/p>","<p><b>from 4989-C/T(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -4.3022e+00 FDR-adjusted p-value = 1.4937e-02<\/p>","<p><b>from 4807-G/A(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -4.3578e+00 FDR-adjusted p-value = 1.5060e-02<\/p>","<p><b>from 650-A/C(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.0954e+00 FDR-adjusted p-value = 1.5965e-02<\/p>","<p><b>from 826-A/G(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -6.2624e+00 FDR-adjusted p-value = 1.6370e-02<\/p>","<p><b>from 4923-A/C(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.5388e+00 FDR-adjusted p-value = 1.6370e-02<\/p>","<p><b>from 4783-G/C(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -3.7310e+00 FDR-adjusted p-value = 1.7135e-02<\/p>","<p><b>from 940-T/G(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -3.7977e+00 FDR-adjusted p-value = 1.8127e-02<\/p>","<p><b>from 4862-G/T(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -4.0603e+00 FDR-adjusted p-value = 1.9359e-02<\/p>","<p><b>from 4846-A/G(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -4.7344e+00 FDR-adjusted p-value = 1.9512e-02<\/p>","<p><b>from 954-G/C(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -4.0257e+00 FDR-adjusted p-value = 2.0692e-02<\/p>","<p><b>from 595-A/G(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -5.1942e+00 FDR-adjusted p-value = 2.1409e-02<\/p>","<p><b>from 4910-G/C(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -3.7644e+00 FDR-adjusted p-value = 2.8290e-02<\/p>","<p><b>from 4858-A/T(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -4.0984e+00 FDR-adjusted p-value = 4.6626e-02<\/p>","<p><b>from 184-A/G(genotype_SNP) to wIS(phenotype)<\/b><br> beta = -3.4064e+00 FDR-adjusted p-value = 4.6626e-02<\/p>","<p><b>from AC(lipidomics) to lpa(phenotype)<\/b><br> beta = -1.0501e+00 FDR-adjusted p-value = 6.6405e-06<\/p>","<p><b>from PA(lipidomics) to lpa(phenotype)<\/b><br> beta = -1.0575e+00 FDR-adjusted p-value = 6.6405e-06<\/p>","<p><b>from NSer(lipidomics) to lpa(phenotype)<\/b><br> beta = -1.0788e+00 FDR-adjusted p-value = 6.6405e-06<\/p>","<p><b>from Cer(lipidomics) to lpa(phenotype)<\/b><br> beta = -1.0174e+00 FDR-adjusted p-value = 1.8462e-05<\/p>","<p><b>from dhCer(lipidomics) to lpa(phenotype)<\/b><br> beta = -1.0211e+00 FDR-adjusted p-value = 1.8462e-05<\/p>","<p><b>from LPI(lipidomics) to lpa(phenotype)<\/b><br> beta = -1.0369e+00 FDR-adjusted p-value = 1.8462e-05<\/p>","<p><b>from AC(lipidomics) to wIS(phenotype)<\/b><br> beta = 4.0039e+00 FDR-adjusted p-value = 3.9412e-05<\/p>","<p><b>from dhCer(lipidomics) to wIS(phenotype)<\/b><br> beta = 4.0447e+00 FDR-adjusted p-value = 3.9412e-05<\/p>","<p><b>from LPEp(lipidomics) to wIS(phenotype)<\/b><br> beta = 4.0598e+00 FDR-adjusted p-value = 3.9412e-05<\/p>","<p><b>from LPI(lipidomics) to wIS(phenotype)<\/b><br> beta = 4.1485e+00 FDR-adjusted p-value = 3.9412e-05<\/p>","<p><b>from Cer(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.9889e+00 FDR-adjusted p-value = 4.1588e-05<\/p>","<p><b>from LPS(lipidomics) to wIS(phenotype)<\/b><br> beta = 4.0720e+00 FDR-adjusted p-value = 4.1588e-05<\/p>","<p><b>from Sulf(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.9569e+00 FDR-adjusted p-value = 4.9732e-05<\/p>","<p><b>from PEp(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.9684e+00 FDR-adjusted p-value = 4.9732e-05<\/p>","<p><b>from PA(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.8815e+00 FDR-adjusted p-value = 4.9732e-05<\/p>","<p><b>from PE(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.9369e+00 FDR-adjusted p-value = 4.9732e-05<\/p>","<p><b>from LPEp(lipidomics) to lpa(phenotype)<\/b><br> beta = -9.6993e-01 FDR-adjusted p-value = 5.9818e-05<\/p>","<p><b>from Sulf(lipidomics) to lpa(phenotype)<\/b><br> beta = -9.8059e-01 FDR-adjusted p-value = 5.9818e-05<\/p>","<p><b>from LPE(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.8502e+00 FDR-adjusted p-value = 6.7106e-05<\/p>","<p><b>from LPS(lipidomics) to lpa(phenotype)<\/b><br> beta = -9.7552e-01 FDR-adjusted p-value = 7.7561e-05<\/p>","<p><b>from PS(lipidomics) to lpa(phenotype)<\/b><br> beta = -9.7339e-01 FDR-adjusted p-value = 7.9117e-05<\/p>","<p><b>from BMP(lipidomics) to lpa(phenotype)<\/b><br> beta = -9.3406e-01 FDR-adjusted p-value = 8.4776e-05<\/p>","<p><b>from PS(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.8555e+00 FDR-adjusted p-value = 9.5301e-05<\/p>","<p><b>from NSer(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.7493e+00 FDR-adjusted p-value = 9.7752e-05<\/p>","<p><b>from BMP(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.6744e+00 FDR-adjusted p-value = 1.1328e-04<\/p>","<p><b>from LPE(lipidomics) to lpa(phenotype)<\/b><br> beta = -9.3461e-01 FDR-adjusted p-value = 1.1989e-04<\/p>","<p><b>from DG(lipidomics) to lpa(phenotype)<\/b><br> beta = -9.5628e-01 FDR-adjusted p-value = 1.2066e-04<\/p>","<p><b>from NAPS(lipidomics) to lpa(phenotype)<\/b><br> beta = -9.3703e-01 FDR-adjusted p-value = 1.2066e-04<\/p>","<p><b>from PEp(lipidomics) to lpa(phenotype)<\/b><br> beta = -9.3472e-01 FDR-adjusted p-value = 1.2066e-04<\/p>","<p><b>from PE(lipidomics) to lpa(phenotype)<\/b><br> beta = -9.2918e-01 FDR-adjusted p-value = 1.2314e-04<\/p>","<p><b>from MG(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.7824e+00 FDR-adjusted p-value = 1.2620e-04<\/p>","<p><b>from NAPS(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.7404e+00 FDR-adjusted p-value = 1.3087e-04<\/p>","<p><b>from NSer(lipidomics) to lpa(phenotype)<\/b><br> beta = -4.2900e+01 FDR-adjusted p-value = 1.3287e-04<\/p>","<p><b>from PA(lipidomics) to lpa(phenotype)<\/b><br> beta = -4.2573e+01 FDR-adjusted p-value = 1.3287e-04<\/p>","<p><b>from AC(lipidomics) to lpa(phenotype)<\/b><br> beta = -4.2824e+01 FDR-adjusted p-value = 1.3287e-04<\/p>","<p><b>from DG(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.7462e+00 FDR-adjusted p-value = 1.7717e-04<\/p>","<p><b>from PG(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.8324e+00 FDR-adjusted p-value = 1.7717e-04<\/p>","<p><b>from dhCer(lipidomics) to lpa(phenotype)<\/b><br> beta = -4.1406e+01 FDR-adjusted p-value = 1.9892e-04<\/p>","<p><b>from Cer(lipidomics) to lpa(phenotype)<\/b><br> beta = -4.1421e+01 FDR-adjusted p-value = 1.9892e-04<\/p>","<p><b>from PG(lipidomics) to lpa(phenotype)<\/b><br> beta = -9.3874e-01 FDR-adjusted p-value = 2.5301e-04<\/p>","<p><b>from LPI(lipidomics) to lpa(phenotype)<\/b><br> beta = -4.1039e+01 FDR-adjusted p-value = 2.7217e-04<\/p>","<p><b>from LPS(lipidomics) to lpa(phenotype)<\/b><br> beta = -3.9793e+01 FDR-adjusted p-value = 5.4367e-04<\/p>","<p><b>from LPEp(lipidomics) to lpa(phenotype)<\/b><br> beta = -3.8170e+01 FDR-adjusted p-value = 6.3447e-04<\/p>","<p><b>from Sulf(lipidomics) to lpa(phenotype)<\/b><br> beta = -3.8813e+01 FDR-adjusted p-value = 6.3447e-04<\/p>","<p><b>from MhCer(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.3440e+00 FDR-adjusted p-value = 6.8206e-04<\/p>","<p><b>from BMP(lipidomics) to lpa(phenotype)<\/b><br> beta = -3.6487e+01 FDR-adjusted p-value = 1.1002e-03<\/p>","<p><b>from PS(lipidomics) to lpa(phenotype)<\/b><br> beta = -3.7287e+01 FDR-adjusted p-value = 1.1622e-03<\/p>","<p><b>from PEp(lipidomics) to lpa(phenotype)<\/b><br> beta = -3.6993e+01 FDR-adjusted p-value = 1.1622e-03<\/p>","<p><b>from MhCer(lipidomics) to lpa(phenotype)<\/b><br> beta = -8.0894e-01 FDR-adjusted p-value = 1.1669e-03<\/p>","<p><b>from MG(lipidomics) to lpa(phenotype)<\/b><br> beta = -8.2942e-01 FDR-adjusted p-value = 1.1832e-03<\/p>","<p><b>from PE(lipidomics) to lpa(phenotype)<\/b><br> beta = -3.6320e+01 FDR-adjusted p-value = 1.3946e-03<\/p>","<p><b>from LPE(lipidomics) to lpa(phenotype)<\/b><br> beta = -3.5655e+01 FDR-adjusted p-value = 1.5576e-03<\/p>","<p><b>from DG(lipidomics) to lpa(phenotype)<\/b><br> beta = -3.6248e+01 FDR-adjusted p-value = 1.8453e-03<\/p>","<p><b>from PG(lipidomics) to lpa(phenotype)<\/b><br> beta = -3.6695e+01 FDR-adjusted p-value = 1.9498e-03<\/p>","<p><b>from LPCe(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.1319e+00 FDR-adjusted p-value = 2.0060e-03<\/p>","<p><b>from AcylPG(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.2243e+00 FDR-adjusted p-value = 2.3507e-03<\/p>","<p><b>from NAPS(lipidomics) to lpa(phenotype)<\/b><br> beta = -3.4547e+01 FDR-adjusted p-value = 2.4533e-03<\/p>","<p><b>from LPC(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.0788e+00 FDR-adjusted p-value = 2.4880e-03<\/p>","<p><b>from MG(lipidomics) to lpa(phenotype)<\/b><br> beta = -3.2962e+01 FDR-adjusted p-value = 4.6657e-03<\/p>","<p><b>from MhCer(lipidomics) to lpa(phenotype)<\/b><br> beta = -3.0666e+01 FDR-adjusted p-value = 7.2095e-03<\/p>","<p><b>from PC(lipidomics) to wIS(phenotype)<\/b><br> beta = 2.8509e+00 FDR-adjusted p-value = 8.3766e-03<\/p>","<p><b>from LPCe(lipidomics) to lpa(phenotype)<\/b><br> beta = -6.8206e-01 FDR-adjusted p-value = 9.7610e-03<\/p>","<p><b>from AcylPG(lipidomics) to lpa(phenotype)<\/b><br> beta = -7.0054e-01 FDR-adjusted p-value = 1.0991e-02<\/p>","<p><b>from TG(lipidomics) to wIS(phenotype)<\/b><br> beta = 3.0397e+00 FDR-adjusted p-value = 1.1736e-02<\/p>","<p><b>from PCe(lipidomics) to wIS(phenotype)<\/b><br> beta = 2.7327e+00 FDR-adjusted p-value = 1.2596e-02<\/p>","<p><b>from NAPE(lipidomics) to wIS(phenotype)<\/b><br> beta = 2.3445e+00 FDR-adjusted p-value = 4.0540e-02<\/p>","<p><b>from TG(lipidomics) to lpa(phenotype)<\/b><br> beta = -6.4828e-01 FDR-adjusted p-value = 4.1034e-02<\/p>","<p><b>from Apolipoprotein B-100(proteomics) to lpa(phenotype)<\/b><br> beta = 4.0960e+01 FDR-adjusted p-value = 6.4824e-09<\/p>","<p><b>from Apolipoprotein B-100(proteomics) to lpa(phenotype)<\/b><br> beta = 2.1865e+01 FDR-adjusted p-value = 3.8350e-08<\/p>","<p><b>from Apolipoprotein B-100(proteomics) to lpa(phenotype)<\/b><br> beta = 9.0475e+02 FDR-adjusted p-value = 1.5146e-06<\/p>","<p><b>from Apolipoprotein B-100(proteomics) to lpa(phenotype)<\/b><br> beta = 8.6755e-01 FDR-adjusted p-value = 2.1065e-06<\/p>","<p><b>from Complement C1s subcomponent(proteomics) to lpa(phenotype)<\/b><br> beta = 1.5690e+00 FDR-adjusted p-value = 3.6709e-06<\/p>","<p><b>from Apolipoprotein(a)(proteomics) to lpa(phenotype)<\/b><br> beta = 3.1564e+01 FDR-adjusted p-value = 1.4179e-05<\/p>","<p><b>from Complement C1s subcomponent(proteomics) to lpa(phenotype)<\/b><br> beta = 6.6099e+01 FDR-adjusted p-value = 1.7667e-05<\/p>","<p><b>from Apolipoprotein(a)(proteomics) to lpa(phenotype)<\/b><br> beta = 5.8398e-01 FDR-adjusted p-value = 2.3728e-03<\/p>","<p><b>from Apolipoprotein D(proteomics) to lpa(phenotype)<\/b><br> beta = 1.4166e+02 FDR-adjusted p-value = 3.2060e-03<\/p>","<p><b>from Apolipoprotein B-100(proteomics) to wIS(phenotype)<\/b><br> beta = -6.8822e+01 FDR-adjusted p-value = 5.6818e-03<\/p>","<p><b>from Complement factor H(proteomics) to lpa(phenotype)<\/b><br> beta = 1.2219e+01 FDR-adjusted p-value = 6.4097e-03<\/p>","<p><b>from Complement C1q subcomponent subunit B(proteomics) to lpa(phenotype)<\/b><br> beta = -1.0275e+03 FDR-adjusted p-value = 9.1058e-03<\/p>","<p><b>from Apolipoprotein E(proteomics) to lpa(phenotype)<\/b><br> beta = 6.9074e+02 FDR-adjusted p-value = 9.1058e-03<\/p>","<p><b>from Serum amyloid A-4 protein(proteomics) to lpa(phenotype)<\/b><br> beta = 1.2863e+01 FDR-adjusted p-value = 1.7926e-02<\/p>","<p><b>from Apolipoprotein E(proteomics) to lpa(phenotype)<\/b><br> beta = 3.0184e+01 FDR-adjusted p-value = 1.8512e-02<\/p>","<p><b>from Immunoglobulin heavy constant mu(proteomics) to lpa(phenotype)<\/b><br> beta = -7.8901e+02 FDR-adjusted p-value = 2.0347e-02<\/p>","<p><b>from Serum amyloid A-4 protein(proteomics) to lpa(phenotype)<\/b><br> beta = 5.2846e+02 FDR-adjusted p-value = 2.0347e-02<\/p>","<p><b>from Complement C1q subcomponent subunit C(proteomics) to lpa(phenotype)<\/b><br> beta = -9.5822e+02 FDR-adjusted p-value = 2.2455e-02<\/p>","<p><b>from Apolipoprotein E(proteomics) to lpa(phenotype)<\/b><br> beta = 1.5192e+01 FDR-adjusted p-value = 2.2502e-02<\/p>","<p><b>from Apolipoprotein E(proteomics) to lpa(phenotype)<\/b><br> beta = 6.7841e-01 FDR-adjusted p-value = 2.4700e-02<\/p>","<p><b>from Immunoglobulin heavy constant mu(proteomics) to lpa(phenotype)<\/b><br> beta = -1.8280e+01 FDR-adjusted p-value = 2.4700e-02<\/p>","<p><b>from Serum amyloid A-4 protein(proteomics) to lpa(phenotype)<\/b><br> beta = 6.1240e-01 FDR-adjusted p-value = 2.4700e-02<\/p>","<p><b>from Serum amyloid A-4 protein(proteomics) to lpa(phenotype)<\/b><br> beta = 2.5780e+01 FDR-adjusted p-value = 2.7743e-02<\/p>","<p><b>from Immunoglobulin heavy constant mu(proteomics) to lpa(phenotype)<\/b><br> beta = -8.2529e-01 FDR-adjusted p-value = 2.9196e-02<\/p>","<p><b>from Immunoglobulin heavy variable 1-2(proteomics) to lpa(phenotype)<\/b><br> beta = -1.1770e+00 FDR-adjusted p-value = 2.9196e-02<\/p>","<p><b>from Complement factor H(proteomics) to lpa(phenotype)<\/b><br> beta = 2.4556e-01 FDR-adjusted p-value = 4.0394e-02<\/p>","<p><b>from Immunoglobulin heavy constant mu(proteomics) to lpa(phenotype)<\/b><br> beta = -3.3950e+01 FDR-adjusted p-value = 4.6179e-02<\/p>","<p><b>from Apolipoprotein D(proteomics) to lpa(phenotype)<\/b><br> beta = 2.4162e+00 FDR-adjusted p-value = 4.9364e-02<\/p>","<p><b>from 412-C/T(genotype_SNP) to Glucose (mg/dL)(phenotype)<\/b><br> beta = -1.4522e+01 FDR-adjusted p-value = 2.2047e-02<\/p>","<p><b>from 166-G/A(genotype_SNP) to Hypertension(phenotype)<\/b><br> beta = 3.3610e-01 FDR-adjusted p-value = 3.1581e-02<\/p>","<p><b>from 387-G/C(genotype_SNP) to Hypertension(phenotype)<\/b><br> beta = 3.4500e-01 FDR-adjusted p-value = 3.1581e-02<\/p>","<p><b>from 727-T/C(genotype_SNP) to Hypertension(phenotype)<\/b><br> beta = 7.4540e-01 FDR-adjusted p-value = 3.1581e-02<\/p>","<p><b>from 4785-C/A(genotype_SNP) to Hypertension(phenotype)<\/b><br> beta = 3.7300e-01 FDR-adjusted p-value = 3.1581e-02<\/p>","<p><b>from 402-G/A(genotype_SNP) to Hypertension(phenotype)<\/b><br> beta = 3.4540e-01 FDR-adjusted p-value = 3.4797e-02<\/p>","<p><b>from 4527-A/T(genotype_SNP) to Hypertension(phenotype)<\/b><br> beta = 4.1260e-01 FDR-adjusted p-value = 3.5983e-02<\/p>","<p><b>from 4653-A/T(genotype_SNP) to Hypertension(phenotype)<\/b><br> beta = 7.8500e-01 FDR-adjusted p-value = 3.7634e-02<\/p>","<p><b>from 723-A/G(genotype_SNP) to Hypertension(phenotype)<\/b><br> beta = 6.7730e-01 FDR-adjusted p-value = 3.7634e-02<\/p>","<p><b>from 4541-G/A(genotype_SNP) to Hypertension(phenotype)<\/b><br> beta = 4.2740e-01 FDR-adjusted p-value = 4.3197e-02<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to PA(lipidomics)<\/b><br> beta = -1.2064e+00 FDR-adjusted p-value = 1.4610e-06<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to AC(lipidomics)<\/b><br> beta = -1.2082e+00 FDR-adjusted p-value = 2.1217e-06<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to Cer(lipidomics)<\/b><br> beta = -1.1785e+00 FDR-adjusted p-value = 4.7908e-06<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to dhCer(lipidomics)<\/b><br> beta = -1.1774e+00 FDR-adjusted p-value = 5.0617e-06<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to LPS(lipidomics)<\/b><br> beta = -1.1450e+00 FDR-adjusted p-value = 9.6845e-06<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to PS(lipidomics)<\/b><br> beta = -1.1383e+00 FDR-adjusted p-value = 1.1755e-05<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to Sulf(lipidomics)<\/b><br> beta = -1.1477e+00 FDR-adjusted p-value = 1.3215e-05<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to NAPS(lipidomics)<\/b><br> beta = -1.1050e+00 FDR-adjusted p-value = 6.6443e-05<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to NSer(lipidomics)<\/b><br> beta = -6.0696e-01 FDR-adjusted p-value = 6.7087e-05<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to NSer(lipidomics)<\/b><br> beta = -1.1066e+00 FDR-adjusted p-value = 6.7087e-05<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to LPI(lipidomics)<\/b><br> beta = -1.1074e+00 FDR-adjusted p-value = 6.7401e-05<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to AC(lipidomics)<\/b><br> beta = -6.0526e-01 FDR-adjusted p-value = 7.0848e-05<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to PEp(lipidomics)<\/b><br> beta = -1.1014e+00 FDR-adjusted p-value = 9.8548e-05<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to LPEp(lipidomics)<\/b><br> beta = -1.1118e+00 FDR-adjusted p-value = 1.1372e-04<\/p>","<p><b>from Complement_factor_B(proteomics) to FC(lipidomics)<\/b><br> beta = -6.5895e-01 FDR-adjusted p-value = 1.2244e-04<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to PA(lipidomics)<\/b><br> beta = -5.8981e-01 FDR-adjusted p-value = 1.4633e-04<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to BMP(lipidomics)<\/b><br> beta = -1.1167e+00 FDR-adjusted p-value = 1.4707e-04<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to LPI(lipidomics)<\/b><br> beta = -5.7795e-01 FDR-adjusted p-value = 1.7604e-04<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to PE(lipidomics)<\/b><br> beta = -1.0799e+00 FDR-adjusted p-value = 2.0992e-04<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to dhCer(lipidomics)<\/b><br> beta = -5.7893e-01 FDR-adjusted p-value = 2.4094e-04<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to MhCer(lipidomics)<\/b><br> beta = -1.0909e+00 FDR-adjusted p-value = 2.7689e-04<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to Cer(lipidomics)<\/b><br> beta = -5.7512e-01 FDR-adjusted p-value = 3.0170e-04<\/p>","<p><b>from Apolipoprotein_E(proteomics) to dhCer(lipidomics)<\/b><br> beta = -6.4665e-01 FDR-adjusted p-value = 3.2695e-04<\/p>","<p><b>from Apolipoprotein_E(proteomics) to AC(lipidomics)<\/b><br> beta = -6.5341e-01 FDR-adjusted p-value = 3.4134e-04<\/p>","<p><b>from Apolipoprotein_E(proteomics) to PA(lipidomics)<\/b><br> beta = -6.4608e-01 FDR-adjusted p-value = 3.8908e-04<\/p>","<p><b>from Apolipoprotein_E(proteomics) to Cer(lipidomics)<\/b><br> beta = -6.4233e-01 FDR-adjusted p-value = 4.0247e-04<\/p>","<p><b>from Apolipoprotein_E(proteomics) to LPS(lipidomics)<\/b><br> beta = -6.3109e-01 FDR-adjusted p-value = 4.1659e-04<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to LPS(lipidomics)<\/b><br> beta = -5.5634e-01 FDR-adjusted p-value = 4.1659e-04<\/p>","<p><b>from Apolipoprotein_E(proteomics) to NSer(lipidomics)<\/b><br> beta = -6.3819e-01 FDR-adjusted p-value = 5.6672e-04<\/p>","<p><b>from Apolipoprotein_E(proteomics) to PS(lipidomics)<\/b><br> beta = -6.3063e-01 FDR-adjusted p-value = 5.9457e-04<\/p>","<p><b>from Complement_factor_B(proteomics) to TG(lipidomics)<\/b><br> beta = 5.5768e-01 FDR-adjusted p-value = 6.4177e-04<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to LPE(lipidomics)<\/b><br> beta = -1.0492e+00 FDR-adjusted p-value = 7.4997e-04<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to DG(lipidomics)<\/b><br> beta = -1.0148e+00 FDR-adjusted p-value = 7.8744e-04<\/p>","<p><b>from Complement_factor_B(proteomics) to MhCer(lipidomics)<\/b><br> beta = 5.8168e-01 FDR-adjusted p-value = 8.7503e-04<\/p>","<p><b>from Complement_factor_B(proteomics) to PG(lipidomics)<\/b><br> beta = 5.4888e-01 FDR-adjusted p-value = 1.2060e-03<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to PG(lipidomics)<\/b><br> beta = -9.6988e-01 FDR-adjusted p-value = 1.2060e-03<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to PS(lipidomics)<\/b><br> beta = -5.3092e-01 FDR-adjusted p-value = 1.2327e-03<\/p>","<p><b>from Properdin(proteomics) to PS(lipidomics)<\/b><br> beta = 5.8907e-01 FDR-adjusted p-value = 1.2818e-03<\/p>","<p><b>from Apolipoprotein_E(proteomics) to LPI(lipidomics)<\/b><br> beta = -6.0868e-01 FDR-adjusted p-value = 1.3607e-03<\/p>","<p><b>from Apolipoprotein_E(proteomics) to Sulf(lipidomics)<\/b><br> beta = -6.1699e-01 FDR-adjusted p-value = 1.5063e-03<\/p>","<p><b>from Apolipoprotein_E(proteomics) to NAPS(lipidomics)<\/b><br> beta = -6.1406e-01 FDR-adjusted p-value = 1.5140e-03<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to MG(lipidomics)<\/b><br> beta = -1.0048e+00 FDR-adjusted p-value = 1.6010e-03<\/p>","<p><b>from Complement_factor_B(proteomics) to PEp(lipidomics)<\/b><br> beta = 5.9333e-01 FDR-adjusted p-value = 1.6681e-03<\/p>","<p><b>from Complement_factor_B(proteomics) to Cer(lipidomics)<\/b><br> beta = 5.5938e-01 FDR-adjusted p-value = 1.6824e-03<\/p>","<p><b>from Complement_factor_B(proteomics) to dhCer(lipidomics)<\/b><br> beta = 5.6737e-01 FDR-adjusted p-value = 1.7358e-03<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to Sulf(lipidomics)<\/b><br> beta = -5.2903e-01 FDR-adjusted p-value = 1.7615e-03<\/p>","<p><b>from Properdin(proteomics) to LPS(lipidomics)<\/b><br> beta = 5.6937e-01 FDR-adjusted p-value = 1.9853e-03<\/p>","<p><b>from Complement_factor_B(proteomics) to MG(lipidomics)<\/b><br> beta = 5.6501e-01 FDR-adjusted p-value = 2.0604e-03<\/p>","<p><b>from Properdin(proteomics) to PEp(lipidomics)<\/b><br> beta = 5.9366e-01 FDR-adjusted p-value = 2.0751e-03<\/p>","<p><b>from Apolipoprotein_E(proteomics) to PEp(lipidomics)<\/b><br> beta = -5.9325e-01 FDR-adjusted p-value = 2.0751e-03<\/p>","<p><b>from Apolipoprotein_E(proteomics) to LPEp(lipidomics)<\/b><br> beta = -6.0794e-01 FDR-adjusted p-value = 2.1005e-03<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to LPEp(lipidomics)<\/b><br> beta = -5.3438e-01 FDR-adjusted p-value = 2.1005e-03<\/p>","<p><b>from Properdin(proteomics) to dhCer(lipidomics)<\/b><br> beta = 5.7593e-01 FDR-adjusted p-value = 2.2286e-03<\/p>","<p><b>from Complement_factor_B(proteomics) to PA(lipidomics)<\/b><br> beta = 5.4292e-01 FDR-adjusted p-value = 2.3997e-03<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to PS(lipidomics)<\/b><br> beta = -3.9076e-01 FDR-adjusted p-value = 2.4369e-03<\/p>","<p><b>from Complement_factor_B(proteomics) to PS(lipidomics)<\/b><br> beta = 5.4786e-01 FDR-adjusted p-value = 2.5317e-03<\/p>","<p><b>from Complement_factor_B(proteomics) to LPS(lipidomics)<\/b><br> beta = 5.3334e-01 FDR-adjusted p-value = 2.6436e-03<\/p>","<p><b>from Properdin(proteomics) to Cer(lipidomics)<\/b><br> beta = 5.6179e-01 FDR-adjusted p-value = 2.7109e-03<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to PEp(lipidomics)<\/b><br> beta = -5.0847e-01 FDR-adjusted p-value = 2.7188e-03<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to NAPS(lipidomics)<\/b><br> beta = -5.1552e-01 FDR-adjusted p-value = 2.9131e-03<\/p>","<p><b>from Properdin(proteomics) to NAPS(lipidomics)<\/b><br> beta = 5.8970e-01 FDR-adjusted p-value = 2.9131e-03<\/p>","<p><b>from Apolipoprotein_E(proteomics) to PE(lipidomics)<\/b><br> beta = -5.8805e-01 FDR-adjusted p-value = 2.9275e-03<\/p>","<p><b>from Properdin(proteomics) to PE(lipidomics)<\/b><br> beta = 5.7359e-01 FDR-adjusted p-value = 2.9275e-03<\/p>","<p><b>from Complement_factor_B(proteomics) to PE(lipidomics)<\/b><br> beta = 5.7535e-01 FDR-adjusted p-value = 2.9275e-03<\/p>","<p><b>from Properdin(proteomics) to LPI(lipidomics)<\/b><br> beta = 5.4900e-01 FDR-adjusted p-value = 3.1013e-03<\/p>","<p><b>from Complement_factor_B(proteomics) to LPI(lipidomics)<\/b><br> beta = 5.3255e-01 FDR-adjusted p-value = 3.1013e-03<\/p>","<p><b>from Complement_factor_B(proteomics) to NAPS(lipidomics)<\/b><br> beta = 5.6491e-01 FDR-adjusted p-value = 3.1093e-03<\/p>","<p><b>from Properdin(proteomics) to PG(lipidomics)<\/b><br> beta = 5.2758e-01 FDR-adjusted p-value = 3.1682e-03<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to NAPS(lipidomics)<\/b><br> beta = -3.7763e-01 FDR-adjusted p-value = 3.4675e-03<\/p>","<p><b>from Complement_C3(proteomics) to FC(lipidomics)<\/b><br> beta = -4.4007e-01 FDR-adjusted p-value = 3.5146e-03<\/p>","<p><b>from Apolipoprotein(a)(proteomics) to LPI(lipidomics)<\/b><br> beta = -4.3162e-01 FDR-adjusted p-value = 4.0625e-03<\/p>","<p><b>from Apolipoprotein(a)(proteomics) to AC(lipidomics)<\/b><br> beta = -4.3575e-01 FDR-adjusted p-value = 4.5070e-03<\/p>","<p><b>from Properdin(proteomics) to AC(lipidomics)<\/b><br> beta = 5.7026e-01 FDR-adjusted p-value = 4.5070e-03<\/p>","<p><b>from Complement_factor_B(proteomics) to AC(lipidomics)<\/b><br> beta = 5.5125e-01 FDR-adjusted p-value = 4.5070e-03<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to AC(lipidomics)<\/b><br> beta = -3.9510e-01 FDR-adjusted p-value = 4.5070e-03<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to dhCer(lipidomics)<\/b><br> beta = -3.8458e-01 FDR-adjusted p-value = 4.8503e-03<\/p>","<p><b>from Properdin(proteomics) to PA(lipidomics)<\/b><br> beta = 5.3958e-01 FDR-adjusted p-value = 4.9930e-03<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to PG(lipidomics)<\/b><br> beta = -4.7628e-01 FDR-adjusted p-value = 5.0841e-03<\/p>","<p><b>from Complement_factor_B(proteomics) to Sulf(lipidomics)<\/b><br> beta = 5.3365e-01 FDR-adjusted p-value = 5.2726e-03<\/p>","<p><b>from Apolipoprotein(a)(proteomics) to NSer(lipidomics)<\/b><br> beta = -4.2582e-01 FDR-adjusted p-value = 5.2915e-03<\/p>","<p><b>from Properdin(proteomics) to MG(lipidomics)<\/b><br> beta = 5.4183e-01 FDR-adjusted p-value = 6.0333e-03<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to PE(lipidomics)<\/b><br> beta = -4.8737e-01 FDR-adjusted p-value = 6.2349e-03<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to PEp(lipidomics)<\/b><br> beta = -3.7270e-01 FDR-adjusted p-value = 6.5150e-03<\/p>","<p><b>from Apolipoprotein_C-I(proteomics) to PS(lipidomics)<\/b><br> beta = -2.1551e-01 FDR-adjusted p-value = 6.9933e-03<\/p>","<p><b>from Apolipoprotein(a)(proteomics) to PA(lipidomics)<\/b><br> beta = -4.2094e-01 FDR-adjusted p-value = 7.1835e-03<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to LPS(lipidomics)<\/b><br> beta = -3.6721e-01 FDR-adjusted p-value = 7.1998e-03<\/p>","<p><b>from Apolipoprotein_C-I(proteomics) to AC(lipidomics)<\/b><br> beta = -2.2540e-01 FDR-adjusted p-value = 7.2145e-03<\/p>","<p><b>from Apolipoprotein(a)(proteomics) to dhCer(lipidomics)<\/b><br> beta = -4.1366e-01 FDR-adjusted p-value = 7.3840e-03<\/p>","<p><b>from Apolipoprotein_C-I(proteomics) to dhCer(lipidomics)<\/b><br> beta = -2.1906e-01 FDR-adjusted p-value = 7.3840e-03<\/p>","<p><b>from Complement_factor_B(proteomics) to CE(lipidomics)<\/b><br> beta = 5.4958e-01 FDR-adjusted p-value = 7.5315e-03<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to PE(lipidomics)<\/b><br> beta = -3.7337e-01 FDR-adjusted p-value = 7.5608e-03<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to Cer(lipidomics)<\/b><br> beta = -3.7783e-01 FDR-adjusted p-value = 7.7240e-03<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to LPE(lipidomics)<\/b><br> beta = -4.9683e-01 FDR-adjusted p-value = 8.2389e-03<\/p>","<p><b>from Apolipoprotein_E(proteomics) to LPE(lipidomics)<\/b><br> beta = -5.7563e-01 FDR-adjusted p-value = 8.2389e-03<\/p>","<p><b>from Apolipoprotein(a)(proteomics) to Cer(lipidomics)<\/b><br> beta = -4.1550e-01 FDR-adjusted p-value = 8.5572e-03<\/p>","<p><b>from Apolipoprotein(a)(proteomics) to LPS(lipidomics)<\/b><br> beta = -4.0543e-01 FDR-adjusted p-value = 8.6106e-03<\/p>","<p><b>from Properdin(proteomics) to MhCer(lipidomics)<\/b><br> beta = 5.3434e-01 FDR-adjusted p-value = 9.3333e-03<\/p>","<p><b>from Apolipoprotein_E(proteomics) to MhCer(lipidomics)<\/b><br> beta = -5.6051e-01 FDR-adjusted p-value = 9.3333e-03<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to Sulf(lipidomics)<\/b><br> beta = -3.6785e-01 FDR-adjusted p-value = 9.4205e-03<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to AcylPG(lipidomics)<\/b><br> beta = -9.1818e-01 FDR-adjusted p-value = 9.5277e-03<\/p>","<p><b>from Complement_factor_B(proteomics) to AcylPG(lipidomics)<\/b><br> beta = 5.2253e-01 FDR-adjusted p-value = 9.7278e-03<\/p>","<p><b>from Properdin(proteomics) to Sulf(lipidomics)<\/b><br> beta = 5.2555e-01 FDR-adjusted p-value = 1.0041e-02<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to PA(lipidomics)<\/b><br> beta = -3.7076e-01 FDR-adjusted p-value = 1.0289e-02<\/p>","<p><b>from Apolipoprotein_C-I(proteomics) to Cer(lipidomics)<\/b><br> beta = -2.1585e-01 FDR-adjusted p-value = 1.0453e-02<\/p>","<p><b>from Transthyretin(proteomics) to MhCer(lipidomics)<\/b><br> beta = -1.0147e+00 FDR-adjusted p-value = 1.0722e-02<\/p>","<p><b>from Apolipoprotein_E(proteomics) to MG(lipidomics)<\/b><br> beta = -5.3400e-01 FDR-adjusted p-value = 1.1344e-02<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to MG(lipidomics)<\/b><br> beta = -4.6567e-01 FDR-adjusted p-value = 1.1344e-02<\/p>","<p><b>from Apolipoprotein_E(proteomics) to PG(lipidomics)<\/b><br> beta = -5.1325e-01 FDR-adjusted p-value = 1.1654e-02<\/p>","<p><b>from Apolipoprotein_C-I(proteomics) to LPS(lipidomics)<\/b><br> beta = -2.0686e-01 FDR-adjusted p-value = 1.2025e-02<\/p>","<p><b>from Properdin(proteomics) to DG(lipidomics)<\/b><br> beta = 5.4172e-01 FDR-adjusted p-value = 1.2249e-02<\/p>","<p><b>from Complement_factor_B(proteomics) to DG(lipidomics)<\/b><br> beta = 5.2321e-01 FDR-adjusted p-value = 1.2249e-02<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to DG(lipidomics)<\/b><br> beta = -4.7037e-01 FDR-adjusted p-value = 1.2249e-02<\/p>","<p><b>from Complement_factor_B(proteomics) to BMP(lipidomics)<\/b><br> beta = 5.4255e-01 FDR-adjusted p-value = 1.3299e-02<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to BMP(lipidomics)<\/b><br> beta = -4.9274e-01 FDR-adjusted p-value = 1.3299e-02<\/p>","<p><b>from Serum_amyloid_A-4_protein(proteomics) to PS(lipidomics)<\/b><br> beta = -4.6116e-01 FDR-adjusted p-value = 1.4158e-02<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to MhCer(lipidomics)<\/b><br> beta = -3.7534e-01 FDR-adjusted p-value = 1.4890e-02<\/p>","<p><b>from Properdin(proteomics) to TG(lipidomics)<\/b><br> beta = 4.9866e-01 FDR-adjusted p-value = 1.4982e-02<\/p>","<p><b>from Clusterin(proteomics) to PS(lipidomics)<\/b><br> beta = -6.4698e-01 FDR-adjusted p-value = 1.5049e-02<\/p>","<p><b>from Clusterin(proteomics) to NAPS(lipidomics)<\/b><br> beta = -6.6187e-01 FDR-adjusted p-value = 1.5118e-02<\/p>","<p><b>from Apolipoprotein_E(proteomics) to DG(lipidomics)<\/b><br> beta = -5.1457e-01 FDR-adjusted p-value = 1.5539e-02<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to DG(lipidomics)<\/b><br> beta = -3.4783e-01 FDR-adjusted p-value = 1.6014e-02<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to BMP(lipidomics)<\/b><br> beta = -3.8153e-01 FDR-adjusted p-value = 1.6115e-02<\/p>","<p><b>from Complement_factor_B(proteomics) to LPEp(lipidomics)<\/b><br> beta = 5.1118e-01 FDR-adjusted p-value = 1.6255e-02<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to LPEp(lipidomics)<\/b><br> beta = -3.6431e-01 FDR-adjusted p-value = 1.6255e-02<\/p>","<p><b>from Properdin(proteomics) to NSer(lipidomics)<\/b><br> beta = 5.3208e-01 FDR-adjusted p-value = 1.6509e-02<\/p>","<p><b>from Complement_factor_B(proteomics) to NSer(lipidomics)<\/b><br> beta = 5.0474e-01 FDR-adjusted p-value = 1.6509e-02<\/p>","<p><b>from Complement_C1s_subcomponent(proteomics) to TG(lipidomics)<\/b><br> beta = -7.6481e-01 FDR-adjusted p-value = 1.6655e-02<\/p>","<p><b>from Apolipoprotein_E(proteomics) to BMP(lipidomics)<\/b><br> beta = -5.3723e-01 FDR-adjusted p-value = 1.7113e-02<\/p>","<p><b>from Apolipoprotein_C-I(proteomics) to PE(lipidomics)<\/b><br> beta = -2.0796e-01 FDR-adjusted p-value = 1.7409e-02<\/p>","<p><b>from Apolipoprotein_C-I(proteomics) to PA(lipidomics)<\/b><br> beta = -2.0964e-01 FDR-adjusted p-value = 1.7624e-02<\/p>","<p><b>from Serum_amyloid_A-4_protein(proteomics) to PA(lipidomics)<\/b><br> beta = -4.5732e-01 FDR-adjusted p-value = 1.7765e-02<\/p>","<p><b>from Transthyretin(proteomics) to PA(lipidomics)<\/b><br> beta = -9.3583e-01 FDR-adjusted p-value = 1.7765e-02<\/p>","<p><b>from Sulfhydryl_oxidase_1(proteomics) to LacCer(lipidomics)<\/b><br> beta = 4.3992e-01 FDR-adjusted p-value = 1.7787e-02<\/p>","<p><b>from Complement_factor_B(proteomics) to LPE(lipidomics)<\/b><br> beta = 4.7680e-01 FDR-adjusted p-value = 1.8434e-02<\/p>","<p><b>from Apolipoprotein(a)(proteomics) to LPEp(lipidomics)<\/b><br> beta = -3.9167e-01 FDR-adjusted p-value = 1.9152e-02<\/p>","<p><b>from Apolipoprotein_C-I(proteomics) to LPEp(lipidomics)<\/b><br> beta = -2.0958e-01 FDR-adjusted p-value = 1.9152e-02<\/p>","<p><b>from Properdin(proteomics) to LPEp(lipidomics)<\/b><br> beta = 5.0255e-01 FDR-adjusted p-value = 1.9152e-02<\/p>","<p><b>from Apolipoprotein(a)(proteomics) to Sulf(lipidomics)<\/b><br> beta = -3.8659e-01 FDR-adjusted p-value = 1.9291e-02<\/p>","<p><b>from Apolipoprotein_C-I(proteomics) to Sulf(lipidomics)<\/b><br> beta = -2.0165e-01 FDR-adjusted p-value = 1.9291e-02<\/p>","<p><b>from Transthyretin(proteomics) to Sulf(lipidomics)<\/b><br> beta = -9.1718e-01 FDR-adjusted p-value = 1.9321e-02<\/p>","<p><b>from Clusterin(proteomics) to Sulf(lipidomics)<\/b><br> beta = -6.3618e-01 FDR-adjusted p-value = 1.9321e-02<\/p>","<p><b>from Prenylcysteine_oxidase_1(proteomics) to Sulf(lipidomics)<\/b><br> beta = -4.8375e-01 FDR-adjusted p-value = 1.9321e-02<\/p>","<p><b>from Apolipoprotein_C-IV(proteomics) to PS(lipidomics)<\/b><br> beta = -5.3582e-01 FDR-adjusted p-value = 1.9538e-02<\/p>","<p><b>from Apolipoprotein_C-I(proteomics) to PEp(lipidomics)<\/b><br> beta = -2.0401e-01 FDR-adjusted p-value = 1.9920e-02<\/p>","<p><b>from Transthyretin(proteomics) to Cer(lipidomics)<\/b><br> beta = -9.3046e-01 FDR-adjusted p-value = 2.0007e-02<\/p>","<p><b>from Serum_amyloid_A-4_protein(proteomics) to NSer(lipidomics)<\/b><br> beta = -4.6465e-01 FDR-adjusted p-value = 2.0456e-02<\/p>","<p><b>from Apolipoprotein(a)(proteomics) to PS(lipidomics)<\/b><br> beta = -3.7129e-01 FDR-adjusted p-value = 2.0712e-02<\/p>","<p><b>from Clusterin(proteomics) to dhCer(lipidomics)<\/b><br> beta = -6.3977e-01 FDR-adjusted p-value = 2.0719e-02<\/p>","<p><b>from Transthyretin(proteomics) to dhCer(lipidomics)<\/b><br> beta = -9.2015e-01 FDR-adjusted p-value = 2.0719e-02<\/p>","<p><b>from Apolipoprotein(a)(proteomics) to NAPS(lipidomics)<\/b><br> beta = -3.8497e-01 FDR-adjusted p-value = 2.1567e-02<\/p>","<p><b>from Properdin(proteomics) to BMP(lipidomics)<\/b><br> beta = 5.1139e-01 FDR-adjusted p-value = 2.2134e-02<\/p>","<p><b>from Complement_factor_B(proteomics) to PI(lipidomics)<\/b><br> beta = 4.7689e-01 FDR-adjusted p-value = 2.2772e-02<\/p>","<p><b>from Complement_C3(proteomics) to TG(lipidomics)<\/b><br> beta = 3.5494e-01 FDR-adjusted p-value = 2.3099e-02<\/p>","<p><b>from Clusterin(proteomics) to NSer(lipidomics)<\/b><br> beta = -6.4702e-01 FDR-adjusted p-value = 2.3482e-02<\/p>","<p><b>from Clusterin(proteomics) to AC(lipidomics)<\/b><br> beta = -6.3918e-01 FDR-adjusted p-value = 2.4040e-02<\/p>","<p><b>from Serum_amyloid_A-4_protein(proteomics) to AC(lipidomics)<\/b><br> beta = -4.5310e-01 FDR-adjusted p-value = 2.4040e-02<\/p>","<p><b>from Serum_amyloid_A-4_protein(proteomics) to dhCer(lipidomics)<\/b><br> beta = -4.4084e-01 FDR-adjusted p-value = 2.4395e-02<\/p>","<p><b>from Clusterin(proteomics) to LPS(lipidomics)<\/b><br> beta = -6.2463e-01 FDR-adjusted p-value = 2.4973e-02<\/p>","<p><b>from Clusterin(proteomics) to LPEp(lipidomics)<\/b><br> beta = -6.3655e-01 FDR-adjusted p-value = 2.6021e-02<\/p>","<p><b>from Transthyretin(proteomics) to PEp(lipidomics)<\/b><br> beta = -9.1564e-01 FDR-adjusted p-value = 2.6062e-02<\/p>","<p><b>from Clusterin(proteomics) to Cer(lipidomics)<\/b><br> beta = -6.2723e-01 FDR-adjusted p-value = 2.6265e-02<\/p>","<p><b>from Serum_amyloid_A-4_protein(proteomics) to Cer(lipidomics)<\/b><br> beta = -4.3810e-01 FDR-adjusted p-value = 2.6316e-02<\/p>","<p><b>from Transthyretin(proteomics) to LPS(lipidomics)<\/b><br> beta = -8.7811e-01 FDR-adjusted p-value = 2.7996e-02<\/p>","<p><b>from Serum_amyloid_A-4_protein(proteomics) to LPS(lipidomics)<\/b><br> beta = -4.3139e-01 FDR-adjusted p-value = 2.7996e-02<\/p>","<p><b>from Transthyretin(proteomics) to PS(lipidomics)<\/b><br> beta = -8.6425e-01 FDR-adjusted p-value = 2.8405e-02<\/p>","<p><b>from Apolipoprotein_C-I(proteomics) to NSer(lipidomics)<\/b><br> beta = -1.9760e-01 FDR-adjusted p-value = 2.8467e-02<\/p>","<p><b>from Transthyretin(proteomics) to PE(lipidomics)<\/b><br> beta = -9.0505e-01 FDR-adjusted p-value = 2.9317e-02<\/p>","<p><b>from Apolipoprotein_E(proteomics) to AcylPG(lipidomics)<\/b><br> beta = -4.8262e-01 FDR-adjusted p-value = 2.9327e-02<\/p>","<p><b>from Transthyretin(proteomics) to AcylPG(lipidomics)<\/b><br> beta = -9.0141e-01 FDR-adjusted p-value = 2.9327e-02<\/p>","<p><b>from Properdin(proteomics) to AcylPG(lipidomics)<\/b><br> beta = 4.6825e-01 FDR-adjusted p-value = 2.9327e-02<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to AcylPG(lipidomics)<\/b><br> beta = -3.4353e-01 FDR-adjusted p-value = 2.9327e-02<\/p>","<p><b>from Apolipoprotein_C-I(proteomics) to MhCer(lipidomics)<\/b><br> beta = -2.0760e-01 FDR-adjusted p-value = 2.9573e-02<\/p>","<p><b>from Apolipoprotein_B-100(proteomics) to MhCer(lipidomics)<\/b><br> beta = -4.3483e-01 FDR-adjusted p-value = 2.9573e-02<\/p>","<p><b>from Clusterin(proteomics) to PA(lipidomics)<\/b><br> beta = -6.1854e-01 FDR-adjusted p-value = 2.9911e-02<\/p>","<p><b>from Complement_factor_H(proteomics) to AC(lipidomics)<\/b><br> beta = -1.8181e-01 FDR-adjusted p-value = 3.1200e-02<\/p>","<p><b>from Sulfhydryl_oxidase_1(proteomics) to PI(lipidomics)<\/b><br> beta = 3.7163e-01 FDR-adjusted p-value = 3.1743e-02<\/p>","<p><b>from Complement_factor_H(proteomics) to PA(lipidomics)<\/b><br> beta = -1.7114e-01 FDR-adjusted p-value = 3.2636e-02<\/p>","<p><b>from Serum_amyloid_A-4_protein(proteomics) to NAPS(lipidomics)<\/b><br> beta = -4.3086e-01 FDR-adjusted p-value = 3.3600e-02<\/p>","<p><b>from Transthyretin(proteomics) to LPE(lipidomics)<\/b><br> beta = -9.3387e-01 FDR-adjusted p-value = 3.4616e-02<\/p>","<p><b>from Apolipoprotein_A-II(proteomics) to PS(lipidomics)<\/b><br> beta = -5.6765e-01 FDR-adjusted p-value = 3.4678e-02<\/p>","<p><b>from Apolipoprotein(a)(proteomics) to PEp(lipidomics)<\/b><br> beta = -3.6886e-01 FDR-adjusted p-value = 3.5153e-02<\/p>","<p><b>from Complement_factor_H(proteomics) to DG(lipidomics)<\/b><br> beta = -1.8348e-01 FDR-adjusted p-value = 3.5513e-02<\/p>","<p><b>from Clusterin(proteomics) to PEp(lipidomics)<\/b><br> beta = -6.0530e-01 FDR-adjusted p-value = 3.6836e-02<\/p>","<p><b>from Transthyretin(proteomics) to NAPE(lipidomics)<\/b><br> beta = -9.1960e-01 FDR-adjusted p-value = 3.6852e-02<\/p>","<p><b>from Complement_factor_B(proteomics) to NAPE(lipidomics)<\/b><br> beta = 4.8773e-01 FDR-adjusted p-value = 3.6852e-02<\/p>","<p><b>from Sulfhydryl_oxidase_1(proteomics) to LPE(lipidomics)<\/b><br> beta = 3.3518e-01 FDR-adjusted p-value = 3.7075e-02<\/p>","<p><b>from Properdin(proteomics) to LPE(lipidomics)<\/b><br> beta = 4.5815e-01 FDR-adjusted p-value = 3.7075e-02<\/p>","<p><b>from Serum_amyloid_A-4_protein(proteomics) to Sulf(lipidomics)<\/b><br> beta = -4.1850e-01 FDR-adjusted p-value = 3.7093e-02<\/p>","<p><b>from Complement_C3(proteomics) to PEp(lipidomics)<\/b><br> beta = 3.6065e-01 FDR-adjusted p-value = 3.8718e-02<\/p>","<p><b>from Apolipoprotein_C-I(proteomics) to LPE(lipidomics)<\/b><br> beta = -1.9411e-01 FDR-adjusted p-value = 3.9382e-02<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to LPE(lipidomics)<\/b><br> beta = -3.2535e-01 FDR-adjusted p-value = 3.9975e-02<\/p>","<p><b>from Prenylcysteine_oxidase_1(proteomics) to LPS(lipidomics)<\/b><br> beta = -4.8233e-01 FDR-adjusted p-value = 4.0866e-02<\/p>","<p><b>from Apolipoprotein_C-IV(proteomics) to LPS(lipidomics)<\/b><br> beta = -4.9291e-01 FDR-adjusted p-value = 4.0866e-02<\/p>","<p><b>from Complement_factor_H(proteomics) to LPEp(lipidomics)<\/b><br> beta = -1.7869e-01 FDR-adjusted p-value = 4.1007e-02<\/p>","<p><b>from Immunoglobulin_heavy_variable_1-2(proteomics) to Sulf(lipidomics)<\/b><br> beta = 8.8174e-01 FDR-adjusted p-value = 4.1323e-02<\/p>","<p><b>from Apolipoprotein_C-IV(proteomics) to NSer(lipidomics)<\/b><br> beta = -5.1467e-01 FDR-adjusted p-value = 4.1844e-02<\/p>","<p><b>from Complement_factor_H(proteomics) to LPI(lipidomics)<\/b><br> beta = -1.7064e-01 FDR-adjusted p-value = 4.2875e-02<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to LPI(lipidomics)<\/b><br> beta = -3.2154e-01 FDR-adjusted p-value = 4.2875e-02<\/p>","<p><b>from Serum_amyloid_A-4_protein(proteomics) to LPI(lipidomics)<\/b><br> beta = -4.2893e-01 FDR-adjusted p-value = 4.2875e-02<\/p>","<p><b>from Apolipoprotein_C-I(proteomics) to LPI(lipidomics)<\/b><br> beta = -1.8604e-01 FDR-adjusted p-value = 4.3188e-02<\/p>","<p><b>from Immunoglobulin_heavy_variable_1-2(proteomics) to LPI(lipidomics)<\/b><br> beta = 9.1528e-01 FDR-adjusted p-value = 4.3188e-02<\/p>","<p><b>from Inter-alpha-trypsin_inhibitor_heavy_chain_H4(proteomics) to NSer(lipidomics)<\/b><br> beta = -8.2995e-01 FDR-adjusted p-value = 4.3896e-02<\/p>","<p><b>from Immunoglobulin_heavy_variable_1-2(proteomics) to LPE(lipidomics)<\/b><br> beta = 9.0472e-01 FDR-adjusted p-value = 4.4584e-02<\/p>","<p><b>from Apolipoprotein(a)(proteomics) to LPE(lipidomics)<\/b><br> beta = -3.4808e-01 FDR-adjusted p-value = 4.4584e-02<\/p>","<p><b>from Complement_factor_B(proteomics) to GM3(lipidomics)<\/b><br> beta = 4.5763e-01 FDR-adjusted p-value = 4.4736e-02<\/p>","<p><b>from Sulfhydryl_oxidase_1(proteomics) to GM3(lipidomics)<\/b><br> beta = 3.6786e-01 FDR-adjusted p-value = 4.4736e-02<\/p>","<p><b>from Transthyretin(proteomics) to NSer(lipidomics)<\/b><br> beta = -8.2459e-01 FDR-adjusted p-value = 4.5726e-02<\/p>","<p><b>from Complement_factor_H(proteomics) to NSer(lipidomics)<\/b><br> beta = -1.6784e-01 FDR-adjusted p-value = 4.5726e-02<\/p>","<p><b>from Transthyretin(proteomics) to AC(lipidomics)<\/b><br> beta = -8.4885e-01 FDR-adjusted p-value = 4.6373e-02<\/p>","<p><b>from Apolipoprotein_C-IV(proteomics) to AC(lipidomics)<\/b><br> beta = -5.1844e-01 FDR-adjusted p-value = 4.6373e-02<\/p>","<p><b>from Complement_factor_H(proteomics) to LPS(lipidomics)<\/b><br> beta = -1.6514e-01 FDR-adjusted p-value = 4.6507e-02<\/p>","<p><b>from Clusterin(proteomics) to PE(lipidomics)<\/b><br> beta = -5.9726e-01 FDR-adjusted p-value = 4.7090e-02<\/p>","<p><b>from Transthyretin(proteomics) to LacCer(lipidomics)<\/b><br> beta = -9.1765e-01 FDR-adjusted p-value = 4.8040e-02<\/p>","<p><b>from Complement_factor_B(proteomics) to LacCer(lipidomics)<\/b><br> beta = 4.7189e-01 FDR-adjusted p-value = 4.8040e-02<\/p>","<p><b>from Serum_paraoxonase_arylesterase_1(proteomics) to NSer(lipidomics)<\/b><br> beta = -3.0494e-01 FDR-adjusted p-value = 4.8066e-02<\/p>","<p><b>from Immunoglobulin_heavy_variable_1-2(proteomics) to NSer(lipidomics)<\/b><br> beta = 8.9229e-01 FDR-adjusted p-value = 4.8066e-02<\/p>","<p><b>from Transthyretin(proteomics) to TG(lipidomics)<\/b><br> beta = -7.6909e-01 FDR-adjusted p-value = 4.8923e-02<\/p>","<p><b>from Properdin(proteomics) to FC(lipidomics)<\/b><br> beta = -5.1466e-01 FDR-adjusted p-value = 4.8975e-02<\/p>","<p><b>from Transthyretin(proteomics) to FC(lipidomics)<\/b><br> beta = 8.2991e-01 FDR-adjusted p-value = 4.8975e-02<\/p>","<p><b>from Apolipoprotein(a)(proteomics) to BMP(lipidomics)<\/b><br> beta = -3.7078e-01 FDR-adjusted p-value = 4.9048e-02<\/p>","<p><b>from Complement_C3(proteomics) to LPS(lipidomics)<\/b><br> beta = 3.3889e-01 FDR-adjusted p-value = 4.9166e-02<\/p>","<p><b>from Heparin_cofactor_2(proteomics) to FC(lipidomics)<\/b><br> beta = -4.2323e-01 FDR-adjusted p-value = 4.9864e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to LPI(lipidomics)<\/b><br> beta = 1.0423e-01 FDR-adjusted p-value = 1.5285e-03<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to dhCer(lipidomics)<\/b><br> beta = 9.6371e-02 FDR-adjusted p-value = 4.3995e-03<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to NSer(lipidomics)<\/b><br> beta = 9.6383e-02 FDR-adjusted p-value = 4.6264e-03<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to LPS(lipidomics)<\/b><br> beta = 9.3842e-02 FDR-adjusted p-value = 4.8844e-03<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to AC(lipidomics)<\/b><br> beta = 9.6035e-02 FDR-adjusted p-value = 5.1515e-03<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Cer(lipidomics)<\/b><br> beta = 9.3015e-02 FDR-adjusted p-value = 6.2881e-03<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to LPEp(lipidomics)<\/b><br> beta = 9.3203e-02 FDR-adjusted p-value = 6.5080e-03<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to PEp(lipidomics)<\/b><br> beta = 9.0309e-02 FDR-adjusted p-value = 7.7513e-03<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to PA(lipidomics)<\/b><br> beta = 9.0105e-02 FDR-adjusted p-value = 8.8031e-03<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Sulf(lipidomics)<\/b><br> beta = 8.8883e-02 FDR-adjusted p-value = 8.8401e-03<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to PS(lipidomics)<\/b><br> beta = 8.7435e-02 FDR-adjusted p-value = 9.3170e-03<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to PE(lipidomics)<\/b><br> beta = 8.6063e-02 FDR-adjusted p-value = 1.1645e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to LPE(lipidomics)<\/b><br> beta = 8.5797e-02 FDR-adjusted p-value = 1.2637e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to MG(lipidomics)<\/b><br> beta = 8.3729e-02 FDR-adjusted p-value = 1.3205e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to PG(lipidomics)<\/b><br> beta = 7.9919e-02 FDR-adjusted p-value = 1.4639e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to NAPS(lipidomics)<\/b><br> beta = 8.1937e-02 FDR-adjusted p-value = 1.6405e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to BMP(lipidomics)<\/b><br> beta = 8.0614e-02 FDR-adjusted p-value = 2.2564e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to DG(lipidomics)<\/b><br> beta = 7.2859e-02 FDR-adjusted p-value = 3.1255e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to MhCer(lipidomics)<\/b><br> beta = 7.1521e-02 FDR-adjusted p-value = 4.3843e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to LPC(lipidomics)<\/b><br> beta = 6.9551e-02 FDR-adjusted p-value = 4.7563e-02<\/p>","<p><b>from 4970-T/A(genotype_SNP) to LPEp(lipidomics)<\/b><br> beta = -1.8075e+00 FDR-adjusted p-value = 1.9775e-04<\/p>","<p><b>from 4970-T/A(genotype_SNP) to LPS(lipidomics)<\/b><br> beta = -1.7622e+00 FDR-adjusted p-value = 2.2935e-04<\/p>","<p><b>from 4970-T/A(genotype_SNP) to Sulf(lipidomics)<\/b><br> beta = -1.7490e+00 FDR-adjusted p-value = 4.5085e-04<\/p>","<p><b>from 4970-T/A(genotype_SNP) to Cer(lipidomics)<\/b><br> beta = -1.7139e+00 FDR-adjusted p-value = 1.3759e-03<\/p>","<p><b>from 4967-G/A(genotype_SNP) to LPS(lipidomics)<\/b><br> beta = -1.5733e+00 FDR-adjusted p-value = 1.6483e-03<\/p>","<p><b>from 4970-T/A(genotype_SNP) to LPE(lipidomics)<\/b><br> beta = -1.6985e+00 FDR-adjusted p-value = 1.8218e-03<\/p>","<p><b>from 4970-T/A(genotype_SNP) to dhCer(lipidomics)<\/b><br> beta = -1.6967e+00 FDR-adjusted p-value = 1.9671e-03<\/p>","<p><b>from 4967-G/A(genotype_SNP) to LPEp(lipidomics)<\/b><br> beta = -1.5863e+00 FDR-adjusted p-value = 2.5521e-03<\/p>","<p><b>from 4970-T/A(genotype_SNP) to PEp(lipidomics)<\/b><br> beta = -1.6627e+00 FDR-adjusted p-value = 3.0505e-03<\/p>","<p><b>from 4970-T/A(genotype_SNP) to BMP(lipidomics)<\/b><br> beta = -1.6963e+00 FDR-adjusted p-value = 3.4132e-03<\/p>","<p><b>from 4970-T/A(genotype_SNP) to AC(lipidomics)<\/b><br> beta = -1.6874e+00 FDR-adjusted p-value = 3.5021e-03<\/p>","<p><b>from 4970-T/A(genotype_SNP) to MhCer(lipidomics)<\/b><br> beta = -1.6814e+00 FDR-adjusted p-value = 3.6207e-03<\/p>","<p><b>from 4967-G/A(genotype_SNP) to Sulf(lipidomics)<\/b><br> beta = -1.5435e+00 FDR-adjusted p-value = 3.7852e-03<\/p>","<p><b>from 4970-T/A(genotype_SNP) to MG(lipidomics)<\/b><br> beta = -1.6199e+00 FDR-adjusted p-value = 4.6529e-03<\/p>","<p><b>from 4967-G/A(genotype_SNP) to AC(lipidomics)<\/b><br> beta = -1.5563e+00 FDR-adjusted p-value = 5.3040e-03<\/p>","<p><b>from 4966-G/A(genotype_SNP) to LPS(lipidomics)<\/b><br> beta = -1.4837e+00 FDR-adjusted p-value = 5.3779e-03<\/p>","<p><b>from 4967-G/A(genotype_SNP) to Cer(lipidomics)<\/b><br> beta = -1.5372e+00 FDR-adjusted p-value = 5.3863e-03<\/p>","<p><b>from 4970-T/A(genotype_SNP) to PE(lipidomics)<\/b><br> beta = -1.6304e+00 FDR-adjusted p-value = 5.4773e-03<\/p>","<p><b>from 4970-T/A(genotype_SNP) to PA(lipidomics)<\/b><br> beta = -1.6501e+00 FDR-adjusted p-value = 5.6259e-03<\/p>","<p><b>from 4970-T/A(genotype_SNP) to AcylPG(lipidomics)<\/b><br> beta = -1.5705e+00 FDR-adjusted p-value = 6.2460e-03<\/p>","<p><b>from 4967-G/A(genotype_SNP) to dhCer(lipidomics)<\/b><br> beta = -1.5252e+00 FDR-adjusted p-value = 6.6604e-03<\/p>","<p><b>from 4966-G/A(genotype_SNP) to LPEp(lipidomics)<\/b><br> beta = -1.5021e+00 FDR-adjusted p-value = 6.9103e-03<\/p>","<p><b>from 4966-G/A(genotype_SNP) to AC(lipidomics)<\/b><br> beta = -1.5073e+00 FDR-adjusted p-value = 7.3889e-03<\/p>","<p><b>from 4970-T/A(genotype_SNP) to PS(lipidomics)<\/b><br> beta = -1.5959e+00 FDR-adjusted p-value = 7.5133e-03<\/p>","<p><b>from 4967-G/A(genotype_SNP) to LPE(lipidomics)<\/b><br> beta = -1.5154e+00 FDR-adjusted p-value = 7.6703e-03<\/p>","<p><b>from 4966-G/A(genotype_SNP) to LPE(lipidomics)<\/b><br> beta = -1.4766e+00 FDR-adjusted p-value = 8.9087e-03<\/p>","<p><b>from 4970-T/A(genotype_SNP) to LPI(lipidomics)<\/b><br> beta = -1.5971e+00 FDR-adjusted p-value = 9.3311e-03<\/p>","<p><b>from 4966-G/A(genotype_SNP) to Cer(lipidomics)<\/b><br> beta = -1.4748e+00 FDR-adjusted p-value = 9.4236e-03<\/p>","<p><b>from 4966-G/A(genotype_SNP) to Sulf(lipidomics)<\/b><br> beta = -1.4610e+00 FDR-adjusted p-value = 9.6313e-03<\/p>","<p><b>from 4951-A/C(genotype_SNP) to LPE(lipidomics)<\/b><br> beta = -1.7197e+00 FDR-adjusted p-value = 1.0080e-02<\/p>","<p><b>from 4966-G/A(genotype_SNP) to dhCer(lipidomics)<\/b><br> beta = -1.4692e+00 FDR-adjusted p-value = 1.0324e-02<\/p>","<p><b>from 4967-G/A(genotype_SNP) to BMP(lipidomics)<\/b><br> beta = -1.5099e+00 FDR-adjusted p-value = 1.2578e-02<\/p>","<p><b>from 4966-G/A(genotype_SNP) to BMP(lipidomics)<\/b><br> beta = -1.4805e+00 FDR-adjusted p-value = 1.2578e-02<\/p>","<p><b>from 4951-A/C(genotype_SNP) to LPEp(lipidomics)<\/b><br> beta = -1.7128e+00 FDR-adjusted p-value = 1.2776e-02<\/p>","<p><b>from 4951-A/C(genotype_SNP) to AC(lipidomics)<\/b><br> beta = -1.7119e+00 FDR-adjusted p-value = 1.4770e-02<\/p>","<p><b>from 4908-A/G(genotype_SNP) to AC(lipidomics)<\/b><br> beta = -1.4314e+00 FDR-adjusted p-value = 1.4851e-02<\/p>","<p><b>from 4951-A/C(genotype_SNP) to Sulf(lipidomics)<\/b><br> beta = -1.6760e+00 FDR-adjusted p-value = 1.5063e-02<\/p>","<p><b>from 4951-A/C(genotype_SNP) to NSer(lipidomics)<\/b><br> beta = -1.7284e+00 FDR-adjusted p-value = 1.5207e-02<\/p>","<p><b>from 4967-G/A(genotype_SNP) to NSer(lipidomics)<\/b><br> beta = -1.4378e+00 FDR-adjusted p-value = 1.5207e-02<\/p>","<p><b>from 4970-T/A(genotype_SNP) to NSer(lipidomics)<\/b><br> beta = -1.5649e+00 FDR-adjusted p-value = 1.5207e-02<\/p>","<p><b>from 4908-A/G(genotype_SNP) to NSer(lipidomics)<\/b><br> beta = -1.4522e+00 FDR-adjusted p-value = 1.5207e-02<\/p>","<p><b>from 4951-A/C(genotype_SNP) to Cer(lipidomics)<\/b><br> beta = -1.6864e+00 FDR-adjusted p-value = 1.5800e-02<\/p>","<p><b>from 4970-T/A(genotype_SNP) to PG(lipidomics)<\/b><br> beta = -1.5007e+00 FDR-adjusted p-value = 1.5831e-02<\/p>","<p><b>from 4967-G/A(genotype_SNP) to PEp(lipidomics)<\/b><br> beta = -1.4623e+00 FDR-adjusted p-value = 1.5927e-02<\/p>","<p><b>from 4967-G/A(genotype_SNP) to LPI(lipidomics)<\/b><br> beta = -1.4275e+00 FDR-adjusted p-value = 1.6964e-02<\/p>","<p><b>from 4951-A/C(genotype_SNP) to LPI(lipidomics)<\/b><br> beta = -1.7061e+00 FDR-adjusted p-value = 1.6964e-02<\/p>","<p><b>from 4967-G/A(genotype_SNP) to PE(lipidomics)<\/b><br> beta = -1.4526e+00 FDR-adjusted p-value = 1.8192e-02<\/p>","<p><b>from 4966-G/A(genotype_SNP) to PEp(lipidomics)<\/b><br> beta = -1.4150e+00 FDR-adjusted p-value = 2.0299e-02<\/p>","<p><b>from 4951-A/C(genotype_SNP) to dhCer(lipidomics)<\/b><br> beta = -1.6655e+00 FDR-adjusted p-value = 2.0433e-02<\/p>","<p><b>from 4951-A/C(genotype_SNP) to LPS(lipidomics)<\/b><br> beta = -1.6329e+00 FDR-adjusted p-value = 2.1799e-02<\/p>","<p><b>from 4967-G/A(genotype_SNP) to MhCer(lipidomics)<\/b><br> beta = -1.4653e+00 FDR-adjusted p-value = 2.2160e-02<\/p>","<p><b>from 4966-G/A(genotype_SNP) to PE(lipidomics)<\/b><br> beta = -1.4059e+00 FDR-adjusted p-value = 2.2763e-02<\/p>","<p><b>from 4967-G/A(genotype_SNP) to AcylPG(lipidomics)<\/b><br> beta = -1.3726e+00 FDR-adjusted p-value = 2.4176e-02<\/p>","<p><b>from 4966-G/A(genotype_SNP) to AcylPG(lipidomics)<\/b><br> beta = -1.3563e+00 FDR-adjusted p-value = 2.4176e-02<\/p>","<p><b>from 4967-G/A(genotype_SNP) to PA(lipidomics)<\/b><br> beta = -1.4508e+00 FDR-adjusted p-value = 2.4931e-02<\/p>","<p><b>from 4967-G/A(genotype_SNP) to MG(lipidomics)<\/b><br> beta = -1.3826e+00 FDR-adjusted p-value = 2.7191e-02<\/p>","<p><b>from 4966-G/A(genotype_SNP) to MG(lipidomics)<\/b><br> beta = -1.3805e+00 FDR-adjusted p-value = 2.7191e-02<\/p>","<p><b>from 4966-G/A(genotype_SNP) to PA(lipidomics)<\/b><br> beta = -1.4096e+00 FDR-adjusted p-value = 2.8070e-02<\/p>","<p><b>from 4966-G/A(genotype_SNP) to LPI(lipidomics)<\/b><br> beta = -1.3663e+00 FDR-adjusted p-value = 2.8730e-02<\/p>","<p><b>from 4951-A/C(genotype_SNP) to PA(lipidomics)<\/b><br> beta = -1.6384e+00 FDR-adjusted p-value = 3.0787e-02<\/p>","<p><b>from 4966-G/A(genotype_SNP) to MhCer(lipidomics)<\/b><br> beta = -1.4073e+00 FDR-adjusted p-value = 3.1759e-02<\/p>","<p><b>from 4967-G/A(genotype_SNP) to PG(lipidomics)<\/b><br> beta = -1.3338e+00 FDR-adjusted p-value = 4.3070e-02<\/p>","<p><b>from 4908-A/G(genotype_SNP) to Cer(lipidomics)<\/b><br> beta = -1.3321e+00 FDR-adjusted p-value = 4.5334e-02<\/p>","<p><b>from 4966-G/A(genotype_SNP) to NSer(lipidomics)<\/b><br> beta = -1.3345e+00 FDR-adjusted p-value = 4.6222e-02<\/p>","<p><b>from 4908-A/G(genotype_SNP) to dhCer(lipidomics)<\/b><br> beta = -1.3294e+00 FDR-adjusted p-value = 4.6953e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Immunoglobulin heavy variable 1-3(proteomics)<\/b><br> beta = 3.7605e+04 FDR-adjusted p-value = 6.2843e-03<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Complement C4-A(proteomics)<\/b><br> beta = 5.4313e+06 FDR-adjusted p-value = 6.3733e-03<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Alpha-1-acid glycoprotein 1(proteomics)<\/b><br> beta = 1.7424e+05 FDR-adjusted p-value = 1.0872e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Albumin(proteomics)<\/b><br> beta = 2.2990e+07 FDR-adjusted p-value = 1.1681e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Apolipoprotein D(proteomics)<\/b><br> beta = -2.2387e+05 FDR-adjusted p-value = 1.1811e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Immunoglobulin heavy variable 4-34(proteomics)<\/b><br> beta = 1.6013e+06 FDR-adjusted p-value = 1.3227e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Keratin, type I cytoskeletal 9(proteomics)<\/b><br> beta = -2.2593e+06 FDR-adjusted p-value = 1.6717e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Ceruloplasmin(proteomics)<\/b><br> beta = 7.5507e+04 FDR-adjusted p-value = 1.8704e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Apolipoprotein B-100(proteomics)<\/b><br> beta = -6.7487e+07 FDR-adjusted p-value = 1.9482e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Complement component C7(proteomics)<\/b><br> beta = 3.6820e+04 FDR-adjusted p-value = 2.0185e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Immunoglobulin heavy variable 3-15(proteomics)<\/b><br> beta = 8.1016e+04 FDR-adjusted p-value = 2.1917e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Complement C3(proteomics)<\/b><br> beta = 6.3188e+07 FDR-adjusted p-value = 2.2020e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Complement C1q subcomponent subunit A(proteomics)<\/b><br> beta = 1.3026e+05 FDR-adjusted p-value = 2.2081e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Vitamin D-binding protein(proteomics)<\/b><br> beta = 3.9971e+04 FDR-adjusted p-value = 3.1153e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Probable non-functional immunoglobulin kappa variable 3-7(proteomics)<\/b><br> beta = 5.1766e+05 FDR-adjusted p-value = 3.6486e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Immunoglobulin heavy variable 5-51(proteomics)<\/b><br> beta = 2.0737e+05 FDR-adjusted p-value = 3.7199e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Properdin(proteomics)<\/b><br> beta = 3.7143e+05 FDR-adjusted p-value = 3.9394e-02<\/p>","<p><b>from VNTR_estimate(genotype_VNTR) to Complement C4-B(proteomics)<\/b><br> beta = 6.7540e+05 FDR-adjusted p-value = 4.1322e-02<\/p>","<p><b>from 5055-A/G(genotype_SNP) to Kallistatin(proteomics)<\/b><br> beta = -5.1847e+00 FDR-adjusted p-value = 1.8916e-12<\/p>","<p><b>from 5052-A/T(genotype_SNP) to Kallistatin(proteomics)<\/b><br> beta = -5.1847e+00 FDR-adjusted p-value = 1.8916e-12<\/p>","<p><b>from 5069-C/T(genotype_SNP) to Kallistatin(proteomics)<\/b><br> beta = -5.1847e+00 FDR-adjusted p-value = 1.8916e-12<\/p>","<p><b>from 5037-T/G(genotype_SNP) to Kallistatin(proteomics)<\/b><br> beta = -5.1847e+00 FDR-adjusted p-value = 1.8916e-12<\/p>","<p><b>from 4599-T/A(genotype_SNP) to Kallistatin(proteomics)<\/b><br> beta = 5.1847e+00 FDR-adjusted p-value = 1.8916e-12<\/p>","<p><b>from 640-C/T(genotype_SNP) to Platelet_basic_protein(proteomics)<\/b><br> beta = 6.2027e+00 FDR-adjusted p-value = 2.3780e-09<\/p>","<p><b>from 640-C/T(genotype_SNP) to Basement_membrane-specific_heparan_sulfate_proteoglycan_core_protein(proteomics)<\/b><br> beta = 2.8373e+00 FDR-adjusted p-value = 5.8969e-09<\/p>","<p><b>from 4604-G/C(genotype_SNP) to Complement_C1q_subcomponent_subunit_B(proteomics)<\/b><br> beta = 8.2085e+00 FDR-adjusted p-value = 1.3725e-07<\/p>","<p><b>from 2470-C/T(genotype_SNP) to Complement_C1q_subcomponent_subunit_B(proteomics)<\/b><br> beta = 8.2085e+00 FDR-adjusted p-value = 1.3725e-07<\/p>","<p><b>from 4734-G/T(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -2.5798e+00 FDR-adjusted p-value = 2.1337e-07<\/p>","<p><b>from 612-G/C(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -2.5798e+00 FDR-adjusted p-value = 2.1337e-07<\/p>","<p><b>from 4577-G/A(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = 2.5798e+00 FDR-adjusted p-value = 2.1337e-07<\/p>","<p><b>from 4503-A/G(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = 2.5798e+00 FDR-adjusted p-value = 2.1337e-07<\/p>","<p><b>from 1028-C/G(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -2.5798e+00 FDR-adjusted p-value = 2.1337e-07<\/p>","<p><b>from 421-T/A(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -2.5798e+00 FDR-adjusted p-value = 2.1337e-07<\/p>","<p><b>from 681-A/G(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -2.5798e+00 FDR-adjusted p-value = 2.1337e-07<\/p>","<p><b>from 4604-G/C(genotype_SNP) to Immunoglobulin_heavy_variable_5-51(proteomics)<\/b><br> beta = 4.4968e+00 FDR-adjusted p-value = 4.6049e-07<\/p>","<p><b>from 2470-C/T(genotype_SNP) to Immunoglobulin_heavy_variable_5-51(proteomics)<\/b><br> beta = 4.4968e+00 FDR-adjusted p-value = 4.6049e-07<\/p>","<p><b>from 4604-G/C(genotype_SNP) to Complement_C1q_subcomponent_subunit_A(proteomics)<\/b><br> beta = 1.1073e+01 FDR-adjusted p-value = 5.2961e-07<\/p>","<p><b>from 2470-C/T(genotype_SNP) to Complement_C1q_subcomponent_subunit_A(proteomics)<\/b><br> beta = 1.1073e+01 FDR-adjusted p-value = 5.2961e-07<\/p>","<p><b>from 751-G/C(genotype_SNP) to Immunoglobulin_heavy_variable_1-24(proteomics)<\/b><br> beta = 2.1333e+00 FDR-adjusted p-value = 7.4109e-07<\/p>","<p><b>from 751-G/C(genotype_SNP) to Immunoglobulin_heavy_variable_1-2(proteomics)<\/b><br> beta = 2.1333e+00 FDR-adjusted p-value = 7.4109e-07<\/p>","<p><b>from 520-G/T(genotype_SNP) to Immunoglobulin_heavy_variable_1-24(proteomics)<\/b><br> beta = 2.1333e+00 FDR-adjusted p-value = 7.4109e-07<\/p>","<p><b>from 4612-T/A(genotype_SNP) to Immunoglobulin_heavy_variable_1-24(proteomics)<\/b><br> beta = 2.1333e+00 FDR-adjusted p-value = 7.4109e-07<\/p>","<p><b>from 4612-T/A(genotype_SNP) to Immunoglobulin_heavy_variable_1-2(proteomics)<\/b><br> beta = 2.1333e+00 FDR-adjusted p-value = 7.4109e-07<\/p>","<p><b>from 520-G/T(genotype_SNP) to Immunoglobulin_heavy_variable_1-2(proteomics)<\/b><br> beta = 2.1333e+00 FDR-adjusted p-value = 7.4109e-07<\/p>","<p><b>from 640-C/T(genotype_SNP) to Isoform_1_of_Fibronectin(proteomics)<\/b><br> beta = 2.3112e+01 FDR-adjusted p-value = 1.4582e-06<\/p>","<p><b>from 5040-G/C(genotype_SNP) to Keratin__type_I_cytoskeletal_9(proteomics)<\/b><br> beta = 8.7836e+00 FDR-adjusted p-value = 1.5109e-06<\/p>","<p><b>from 4889-A/T(genotype_SNP) to Keratin__type_I_cytoskeletal_9(proteomics)<\/b><br> beta = 8.7836e+00 FDR-adjusted p-value = 1.5109e-06<\/p>","<p><b>from 494-A/C(genotype_SNP) to Keratin__type_I_cytoskeletal_9(proteomics)<\/b><br> beta = 8.7836e+00 FDR-adjusted p-value = 1.5109e-06<\/p>","<p><b>from 520-G/T(genotype_SNP) to Isoform_1_of_Immunoglobulin_heavy_constant_alpha_1(proteomics)<\/b><br> beta = 7.8612e+00 FDR-adjusted p-value = 7.4360e-06<\/p>","<p><b>from 751-G/C(genotype_SNP) to Isoform_1_of_Immunoglobulin_heavy_constant_alpha_1(proteomics)<\/b><br> beta = 7.8612e+00 FDR-adjusted p-value = 7.4360e-06<\/p>","<p><b>from 4612-T/A(genotype_SNP) to Isoform_1_of_Immunoglobulin_heavy_constant_alpha_1(proteomics)<\/b><br> beta = 7.8612e+00 FDR-adjusted p-value = 7.4360e-06<\/p>","<p><b>from 2470-C/T(genotype_SNP) to Complement_C1q_subcomponent_subunit_C(proteomics)<\/b><br> beta = 7.0097e+00 FDR-adjusted p-value = 1.0975e-05<\/p>","<p><b>from 4604-G/C(genotype_SNP) to Complement_C1q_subcomponent_subunit_C(proteomics)<\/b><br> beta = 7.0097e+00 FDR-adjusted p-value = 1.0975e-05<\/p>","<p><b>from 494-A/C(genotype_SNP) to Immunoglobulin_heavy_constant_alpha_2(proteomics)<\/b><br> beta = 6.1007e+00 FDR-adjusted p-value = 1.5568e-05<\/p>","<p><b>from 5040-G/C(genotype_SNP) to Immunoglobulin_heavy_constant_alpha_2(proteomics)<\/b><br> beta = 6.1007e+00 FDR-adjusted p-value = 1.5568e-05<\/p>","<p><b>from 4889-A/T(genotype_SNP) to Immunoglobulin_heavy_constant_alpha_2(proteomics)<\/b><br> beta = 6.1007e+00 FDR-adjusted p-value = 1.5568e-05<\/p>","<p><b>from 387-G/T(genotype_SNP) to Serum_paraoxonase_arylesterase_1(proteomics)<\/b><br> beta = 7.5440e+00 FDR-adjusted p-value = 2.1071e-05<\/p>","<p><b>from 199-C/T(genotype_SNP) to Serum_paraoxonase_arylesterase_1(proteomics)<\/b><br> beta = 7.5440e+00 FDR-adjusted p-value = 2.1071e-05<\/p>","<p><b>from 728-A/G(genotype_SNP) to Alpha-2-HS-glycoprotein(proteomics)<\/b><br> beta = 1.1736e+01 FDR-adjusted p-value = 2.3041e-05<\/p>","<p><b>from 653-C/A(genotype_SNP) to Alpha-2-HS-glycoprotein(proteomics)<\/b><br> beta = 1.1736e+01 FDR-adjusted p-value = 2.3041e-05<\/p>","<p><b>from 1046-G/A(genotype_SNP) to Alpha-2-HS-glycoprotein(proteomics)<\/b><br> beta = 1.1736e+01 FDR-adjusted p-value = 2.3041e-05<\/p>","<p><b>from 4644-G/A(genotype_SNP) to Alpha-2-HS-glycoprotein(proteomics)<\/b><br> beta = 1.1736e+01 FDR-adjusted p-value = 2.3041e-05<\/p>","<p><b>from 4696-T/A(genotype_SNP) to Alpha-2-HS-glycoprotein(proteomics)<\/b><br> beta = 1.1736e+01 FDR-adjusted p-value = 2.3041e-05<\/p>","<p><b>from 591-G/T(genotype_SNP) to Alpha-2-HS-glycoprotein(proteomics)<\/b><br> beta = 1.1736e+01 FDR-adjusted p-value = 2.3041e-05<\/p>","<p><b>from 707-C/G(genotype_SNP) to Alpha-2-HS-glycoprotein(proteomics)<\/b><br> beta = -1.1736e+01 FDR-adjusted p-value = 2.3041e-05<\/p>","<p><b>from 959-C/G(genotype_SNP) to Alpha-2-HS-glycoprotein(proteomics)<\/b><br> beta = -1.1736e+01 FDR-adjusted p-value = 2.3041e-05<\/p>","<p><b>from 4976-G/T(genotype_SNP) to Immunoglobulin_heavy_variable_1-46(proteomics)<\/b><br> beta = 6.6757e+00 FDR-adjusted p-value = 3.9957e-05<\/p>","<p><b>from 5037-T/G(genotype_SNP) to Keratin__type_I_cytoskeletal_13(proteomics)<\/b><br> beta = -1.6885e+00 FDR-adjusted p-value = 5.2753e-05<\/p>","<p><b>from 4599-T/A(genotype_SNP) to Keratin__type_I_cytoskeletal_13(proteomics)<\/b><br> beta = 1.6885e+00 FDR-adjusted p-value = 5.2753e-05<\/p>","<p><b>from 5055-A/G(genotype_SNP) to Keratin__type_I_cytoskeletal_13(proteomics)<\/b><br> beta = -1.6885e+00 FDR-adjusted p-value = 5.2753e-05<\/p>","<p><b>from 5052-A/T(genotype_SNP) to Keratin__type_I_cytoskeletal_13(proteomics)<\/b><br> beta = -1.6885e+00 FDR-adjusted p-value = 5.2753e-05<\/p>","<p><b>from 5069-C/T(genotype_SNP) to Keratin__type_I_cytoskeletal_13(proteomics)<\/b><br> beta = -1.6885e+00 FDR-adjusted p-value = 5.2753e-05<\/p>","<p><b>from 5052-A/T(genotype_SNP) to Alpha-1-antitrypsin(proteomics)<\/b><br> beta = -5.0238e+00 FDR-adjusted p-value = 6.1785e-05<\/p>","<p><b>from 5037-T/G(genotype_SNP) to Alpha-1-antitrypsin(proteomics)<\/b><br> beta = -5.0238e+00 FDR-adjusted p-value = 6.1785e-05<\/p>","<p><b>from 5055-A/G(genotype_SNP) to Alpha-1-antitrypsin(proteomics)<\/b><br> beta = -5.0238e+00 FDR-adjusted p-value = 6.1785e-05<\/p>","<p><b>from 5069-C/T(genotype_SNP) to Alpha-1-antitrypsin(proteomics)<\/b><br> beta = -5.0238e+00 FDR-adjusted p-value = 6.1785e-05<\/p>","<p><b>from 4599-T/A(genotype_SNP) to Alpha-1-antitrypsin(proteomics)<\/b><br> beta = 5.0238e+00 FDR-adjusted p-value = 6.1785e-05<\/p>","<p><b>from 606-G/T(genotype_SNP) to Immunoglobulin_heavy_constant_alpha_2(proteomics)<\/b><br> beta = 4.3285e+00 FDR-adjusted p-value = 1.1245e-04<\/p>","<p><b>from 578-C/T(genotype_SNP) to Immunoglobulin_heavy_constant_alpha_2(proteomics)<\/b><br> beta = 4.3285e+00 FDR-adjusted p-value = 1.1245e-04<\/p>","<p><b>from 546-G/T(genotype_SNP) to Immunoglobulin_heavy_constant_alpha_2(proteomics)<\/b><br> beta = 4.3285e+00 FDR-adjusted p-value = 1.1245e-04<\/p>","<p><b>from 1005-G/A(genotype_SNP) to BPI_fold-containing_family_B_member_1(proteomics)<\/b><br> beta = 1.7584e+00 FDR-adjusted p-value = 1.3676e-04<\/p>","<p><b>from 917-C/T(genotype_SNP) to BPI_fold-containing_family_B_member_1(proteomics)<\/b><br> beta = 1.7584e+00 FDR-adjusted p-value = 1.3676e-04<\/p>","<p><b>from 4473-C/T(genotype_SNP) to BPI_fold-containing_family_B_member_1(proteomics)<\/b><br> beta = 1.7584e+00 FDR-adjusted p-value = 1.3676e-04<\/p>","<p><b>from 640-C/T(genotype_SNP) to Extracellular_matrix_protein_1(proteomics)<\/b><br> beta = 1.2476e+01 FDR-adjusted p-value = 1.4876e-04<\/p>","<p><b>from 1144-A/G(genotype_SNP) to N-acetylmuramoyl-L-alanine_amidase(proteomics)<\/b><br> beta = 2.6373e+00 FDR-adjusted p-value = 2.2781e-04<\/p>","<p><b>from 781-G/T(genotype_SNP) to N-acetylmuramoyl-L-alanine_amidase(proteomics)<\/b><br> beta = -2.6373e+00 FDR-adjusted p-value = 2.2781e-04<\/p>","<p><b>from 260-G/A(genotype_SNP) to N-acetylmuramoyl-L-alanine_amidase(proteomics)<\/b><br> beta = 2.6373e+00 FDR-adjusted p-value = 2.2781e-04<\/p>","<p><b>from 748-T/G(genotype_SNP) to Serum_amyloid_A-4_protein(proteomics)<\/b><br> beta = 6.0401e+00 FDR-adjusted p-value = 2.8759e-04<\/p>","<p><b>from 5004-C/A(genotype_SNP) to Serum_amyloid_A-4_protein(proteomics)<\/b><br> beta = 6.0401e+00 FDR-adjusted p-value = 2.8759e-04<\/p>","<p><b>from 199-C/T(genotype_SNP) to BPI_fold-containing_family_B_member_1(proteomics)<\/b><br> beta = 1.3164e+00 FDR-adjusted p-value = 4.9320e-04<\/p>","<p><b>from 387-G/T(genotype_SNP) to BPI_fold-containing_family_B_member_1(proteomics)<\/b><br> beta = 1.3164e+00 FDR-adjusted p-value = 4.9320e-04<\/p>","<p><b>from 602-G/A(genotype_SNP) to Keratin__type_II_cytoskeletal_1b(proteomics)<\/b><br> beta = 2.9419e+00 FDR-adjusted p-value = 5.1592e-04<\/p>","<p><b>from 4572-G/A(genotype_SNP) to Keratin__type_II_cytoskeletal_1b(proteomics)<\/b><br> beta = 2.9419e+00 FDR-adjusted p-value = 5.1592e-04<\/p>","<p><b>from 464-A/G(genotype_SNP) to Keratin__type_II_cytoskeletal_1b(proteomics)<\/b><br> beta = 2.9419e+00 FDR-adjusted p-value = 5.1592e-04<\/p>","<p><b>from 5092-G/T(genotype_SNP) to Keratin__type_II_cytoskeletal_1b(proteomics)<\/b><br> beta = -2.9419e+00 FDR-adjusted p-value = 5.1592e-04<\/p>","<p><b>from 4671-C/A(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -1.4671e+00 FDR-adjusted p-value = 5.5520e-04<\/p>","<p><b>from 4674-A/C(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -1.4671e+00 FDR-adjusted p-value = 5.5520e-04<\/p>","<p><b>from 4704-G/C(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -1.4671e+00 FDR-adjusted p-value = 5.5520e-04<\/p>","<p><b>from 4685-T/C(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -1.4671e+00 FDR-adjusted p-value = 5.5520e-04<\/p>","<p><b>from 1053-G/A(genotype_SNP) to Keratin__type_I_cytoskeletal_13(proteomics)<\/b><br> beta = -1.0287e+00 FDR-adjusted p-value = 6.8510e-04<\/p>","<p><b>from 1044-A/G(genotype_SNP) to Keratin__type_I_cytoskeletal_13(proteomics)<\/b><br> beta = -1.0287e+00 FDR-adjusted p-value = 6.8510e-04<\/p>","<p><b>from 4503-A/G(genotype_SNP) to Albumin(proteomics)<\/b><br> beta = 9.3158e+00 FDR-adjusted p-value = 7.3376e-04<\/p>","<p><b>from 681-A/G(genotype_SNP) to Albumin(proteomics)<\/b><br> beta = -9.3158e+00 FDR-adjusted p-value = 7.3376e-04<\/p>","<p><b>from 1028-C/G(genotype_SNP) to Albumin(proteomics)<\/b><br> beta = -9.3158e+00 FDR-adjusted p-value = 7.3376e-04<\/p>","<p><b>from 612-G/C(genotype_SNP) to Albumin(proteomics)<\/b><br> beta = -9.3158e+00 FDR-adjusted p-value = 7.3376e-04<\/p>","<p><b>from 421-T/A(genotype_SNP) to Albumin(proteomics)<\/b><br> beta = -9.3158e+00 FDR-adjusted p-value = 7.3376e-04<\/p>","<p><b>from 4734-G/T(genotype_SNP) to Albumin(proteomics)<\/b><br> beta = -9.3158e+00 FDR-adjusted p-value = 7.3376e-04<\/p>","<p><b>from 4577-G/A(genotype_SNP) to Albumin(proteomics)<\/b><br> beta = 9.3158e+00 FDR-adjusted p-value = 7.3376e-04<\/p>","<p><b>from 520-G/T(genotype_SNP) to Intelectin-1(proteomics)<\/b><br> beta = 7.5391e+00 FDR-adjusted p-value = 7.7568e-04<\/p>","<p><b>from 4976-G/T(genotype_SNP) to Intelectin-1(proteomics)<\/b><br> beta = 8.6833e+00 FDR-adjusted p-value = 7.7568e-04<\/p>","<p><b>from 4672-A/G(genotype_SNP) to Intelectin-1(proteomics)<\/b><br> beta = 3.7793e+00 FDR-adjusted p-value = 7.7568e-04<\/p>","<p><b>from 4612-T/A(genotype_SNP) to Intelectin-1(proteomics)<\/b><br> beta = 7.5391e+00 FDR-adjusted p-value = 7.7568e-04<\/p>","<p><b>from 751-G/C(genotype_SNP) to Intelectin-1(proteomics)<\/b><br> beta = 7.5391e+00 FDR-adjusted p-value = 7.7568e-04<\/p>","<p><b>from 520-G/T(genotype_SNP) to Immunoglobulin_kappa_variable_1-8(proteomics)<\/b><br> beta = 4.6277e+00 FDR-adjusted p-value = 1.5083e-03<\/p>","<p><b>from 4612-T/A(genotype_SNP) to Immunoglobulin_kappa_variable_1-8(proteomics)<\/b><br> beta = 4.6277e+00 FDR-adjusted p-value = 1.5083e-03<\/p>","<p><b>from 751-G/C(genotype_SNP) to Immunoglobulin_kappa_variable_1-8(proteomics)<\/b><br> beta = 4.6277e+00 FDR-adjusted p-value = 1.5083e-03<\/p>","<p><b>from 4441-A/G(genotype_SNP) to Immunoglobulin_heavy_variable_3-72(proteomics)<\/b><br> beta = 6.1375e+00 FDR-adjusted p-value = 1.6138e-03<\/p>","<p><b>from 4919-G/A(genotype_SNP) to Immunoglobulin_heavy_variable_3-7(proteomics)<\/b><br> beta = -6.1375e+00 FDR-adjusted p-value = 1.6138e-03<\/p>","<p><b>from 4441-A/G(genotype_SNP) to Immunoglobulin_heavy_variable_3-7(proteomics)<\/b><br> beta = 6.1375e+00 FDR-adjusted p-value = 1.6138e-03<\/p>","<p><b>from 4919-G/A(genotype_SNP) to Immunoglobulin_heavy_variable_3-72(proteomics)<\/b><br> beta = -6.1375e+00 FDR-adjusted p-value = 1.6138e-03<\/p>","<p><b>from 748-T/G(genotype_SNP) to Complement_component_C8_beta_chain(proteomics)<\/b><br> beta = 4.2993e+00 FDR-adjusted p-value = 1.7577e-03<\/p>","<p><b>from 5004-C/A(genotype_SNP) to Complement_component_C8_beta_chain(proteomics)<\/b><br> beta = 4.2993e+00 FDR-adjusted p-value = 1.7577e-03<\/p>","<p><b>from 199-C/T(genotype_SNP) to Tsukushi(proteomics)<\/b><br> beta = 2.5210e+00 FDR-adjusted p-value = 1.8605e-03<\/p>","<p><b>from 387-G/T(genotype_SNP) to Tsukushi(proteomics)<\/b><br> beta = 2.5210e+00 FDR-adjusted p-value = 1.8605e-03<\/p>","<p><b>from 651-C/G(genotype_SNP) to Cathelicidin_antimicrobial_peptide(proteomics)<\/b><br> beta = 1.5971e+00 FDR-adjusted p-value = 1.9106e-03<\/p>","<p><b>from 4683-T/C(genotype_SNP) to Cathelicidin_antimicrobial_peptide(proteomics)<\/b><br> beta = 1.5971e+00 FDR-adjusted p-value = 1.9106e-03<\/p>","<p><b>from 4842-C/T(genotype_SNP) to Cathelicidin_antimicrobial_peptide(proteomics)<\/b><br> beta = 1.5971e+00 FDR-adjusted p-value = 1.9106e-03<\/p>","<p><b>from 5078-G/A(genotype_SNP) to Cathelicidin_antimicrobial_peptide(proteomics)<\/b><br> beta = 1.5971e+00 FDR-adjusted p-value = 1.9106e-03<\/p>","<p><b>from 5052-A/T(genotype_SNP) to Lipopolysaccharide-binding_protein(proteomics)<\/b><br> beta = -5.1605e+00 FDR-adjusted p-value = 2.0903e-03<\/p>","<p><b>from 4599-T/A(genotype_SNP) to Lipopolysaccharide-binding_protein(proteomics)<\/b><br> beta = 5.1605e+00 FDR-adjusted p-value = 2.0903e-03<\/p>","<p><b>from 5037-T/G(genotype_SNP) to Lipopolysaccharide-binding_protein(proteomics)<\/b><br> beta = -5.1605e+00 FDR-adjusted p-value = 2.0903e-03<\/p>","<p><b>from 5069-C/T(genotype_SNP) to Lipopolysaccharide-binding_protein(proteomics)<\/b><br> beta = -5.1605e+00 FDR-adjusted p-value = 2.0903e-03<\/p>","<p><b>from 5055-A/G(genotype_SNP) to Lipopolysaccharide-binding_protein(proteomics)<\/b><br> beta = -5.1605e+00 FDR-adjusted p-value = 2.0903e-03<\/p>","<p><b>from 4266-G/T(genotype_SNP) to Cathelicidin_antimicrobial_peptide(proteomics)<\/b><br> beta = 1.5982e+00 FDR-adjusted p-value = 2.1914e-03<\/p>","<p><b>from 950-A/G(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -1.4623e+00 FDR-adjusted p-value = 2.2038e-03<\/p>","<p><b>from 372-C/T(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -1.4623e+00 FDR-adjusted p-value = 2.2038e-03<\/p>","<p><b>from 748-T/G(genotype_SNP) to Complement_C1r_subcomponent(proteomics)<\/b><br> beta = 3.2401e+00 FDR-adjusted p-value = 2.2457e-03<\/p>","<p><b>from 5004-C/A(genotype_SNP) to Complement_C1r_subcomponent(proteomics)<\/b><br> beta = 3.2401e+00 FDR-adjusted p-value = 2.2457e-03<\/p>","<p><b>from 4260-A/C(genotype_SNP) to Cathelicidin_antimicrobial_peptide(proteomics)<\/b><br> beta = 1.5979e+00 FDR-adjusted p-value = 3.3556e-03<\/p>","<p><b>from 1323-T/C(genotype_SNP) to Cathelicidin_antimicrobial_peptide(proteomics)<\/b><br> beta = 1.6188e+00 FDR-adjusted p-value = 3.3556e-03<\/p>","<p><b>from 4755-T/G(genotype_SNP) to Ceruloplasmin(proteomics)<\/b><br> beta = -4.2201e+00 FDR-adjusted p-value = 3.6557e-03<\/p>","<p><b>from 520-G/T(genotype_SNP) to Immunoglobulin_heavy_variable_1-18(proteomics)<\/b><br> beta = 4.0300e+00 FDR-adjusted p-value = 3.9620e-03<\/p>","<p><b>from 751-G/C(genotype_SNP) to Immunoglobulin_heavy_variable_1-18(proteomics)<\/b><br> beta = 4.0300e+00 FDR-adjusted p-value = 3.9620e-03<\/p>","<p><b>from 4612-T/A(genotype_SNP) to Immunoglobulin_heavy_variable_1-18(proteomics)<\/b><br> beta = 4.0300e+00 FDR-adjusted p-value = 3.9620e-03<\/p>","<p><b>from 4228-G/A(genotype_SNP) to Cathelicidin_antimicrobial_peptide(proteomics)<\/b><br> beta = 1.5929e+00 FDR-adjusted p-value = 4.5271e-03<\/p>","<p><b>from 5013-G/T(genotype_SNP) to Isoform_1_of_Fibronectin(proteomics)<\/b><br> beta = 1.2279e+01 FDR-adjusted p-value = 6.2942e-03<\/p>","<p><b>from 4755-T/A(genotype_SNP) to Fibrinogen_alpha_chain(proteomics)<\/b><br> beta = -6.2191e+00 FDR-adjusted p-value = 7.1645e-03<\/p>","<p><b>from 690-A/G(genotype_SNP) to Immunoglobulin_heavy_constant_alpha_2(proteomics)<\/b><br> beta = 3.1420e+00 FDR-adjusted p-value = 7.2058e-03<\/p>","<p><b>from 4755-T/A(genotype_SNP) to Isoform_1_of_Fibronectin(proteomics)<\/b><br> beta = -1.0056e+01 FDR-adjusted p-value = 7.2384e-03<\/p>","<p><b>from 4583-T/A(genotype_SNP) to Keratin__type_I_cytoskeletal_9(proteomics)<\/b><br> beta = 5.2470e+00 FDR-adjusted p-value = 7.9805e-03<\/p>","<p><b>from 4755-T/G(genotype_SNP) to Complement_C4-A(proteomics)<\/b><br> beta = -2.4239e+00 FDR-adjusted p-value = 7.9929e-03<\/p>","<p><b>from 5004-C/A(genotype_SNP) to Complement_component_C8_gamma_chain(proteomics)<\/b><br> beta = 2.6827e+00 FDR-adjusted p-value = 9.8131e-03<\/p>","<p><b>from 748-T/G(genotype_SNP) to Complement_component_C8_gamma_chain(proteomics)<\/b><br> beta = 2.6827e+00 FDR-adjusted p-value = 9.8131e-03<\/p>","<p><b>from 4734-G/T(genotype_SNP) to Alpha-1B-glycoprotein(proteomics)<\/b><br> beta = -4.2129e+00 FDR-adjusted p-value = 1.3115e-02<\/p>","<p><b>from 4503-A/G(genotype_SNP) to Alpha-1B-glycoprotein(proteomics)<\/b><br> beta = 4.2129e+00 FDR-adjusted p-value = 1.3115e-02<\/p>","<p><b>from 612-G/C(genotype_SNP) to Alpha-1B-glycoprotein(proteomics)<\/b><br> beta = -4.2129e+00 FDR-adjusted p-value = 1.3115e-02<\/p>","<p><b>from 1028-C/G(genotype_SNP) to Alpha-1B-glycoprotein(proteomics)<\/b><br> beta = -4.2129e+00 FDR-adjusted p-value = 1.3115e-02<\/p>","<p><b>from 4577-G/A(genotype_SNP) to Alpha-1B-glycoprotein(proteomics)<\/b><br> beta = 4.2129e+00 FDR-adjusted p-value = 1.3115e-02<\/p>","<p><b>from 681-A/G(genotype_SNP) to Alpha-1B-glycoprotein(proteomics)<\/b><br> beta = -4.2129e+00 FDR-adjusted p-value = 1.3115e-02<\/p>","<p><b>from 421-T/A(genotype_SNP) to Alpha-1B-glycoprotein(proteomics)<\/b><br> beta = -4.2129e+00 FDR-adjusted p-value = 1.3115e-02<\/p>","<p><b>from 4598-T/C(genotype_SNP) to Kallistatin(proteomics)<\/b><br> beta = 2.5171e+00 FDR-adjusted p-value = 1.5108e-02<\/p>","<p><b>from 4623-T/C(genotype_SNP) to Kallistatin(proteomics)<\/b><br> beta = 2.5171e+00 FDR-adjusted p-value = 1.5108e-02<\/p>","<p><b>from 199-C/T(genotype_SNP) to Apolipoprotein_A-I(proteomics)<\/b><br> beta = 3.7094e+00 FDR-adjusted p-value = 1.5256e-02<\/p>","<p><b>from 387-G/T(genotype_SNP) to Apolipoprotein_A-I(proteomics)<\/b><br> beta = 3.7094e+00 FDR-adjusted p-value = 1.5256e-02<\/p>","<p><b>from 387-G/T(genotype_SNP) to Apolipoprotein_A-II(proteomics)<\/b><br> beta = 3.7094e+00 FDR-adjusted p-value = 1.5256e-02<\/p>","<p><b>from 199-C/T(genotype_SNP) to Apolipoprotein_A-II(proteomics)<\/b><br> beta = 3.7094e+00 FDR-adjusted p-value = 1.5256e-02<\/p>","<p><b>from 717-G/A(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -1.1186e+00 FDR-adjusted p-value = 1.5325e-02<\/p>","<p><b>from 824-C/A(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -1.1186e+00 FDR-adjusted p-value = 1.5325e-02<\/p>","<p><b>from 826-A/G(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -9.6352e-01 FDR-adjusted p-value = 1.6291e-02<\/p>","<p><b>from 5037-T/G(genotype_SNP) to Inter-alpha-trypsin_inhibitor_heavy_chain_H4(proteomics)<\/b><br> beta = -2.4736e+00 FDR-adjusted p-value = 1.6429e-02<\/p>","<p><b>from 5069-C/T(genotype_SNP) to Inter-alpha-trypsin_inhibitor_heavy_chain_H4(proteomics)<\/b><br> beta = -2.4736e+00 FDR-adjusted p-value = 1.6429e-02<\/p>","<p><b>from 5052-A/T(genotype_SNP) to Inter-alpha-trypsin_inhibitor_heavy_chain_H4(proteomics)<\/b><br> beta = -2.4736e+00 FDR-adjusted p-value = 1.6429e-02<\/p>","<p><b>from 4599-T/A(genotype_SNP) to Inter-alpha-trypsin_inhibitor_heavy_chain_H4(proteomics)<\/b><br> beta = 2.4736e+00 FDR-adjusted p-value = 1.6429e-02<\/p>","<p><b>from 5055-A/G(genotype_SNP) to Inter-alpha-trypsin_inhibitor_heavy_chain_H4(proteomics)<\/b><br> beta = -2.4736e+00 FDR-adjusted p-value = 1.6429e-02<\/p>","<p><b>from 907-A/G(genotype_SNP) to Lipopolysaccharide-binding_protein(proteomics)<\/b><br> beta = 2.5864e+00 FDR-adjusted p-value = 1.6669e-02<\/p>","<p><b>from 640-C/T(genotype_SNP) to Fibrinogen_alpha_chain(proteomics)<\/b><br> beta = 1.0788e+01 FDR-adjusted p-value = 1.6696e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Heat_shock_70_kDa_protein_1B(proteomics)<\/b><br> beta = 9.3151e-01 FDR-adjusted p-value = 1.6885e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Fructose-bisphosphate_aldolase_A(proteomics)<\/b><br> beta = 9.4667e-01 FDR-adjusted p-value = 1.7103e-02<\/p>","<p><b>from 4734-G/T(genotype_SNP) to Inter-alpha-trypsin_inhibitor_heavy_chain_H1(proteomics)<\/b><br> beta = -2.4687e+00 FDR-adjusted p-value = 1.7720e-02<\/p>","<p><b>from 1028-C/G(genotype_SNP) to Inter-alpha-trypsin_inhibitor_heavy_chain_H1(proteomics)<\/b><br> beta = -2.4687e+00 FDR-adjusted p-value = 1.7720e-02<\/p>","<p><b>from 612-G/C(genotype_SNP) to Inter-alpha-trypsin_inhibitor_heavy_chain_H1(proteomics)<\/b><br> beta = -2.4687e+00 FDR-adjusted p-value = 1.7720e-02<\/p>","<p><b>from 4503-A/G(genotype_SNP) to Inter-alpha-trypsin_inhibitor_heavy_chain_H1(proteomics)<\/b><br> beta = 2.4687e+00 FDR-adjusted p-value = 1.7720e-02<\/p>","<p><b>from 4577-G/A(genotype_SNP) to Inter-alpha-trypsin_inhibitor_heavy_chain_H1(proteomics)<\/b><br> beta = 2.4687e+00 FDR-adjusted p-value = 1.7720e-02<\/p>","<p><b>from 421-T/A(genotype_SNP) to Inter-alpha-trypsin_inhibitor_heavy_chain_H1(proteomics)<\/b><br> beta = -2.4687e+00 FDR-adjusted p-value = 1.7720e-02<\/p>","<p><b>from 681-A/G(genotype_SNP) to Inter-alpha-trypsin_inhibitor_heavy_chain_H1(proteomics)<\/b><br> beta = -2.4687e+00 FDR-adjusted p-value = 1.7720e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to L-lactate_dehydrogenase_A_chain(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Serpin_B4(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Tubulin_alpha-1A_chain(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Serpin_B3(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Protein-glutamine_gamma-glutamyltransferase_K(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Annexin_A2(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to 14-3-3_protein_sigma(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Cellular_retinoic_acid-binding_protein_2(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Annexin_A1(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Prelamin-A_C(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Endoplasmic_reticulum_chaperone_BiP(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Heat_shock_protein_HSP_90-beta(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Histone_H4(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Elongation_factor_1-alpha_2(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Alpha-enolase(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Triosephosphate_isomerase(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Pyruvate_kinase_PKM(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to ATP_synthase_subunit_alpha__mitochondrial(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Heat_shock_protein_beta-1(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Vimentin(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Galectin-7(proteomics)<\/b><br> beta = 4.8966e-01 FDR-adjusted p-value = 1.7809e-02<\/p>","<p><b>from 4738-T/G(genotype_SNP) to Keratin__type_I_cytoskeletal_9(proteomics)<\/b><br> beta = 4.0170e+00 FDR-adjusted p-value = 1.8039e-02<\/p>","<p><b>from 746-T/C(genotype_SNP) to Keratin__type_I_cytoskeletal_9(proteomics)<\/b><br> beta = 4.0170e+00 FDR-adjusted p-value = 1.8039e-02<\/p>","<p><b>from 4716-G/T(genotype_SNP) to Keratin__type_I_cytoskeletal_9(proteomics)<\/b><br> beta = 4.0170e+00 FDR-adjusted p-value = 1.8039e-02<\/p>","<p><b>from 421-T/A(genotype_SNP) to Hemopexin(proteomics)<\/b><br> beta = -4.5809e+00 FDR-adjusted p-value = 1.8330e-02<\/p>","<p><b>from 4577-G/A(genotype_SNP) to Hemopexin(proteomics)<\/b><br> beta = 4.5809e+00 FDR-adjusted p-value = 1.8330e-02<\/p>","<p><b>from 681-A/G(genotype_SNP) to Hemopexin(proteomics)<\/b><br> beta = -4.5809e+00 FDR-adjusted p-value = 1.8330e-02<\/p>","<p><b>from 612-G/C(genotype_SNP) to Hemopexin(proteomics)<\/b><br> beta = -4.5809e+00 FDR-adjusted p-value = 1.8330e-02<\/p>","<p><b>from 4503-A/G(genotype_SNP) to Hemopexin(proteomics)<\/b><br> beta = 4.5809e+00 FDR-adjusted p-value = 1.8330e-02<\/p>","<p><b>from 4734-G/T(genotype_SNP) to Hemopexin(proteomics)<\/b><br> beta = -4.5809e+00 FDR-adjusted p-value = 1.8330e-02<\/p>","<p><b>from 1028-C/G(genotype_SNP) to Hemopexin(proteomics)<\/b><br> beta = -4.5809e+00 FDR-adjusted p-value = 1.8330e-02<\/p>","<p><b>from 4653-A/T(genotype_SNP) to Zinc-alpha-2-glycoprotein(proteomics)<\/b><br> beta = -2.5541e+00 FDR-adjusted p-value = 1.9436e-02<\/p>","<p><b>from 199-C/T(genotype_SNP) to Prenylcysteine_oxidase_1(proteomics)<\/b><br> beta = 3.6727e+00 FDR-adjusted p-value = 1.9946e-02<\/p>","<p><b>from 387-G/T(genotype_SNP) to Prenylcysteine_oxidase_1(proteomics)<\/b><br> beta = 3.6727e+00 FDR-adjusted p-value = 1.9946e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Protein_S100-A9(proteomics)<\/b><br> beta = 2.1301e+01 FDR-adjusted p-value = 2.0501e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Glyceraldehyde-3-phosphate_dehydrogenase(proteomics)<\/b><br> beta = 8.9531e-01 FDR-adjusted p-value = 2.1640e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to 14-3-3_protein_zeta_delta(proteomics)<\/b><br> beta = 8.1144e-01 FDR-adjusted p-value = 2.1919e-02<\/p>","<p><b>from 1091-C/A(genotype_SNP) to Keratin__type_I_cytoskeletal_13(proteomics)<\/b><br> beta = -7.4638e-01 FDR-adjusted p-value = 2.1980e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Protein_S100-A8(proteomics)<\/b><br> beta = 9.3240e+00 FDR-adjusted p-value = 2.2956e-02<\/p>","<p><b>from 4583-T/A(genotype_SNP) to Immunoglobulin_heavy_constant_alpha_2(proteomics)<\/b><br> beta = 3.4645e+00 FDR-adjusted p-value = 2.3902e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Desmoplakin(proteomics)<\/b><br> beta = 8.3204e-01 FDR-adjusted p-value = 2.4368e-02<\/p>","<p><b>from 1010-G/A(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -1.1841e+00 FDR-adjusted p-value = 2.4472e-02<\/p>","<p><b>from 4755-T/G(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -1.2066e+00 FDR-adjusted p-value = 2.4472e-02<\/p>","<p><b>from 4461-G/C(genotype_SNP) to Complement_C1r_subcomponent(proteomics)<\/b><br> beta = 2.0829e+00 FDR-adjusted p-value = 2.5009e-02<\/p>","<p><b>from 4462-T/C(genotype_SNP) to Complement_C1r_subcomponent(proteomics)<\/b><br> beta = 2.0829e+00 FDR-adjusted p-value = 2.5009e-02<\/p>","<p><b>from 572-T/G(genotype_SNP) to Immunoglobulin_heavy_constant_alpha_2(proteomics)<\/b><br> beta = 2.9303e+00 FDR-adjusted p-value = 2.5685e-02<\/p>","<p><b>from 185-A/G(genotype_SNP) to Immunoglobulin_heavy_constant_alpha_2(proteomics)<\/b><br> beta = 1.6149e+00 FDR-adjusted p-value = 2.5685e-02<\/p>","<p><b>from 4228-G/A(genotype_SNP) to Prenylcysteine_oxidase_1(proteomics)<\/b><br> beta = 4.1759e+00 FDR-adjusted p-value = 2.6380e-02<\/p>","<p><b>from 1205-T/A(genotype_SNP) to Prenylcysteine_oxidase_1(proteomics)<\/b><br> beta = 2.2416e+00 FDR-adjusted p-value = 2.6380e-02<\/p>","<p><b>from 4260-A/C(genotype_SNP) to Prenylcysteine_oxidase_1(proteomics)<\/b><br> beta = 4.1115e+00 FDR-adjusted p-value = 2.6380e-02<\/p>","<p><b>from 1323-T/C(genotype_SNP) to Prenylcysteine_oxidase_1(proteomics)<\/b><br> beta = 4.1606e+00 FDR-adjusted p-value = 2.6380e-02<\/p>","<p><b>from 4842-C/T(genotype_SNP) to Prenylcysteine_oxidase_1(proteomics)<\/b><br> beta = 4.0102e+00 FDR-adjusted p-value = 2.6380e-02<\/p>","<p><b>from 5078-G/A(genotype_SNP) to Prenylcysteine_oxidase_1(proteomics)<\/b><br> beta = 4.0102e+00 FDR-adjusted p-value = 2.6380e-02<\/p>","<p><b>from 4683-T/C(genotype_SNP) to Prenylcysteine_oxidase_1(proteomics)<\/b><br> beta = 4.0102e+00 FDR-adjusted p-value = 2.6380e-02<\/p>","<p><b>from 651-C/G(genotype_SNP) to Prenylcysteine_oxidase_1(proteomics)<\/b><br> beta = 4.0102e+00 FDR-adjusted p-value = 2.6380e-02<\/p>","<p><b>from 4266-G/T(genotype_SNP) to Prenylcysteine_oxidase_1(proteomics)<\/b><br> beta = 4.0150e+00 FDR-adjusted p-value = 2.6380e-02<\/p>","<p><b>from 150-G/T(genotype_SNP) to Serum_amyloid_P-component(proteomics)<\/b><br> beta = 3.5931e+00 FDR-adjusted p-value = 2.6610e-02<\/p>","<p><b>from 1070-A/G(genotype_SNP) to Serum_amyloid_P-component(proteomics)<\/b><br> beta = 3.5931e+00 FDR-adjusted p-value = 2.6610e-02<\/p>","<p><b>from 1269-A/G(genotype_SNP) to Complement_C1r_subcomponent(proteomics)<\/b><br> beta = 2.0789e+00 FDR-adjusted p-value = 2.7462e-02<\/p>","<p><b>from 4922-C/G(genotype_SNP) to Cystatin-A(proteomics)<\/b><br> beta = 4.8318e-01 FDR-adjusted p-value = 2.8177e-02<\/p>","<p><b>from 1269-A/G(genotype_SNP) to Cystatin-A(proteomics)<\/b><br> beta = 3.6664e-01 FDR-adjusted p-value = 2.8177e-02<\/p>","<p><b>from 1144-A/G(genotype_SNP) to Sex_hormone-binding_globulin(proteomics)<\/b><br> beta = 2.6113e+00 FDR-adjusted p-value = 2.8634e-02<\/p>","<p><b>from 260-G/A(genotype_SNP) to Sex_hormone-binding_globulin(proteomics)<\/b><br> beta = 2.6113e+00 FDR-adjusted p-value = 2.8634e-02<\/p>","<p><b>from 781-G/T(genotype_SNP) to Sex_hormone-binding_globulin(proteomics)<\/b><br> beta = -2.6113e+00 FDR-adjusted p-value = 2.8634e-02<\/p>","<p><b>from 1217-A/T(genotype_SNP) to Prenylcysteine_oxidase_1(proteomics)<\/b><br> beta = 1.9744e+00 FDR-adjusted p-value = 2.8898e-02<\/p>","<p><b>from 4976-G/T(genotype_SNP) to Sushi__von_Willebrand_factor_type_A__EGF_and_pentraxin_domain-containing_protein_1(proteomics)<\/b><br> beta = 4.1617e+00 FDR-adjusted p-value = 2.9032e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Keratin__type_I_cytoskeletal_17(proteomics)<\/b><br> beta = 3.9083e+00 FDR-adjusted p-value = 2.9432e-02<\/p>","<p><b>from 708-G/A(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -1.2506e+00 FDR-adjusted p-value = 2.9540e-02<\/p>","<p><b>from 708-G/A(genotype_SNP) to Plasma_protease_C1_inhibitor(proteomics)<\/b><br> beta = -2.0309e+00 FDR-adjusted p-value = 3.1056e-02<\/p>","<p><b>from 717-G/A(genotype_SNP) to Plasma_protease_C1_inhibitor(proteomics)<\/b><br> beta = -1.6931e+00 FDR-adjusted p-value = 3.1056e-02<\/p>","<p><b>from 824-C/A(genotype_SNP) to Plasma_protease_C1_inhibitor(proteomics)<\/b><br> beta = -1.6931e+00 FDR-adjusted p-value = 3.1056e-02<\/p>","<p><b>from 357-C/T(genotype_SNP) to BPI_fold-containing_family_B_member_1(proteomics)<\/b><br> beta = 5.4226e-01 FDR-adjusted p-value = 3.1338e-02<\/p>","<p><b>from 4734-G/T(genotype_SNP) to Haptoglobin(proteomics)<\/b><br> beta = -4.5763e+00 FDR-adjusted p-value = 3.1842e-02<\/p>","<p><b>from 612-G/C(genotype_SNP) to Haptoglobin(proteomics)<\/b><br> beta = -4.5763e+00 FDR-adjusted p-value = 3.1842e-02<\/p>","<p><b>from 1028-C/G(genotype_SNP) to Haptoglobin(proteomics)<\/b><br> beta = -4.5763e+00 FDR-adjusted p-value = 3.1842e-02<\/p>","<p><b>from 4503-A/G(genotype_SNP) to Haptoglobin(proteomics)<\/b><br> beta = 4.5763e+00 FDR-adjusted p-value = 3.1842e-02<\/p>","<p><b>from 421-T/A(genotype_SNP) to Haptoglobin(proteomics)<\/b><br> beta = -4.5763e+00 FDR-adjusted p-value = 3.1842e-02<\/p>","<p><b>from 4577-G/A(genotype_SNP) to Haptoglobin(proteomics)<\/b><br> beta = 4.5763e+00 FDR-adjusted p-value = 3.1842e-02<\/p>","<p><b>from 681-A/G(genotype_SNP) to Haptoglobin(proteomics)<\/b><br> beta = -4.5763e+00 FDR-adjusted p-value = 3.1842e-02<\/p>","<p><b>from 4639-A/T(genotype_SNP) to Zinc-alpha-2-glycoprotein(proteomics)<\/b><br> beta = -2.0056e+00 FDR-adjusted p-value = 3.4523e-02<\/p>","<p><b>from 4640-G/C(genotype_SNP) to Zinc-alpha-2-glycoprotein(proteomics)<\/b><br> beta = -2.0056e+00 FDR-adjusted p-value = 3.4523e-02<\/p>","<p><b>from 4527-A/T(genotype_SNP) to Zinc-alpha-2-glycoprotein(proteomics)<\/b><br> beta = -2.3333e+00 FDR-adjusted p-value = 3.4523e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Zinc-alpha-2-glycoprotein(proteomics)<\/b><br> beta = 2.8923e+00 FDR-adjusted p-value = 3.4523e-02<\/p>","<p><b>from 4426-G/T(genotype_SNP) to Serum_amyloid_A-4_protein(proteomics)<\/b><br> beta = 3.9321e+00 FDR-adjusted p-value = 3.6847e-02<\/p>","<p><b>from 5013-G/T(genotype_SNP) to Fibrinogen_alpha_chain(proteomics)<\/b><br> beta = 6.6136e+00 FDR-adjusted p-value = 3.7688e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Junction_plakoglobin(proteomics)<\/b><br> beta = 3.7758e+00 FDR-adjusted p-value = 3.8397e-02<\/p>","<p><b>from 421-T/A(genotype_SNP) to Vitamin_D-binding_protein(proteomics)<\/b><br> beta = -2.1986e+00 FDR-adjusted p-value = 3.9068e-02<\/p>","<p><b>from 4503-A/G(genotype_SNP) to Vitamin_D-binding_protein(proteomics)<\/b><br> beta = 2.1986e+00 FDR-adjusted p-value = 3.9068e-02<\/p>","<p><b>from 681-A/G(genotype_SNP) to Vitamin_D-binding_protein(proteomics)<\/b><br> beta = -2.1986e+00 FDR-adjusted p-value = 3.9068e-02<\/p>","<p><b>from 612-G/C(genotype_SNP) to Vitamin_D-binding_protein(proteomics)<\/b><br> beta = -2.1986e+00 FDR-adjusted p-value = 3.9068e-02<\/p>","<p><b>from 4734-G/T(genotype_SNP) to Vitamin_D-binding_protein(proteomics)<\/b><br> beta = -2.1986e+00 FDR-adjusted p-value = 3.9068e-02<\/p>","<p><b>from 1028-C/G(genotype_SNP) to Vitamin_D-binding_protein(proteomics)<\/b><br> beta = -2.1986e+00 FDR-adjusted p-value = 3.9068e-02<\/p>","<p><b>from 4577-G/A(genotype_SNP) to Vitamin_D-binding_protein(proteomics)<\/b><br> beta = 2.1986e+00 FDR-adjusted p-value = 3.9068e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Keratin__type_II_cytoskeletal_5(proteomics)<\/b><br> beta = 1.7473e+01 FDR-adjusted p-value = 3.9203e-02<\/p>","<p><b>from 184-A/G(genotype_SNP) to Immunoglobulin_heavy_constant_alpha_2(proteomics)<\/b><br> beta = 1.5173e+00 FDR-adjusted p-value = 3.9359e-02<\/p>","<p><b>from 410-T/A(genotype_SNP) to Angiotensinogen(proteomics)<\/b><br> beta = -1.5859e+00 FDR-adjusted p-value = 3.9429e-02<\/p>","<p><b>from 406-C/T(genotype_SNP) to Angiotensinogen(proteomics)<\/b><br> beta = 4.2226e+00 FDR-adjusted p-value = 3.9429e-02<\/p>","<p><b>from 781-G/T(genotype_SNP) to Angiotensinogen(proteomics)<\/b><br> beta = -4.2153e+00 FDR-adjusted p-value = 3.9429e-02<\/p>","<p><b>from 837-T/C(genotype_SNP) to Angiotensinogen(proteomics)<\/b><br> beta = 4.2226e+00 FDR-adjusted p-value = 3.9429e-02<\/p>","<p><b>from 398-G/T(genotype_SNP) to Angiotensinogen(proteomics)<\/b><br> beta = 4.2226e+00 FDR-adjusted p-value = 3.9429e-02<\/p>","<p><b>from 4436-A/G(genotype_SNP) to Angiotensinogen(proteomics)<\/b><br> beta = -1.7656e+00 FDR-adjusted p-value = 3.9429e-02<\/p>","<p><b>from 4433-C/T(genotype_SNP) to Angiotensinogen(proteomics)<\/b><br> beta = -1.5643e+00 FDR-adjusted p-value = 3.9429e-02<\/p>","<p><b>from 4355-T/C(genotype_SNP) to Angiotensinogen(proteomics)<\/b><br> beta = 4.2226e+00 FDR-adjusted p-value = 3.9429e-02<\/p>","<p><b>from 1144-A/G(genotype_SNP) to Angiotensinogen(proteomics)<\/b><br> beta = 4.2153e+00 FDR-adjusted p-value = 3.9429e-02<\/p>","<p><b>from 411-G/T(genotype_SNP) to Angiotensinogen(proteomics)<\/b><br> beta = 4.2226e+00 FDR-adjusted p-value = 3.9429e-02<\/p>","<p><b>from 4893-C/A(genotype_SNP) to Angiotensinogen(proteomics)<\/b><br> beta = -4.2226e+00 FDR-adjusted p-value = 3.9429e-02<\/p>","<p><b>from 260-G/A(genotype_SNP) to Angiotensinogen(proteomics)<\/b><br> beta = 4.2153e+00 FDR-adjusted p-value = 3.9429e-02<\/p>","<p><b>from 314-C/T(genotype_SNP) to Angiotensinogen(proteomics)<\/b><br> beta = 4.2226e+00 FDR-adjusted p-value = 3.9429e-02<\/p>","<p><b>from 393-G/A(genotype_SNP) to Angiotensinogen(proteomics)<\/b><br> beta = 4.2226e+00 FDR-adjusted p-value = 3.9429e-02<\/p>","<p><b>from 820-T/G(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -8.0495e-01 FDR-adjusted p-value = 3.9921e-02<\/p>","<p><b>from 818-G/C(genotype_SNP) to Plasminogen(proteomics)<\/b><br> beta = -8.0495e-01 FDR-adjusted p-value = 3.9921e-02<\/p>","<p><b>from 835-T/C(genotype_SNP) to Protein_S100-A7(proteomics)<\/b><br> beta = 8.4424e-01 FDR-adjusted p-value = 4.0511e-02<\/p>","<p><b>from 4279-C/A(genotype_SNP) to Angiotensinogen(proteomics)<\/b><br> beta = 4.1872e+00 FDR-adjusted p-value = 4.0903e-02<\/p>","<p><b>from 342-G/T(genotype_SNP) to Monocyte_differentiation_antigen_CD14(proteomics)<\/b><br> beta = 1.4706e+00 FDR-adjusted p-value = 4.2098e-02<\/p>","<p><b>from 353-G/C(genotype_SNP) to Monocyte_differentiation_antigen_CD14(proteomics)<\/b><br> beta = 1.5896e+00 FDR-adjusted p-value = 4.2098e-02<\/p>","<p><b>from 708-G/A(genotype_SNP) to Immunoglobulin_heavy_variable_5-51(proteomics)<\/b><br> beta = -2.0638e+00 FDR-adjusted p-value = 4.2312e-02<\/p>","<p><b>from 387-G/C(genotype_SNP) to Alpha-2-HS-glycoprotein(proteomics)<\/b><br> beta = 2.9716e+00 FDR-adjusted p-value = 4.5253e-02<\/p>","<p><b>from 4788-C/G(genotype_SNP) to Complement_C4-A(proteomics)<\/b><br> beta = -1.6544e+00 FDR-adjusted p-value = 4.7372e-02<\/p>","<p><b>from 4755-T/A(genotype_SNP) to Extracellular_matrix_protein_1(proteomics)<\/b><br> beta = -5.4682e+00 FDR-adjusted p-value = 4.9659e-02<\/p>"]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot"},"manipulation":{"enabled":false},"groups":{"genotype_SNP":{"color":"#0000FF"},"lipidomics":{"color":"#FF0000"},"proteomics":{"color":"#00FF00"},"genotype_VNTR":{"color":"#000033"},"useDefaultGroups":true,"phenotype":{"color":"#FF00B6"}},"physics":{"solver":"forceAtlas2Based","forceAtlas2Based":{"gravitationalConstant":-100}}},"groups":["genotype_SNP","lipidomics","proteomics","genotype_VNTR","phenotype"],"width":"100%","height":null,"idselection":{"enabled":false},"byselection":{"enabled":false},"main":null,"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)","legend":{"width":0.1,"useGroups":true,"position":"right","ncol":1,"stepX":100,"stepY":100,"zoom":true,"main":{"text":"Group","style":"font-family:Georgia, Times New Roman, Times, serif;font-weight:bold;font-size:14px;text-align:center;"}}},"evals":[],"jsHooks":[]}</script>
</div>
<div class="section level3">
<h3 id="visualization-of-restricted-cubic-spline-model">Visualization of restricted cubic spline model<a class="anchor" aria-label="anchor" href="#visualization-of-restricted-cubic-spline-model"></a>
</h3>
<div class="sourceCode" id="cb54"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">df</span><span class="op">)</span></span>
<span><span class="co">#&gt;     factor1      factor2      age      bmi sex</span></span>
<span><span class="co">#&gt; 1 1.0685479  0.680923388 57.94493 20.83511   0</span></span>
<span><span class="co">#&gt; 2 0.9717651  1.013496371 58.20817 25.20727   0</span></span>
<span><span class="co">#&gt; 3 1.0181564 -0.009105223 53.33630 26.60876   0</span></span>
<span><span class="co">#&gt; 4 1.0316431  0.560053416 52.09588 29.94023   0</span></span>
<span><span class="co">#&gt; 5 1.0202134 -1.478541110 55.37792 30.23393   1</span></span>
<span><span class="co">#&gt; 6 0.9946938  0.952096837 51.99340 19.04435   1</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">y</span><span class="op">)</span></span>
<span><span class="co">#&gt; [1] 1 0 0 1 0 1</span></span></code></pre></div>
<div class="sourceCode" id="cb55"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">dd</span> <span class="op">=</span> <span class="fu">rms</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/rms/man/datadist.html" class="external-link">datadist</a></span><span class="op">(</span><span class="va">df</span><span class="op">)</span></span>
<span><span class="va">lrm_fit</span> <span class="op">&lt;-</span> <span class="fu">rms</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/rms/man/lrm.html" class="external-link">lrm</a></span><span class="op">(</span></span>
<span>  <span class="va">y</span> <span class="op">~</span> <span class="fu">rms</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/rms/man/rms.trans.html" class="external-link">rcs</a></span><span class="op">(</span><span class="va">factor1</span>,<span class="fl">3</span><span class="op">)</span><span class="op">*</span><span class="fu">rms</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/rms/man/rms.trans.html" class="external-link">rcs</a></span><span class="op">(</span><span class="va">factor2</span>,<span class="fl">3</span><span class="op">)</span><span class="op">+</span> <span class="va">bmi</span><span class="op">+</span><span class="va">sex</span><span class="op">+</span><span class="va">age</span>,</span>
<span>  data <span class="op">=</span> <span class="va">df</span><span class="op">)</span></span></code></pre></div>
<div class="sourceCode" id="cb56"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu">mirrorstoolkit</span><span class="fu">::</span><span class="fu"><a href="../reference/rms_illustration_3d.html">rms_illustration_3d</a></span><span class="op">(</span></span>
<span>  fit <span class="op">=</span> <span class="va">lrm_fit</span>, </span>
<span>  datadist <span class="op">=</span> <span class="va">dd</span>, </span>
<span>  variable <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"factor1"</span>, <span class="st">"factor2"</span><span class="op">)</span>,</span>
<span>  target_name <span class="op">=</span> <span class="st">"target"</span>,</span>
<span>  density <span class="op">=</span> <span class="fl">30</span><span class="op">)</span></span>
<span><span class="co">#&gt; Warning in formula.character(object, env = baseenv()): Using formula(x) is deprecated when x is a character vector of length &gt; 1.</span></span>
<span><span class="co">#&gt;   Consider formula(paste(x, collapse = " ")) instead.</span></span></code></pre></div>
<div class="plotly html-widget html-fill-item" id="htmlwidget-e5c8c404fe174e4c81bd" style="width:672px;height:432px;"></div>
<script type="application/json" data-for="htmlwidget-e5c8c404fe174e4c81bd">{"x":{"visdat":{"c7d6c58e5c1":["function () ","plotlyVisDat"]},"cur_data":"c7d6c58e5c1","attrs":{"c7d6c58e5c1":{"x":{},"y":{},"z":[[-0.28459620487309267,-0.27322850311029789,-0.26186080134750311,-0.25049309958470845,-0.23912539782191367,-0.22775769605911889,-0.21638999429632411,-0.20502229253352933,-0.19365459077073455,-0.18228688900793977,-0.17088108894259146,-0.15846456770068473,-0.14309661524184036,-0.1227917080842944,-0.095564322746282127,-0.060060637842821452,-0.017579047548868743,0.029886107788708371,0.080340485032271092,0.13183157711909355,0.1833639320796997,0.23489628704030607,0.28642864200091267,0.33796099696151882,0.38949335192212542,0.44102570688273091,0.49255806184333772,0.54409041680394388,0.59562277176454936,0.64715512672515618],[-0.27014104994338339,-0.25882570270728,-0.24751035547117661,-0.23619500823507333,-0.22487966099896994,-0.21356431376286644,-0.20224896652676305,-0.19093361929065966,-0.17961827205455616,-0.16830292481845288,-0.15695380593804598,-0.14470874908201148,-0.12984744219477495,-0.11060984904669069,-0.085235933408112752,-0.052525621490983276,-0.013629854500114558,0.029683515762704338,0.075646635021848874,0.12252873393583363,0.16944740967878336,0.21636608542173341,0.26328476116468325,0.3102034369076333,0.35712211265058336,0.40404078839353275,0.45095946413648258,0.49787813987943264,0.54479681562238158,0.59171549136533208],[-0.25568589501367411,-0.244422902304262,-0.23315990959485,-0.221896916885438,-0.210633924176026,-0.199370931466614,-0.188107938757202,-0.17684494604779,-0.16558195333837789,-0.15431896062896588,-0.14302652293350049,-0.13095293046333822,-0.11659826914770954,-0.098427990009087085,-0.074907544069943377,-0.044990605139145101,-0.0096806614513603728,0.029480923736700193,0.070952785011426545,0.1132258907525735,0.15553088727786679,0.19783588380316053,0.24014088032845404,0.28244587685374756,0.32475087337904129,0.36705586990433392,0.40936086642962788,0.45166586295492139,0.49397085948021402,0.53627585600550776],[-0.24123074008396483,-0.23002010190124422,-0.2188094637185235,-0.207598825535803,-0.19638818735308239,-0.18517754917036167,-0.17396691098764094,-0.16275627280492033,-0.15154563462219961,-0.140334996439479,-0.129099239928955,-0.11719711184466497,-0.10334909610064413,-0.086246130971483481,-0.064579154731773891,-0.037455588787306926,-0.005731468402606299,0.02927833171069627,0.066258935001004327,0.10392304756931348,0.14161436487695045,0.17930568218458764,0.21699699949222484,0.25468831679986181,0.29237963410749901,0.33007095141513554,0.36776226872277273,0.40545358603040993,0.44314490333804624,0.48083622064568365],[-0.22677558515425567,-0.21561730149822644,-0.20445901784219722,-0.19330073418616789,-0.18214245053013867,-0.17098416687410922,-0.15982588321808,-0.14866759956205067,-0.13750931590602133,-0.126351032249992,-0.11517195692440962,-0.10344129322599172,-0.090099923053578612,-0.074064271933879766,-0.054250765393604516,-0.029920572435468751,-0.0017822753538521141,0.029075739684692237,0.061565084990582108,0.094620204386053564,0.127697842476034,0.16077548056601476,0.19385311865599542,0.22693075674597629,0.26000839483595695,0.29308603292593716,0.32616367101591803,0.35924130910589847,0.39231894719587868,0.42539658528585933],[-0.2123204302245465,-0.20121450109520855,-0.19010857196587072,-0.17900264283653278,-0.16789671370719483,-0.15679078457785678,-0.14568485544851895,-0.134578926319181,-0.12347299718984295,-0.11236706806050512,-0.10124467391986414,-0.089685474607318461,-0.076850750006513091,-0.061882412896276273,-0.043922376055435031,-0.022385556083630576,0.0021669176949020708,0.028873147658688203,0.05687123498015989,0.08531736120279354,0.11378132007511754,0.14224527894744188,0.17070923781976621,0.19917319669209033,0.22763715556441488,0.25610111443673855,0.28456507330906311,0.31302903218138722,0.34149299105371089,0.36995694992603523],[-0.19786527529483733,-0.18681170069219077,-0.17575812608954422,-0.16470455148689767,-0.15365097688425111,-0.14259740228160445,-0.13154382767895789,-0.12049025307631134,-0.10943667847366467,-0.098383103871018118,-0.087317390915318649,-0.075929655988645206,-0.063601576959447681,-0.049700553858672558,-0.033593986717265545,-0.014850539731792511,0.0061161107436562556,0.028670555632684058,0.052177384969737561,0.076014518019533406,0.0998647976742012,0.12371507732886899,0.14756535698353679,0.17141563663820458,0.1952659162928726,0.21911619594753995,0.24296647560220797,0.26681675525687598,0.29066703491154333,0.31451731456621113],[-0.18341012036512794,-0.17240890028917277,-0.16140768021321761,-0.15040646013726244,-0.13940524006130717,-0.128404019985352,-0.11740279990939684,-0.10640157983344167,-0.095400359757486286,-0.084399139681531121,-0.073390107910773161,-0.062173837369971841,-0.05035240391238216,-0.037518694821068843,-0.02326559737909617,-0.0073155233799542252,0.010065303792410663,0.028467963606680136,0.047483534959315454,0.066711674836273382,0.085948275273284636,0.10518487571029611,0.12442147614730759,0.14365807658431906,0.16289467702133054,0.18213127745834157,0.20136787789535304,0.2206044783323643,0.23984107876937555,0.25907767920638702],[-0.16895496543541877,-0.15800609988615488,-0.14705723433689111,-0.13610836878762733,-0.12515950323836356,-0.11421063768909956,-0.10326177213983567,-0.092312906590571897,-0.081364041041308011,-0.070415175492044235,-0.059462824906227674,-0.048418018751298697,-0.037103230865316861,-0.02533683578346535,-0.012937208040926684,0.00021949297188383898,0.014014496841164625,0.028265371580676213,0.042789684948893014,0.057408831653013581,0.072031752872368404,0.086654674091723449,0.10127759531107849,0.11590051653043332,0.13052343774978836,0.14514635896914296,0.15976928018849812,0.17439220140785305,0.18901512262720777,0.20363804384656292],[-0.15449981050570949,-0.14360329948313699,-0.13270678846056461,-0.12181027743799211,-0.11091376641541972,-0.10001725539284723,-0.089120744370274729,-0.078224233347702343,-0.067327722325129624,-0.056431211302557238,-0.045535541901682297,-0.03466220013262522,-0.02385405781825134,-0.013154976745861635,-0.0026088187027574206,0.0077545093237221252,0.017963689889918921,0.028062779554672068,0.038095834938471018,0.048105988469753336,0.058115230471451951,0.068124472473150455,0.07813371447484907,0.088142956476547574,0.0981521984782463,0.10816144047994447,0.1181706824816432,0.12817992448334159,0.1381891664850402,0.14819840848673882],[-0.14004215908161244,-0.12919826444848015,-0.11835436981534797,-0.10751047518221568,-0.096666580549083392,-0.085822685915950991,-0.074978791282818813,-0.064134896649686635,-0.053291002016554234,-0.042447107383421945,-0.031608380197121755,-0.020906741741603341,-0.010605419386017889,-0.00097371875130769503,0.0077190545415856127,0.015289275513115652,0.021913051297106745,0.027860883386195745,0.033403273651544607,0.038805049546342008,0.044201228769053302,0.049597407991764819,0.054993587214476336,0.060389766437187742,0.065785945659899148,0.071182124882610331,0.076578304105321959,0.081974483328033365,0.087370662550744993,0.092766841773456066],[-0.12510987537681673,-0.11436838234967384,-0.10362688932253095,-0.092885396295387945,-0.082143903268245055,-0.071402410241102054,-0.060660917213959165,-0.049919424186816275,-0.039177931159673274,-0.028436438132530384,-0.017704279985855886,-0.0072197696538556233,0.0025415783382356505,0.011093269235338488,0.017948808282373885,0.022776480942013455,0.025894420926572015,0.027791283307372017,0.028955723839533043,0.02986614756535011,0.030766461024600233,0.031666774483850357,0.032567087943100814,0.033467401402350938,0.034367714861601284,0.035268028320851408,0.036168341780101643,0.037068655239351878,0.037968968698602445,0.038869282157852458],[-0.1082108623581014,-0.097778065470144893,-0.087345268582188385,-0.076912471694231987,-0.066479674806275479,-0.05604687791831886,-0.045614081030362463,-0.035181284142405955,-0.024748487254449336,-0.014315690366492828,-0.0038957394672808343,0.0061834163836628209,0.015267408417555806,0.022686757665815738,0.027771985159860901,0.030066609295324898,0.030008422725403161,0.028269877289967993,0.025523425769884533,0.022427414654886513,0.019317490520759928,0.016207566386633454,0.013097642252507091,0.0099877181183803954,0.0068777939842540325,0.0037678698501273367,0.00065794571600108487,-0.0024519784181255,-0.0055619025522516408,-0.0086718266863784477],[-0.087719233495244908,-0.077971970068865026,-0.068224706642485144,-0.058477443216105263,-0.04873017978972527,-0.038982916363345277,-0.029235652936965284,-0.019488389510585402,-0.0097411260842054093,6.1373421744725931e-06,0.0097382425782027715,0.019068211683226255,0.027223893400161847,0.033415306525988786,0.036852469857686976,0.036996737815787717,0.034364703128479834,0.029749854969232992,0.023945683621883318,0.017729034030589452,0.011495967159434017,0.0052629002882785825,-0.00097016658287651936,-0.0072032334540320653,-0.013436300325187389,-0.019669367196342824,-0.025902434067497926,-0.032135500938653472,-0.038368567809808352,-0.044601634680964009],[-0.062009102258025806,-0.053494752404805612,-0.044980402551585308,-0.036466052698365115,-0.027951702845144921,-0.019437352991924617,-0.010923003138704312,-0.0024086532854841192,0.0061056965677360742,0.014620046420956379,0.023118667370193746,0.031200011557109053,0.038062855834273268,0.042887475801722585,0.044854147059493088,0.043403943746140428,0.039072908570682463,0.032684405980416775,0.025061801574812415,0.017011188908097385,0.0089435408425243557,0.00087589277695154788,-0.0071917552886212599,-0.01525940335419429,-0.023327051419767098,-0.031394699485339905,-0.039462347550912824,-0.047529995616485854,-0.055597643682058218,-0.063665291747631358],[-0.029500405802327823,-0.022932085876526176,-0.016363765950724307,-0.0097954460249225495,-0.0032271260991206807,0.003341193826680966,0.0099095137524829457,0.016477833678284592,0.023046153604086794,0.02961447352988833,0.036168762615348626,0.042350823372836688,0.047445931237196648,0.05072285775204699,0.05145037446100631,0.049129896114064042,0.044239595231992834,0.037513947345670373,0.029687429013759759,0.021479109408025776,0.013255593513866848,0.0050320776197079198,-0.0031914382744508973,-0.011414954168609714,-0.019638470062768532,-0.02786198595692746,-0.036085501851086055,-0.044309017745245316,-0.0525325336394038,-0.060756049533562728],[0.0099493511004579149,0.013843578108835075,0.017737805117212457,0.021632032125589618,0.025526259133967,0.029420486142344271,0.033314713150721542,0.037208940159098813,0.041103167167476085,0.044997394175853578,0.048881604737474382,0.052500086010066838,0.055342604794603401,0.056887145964266805,0.056611694392239009,0.054160316115280938,0.04987437269679007,0.044278197310688272,0.037896123864618381,0.031241487119575528,0.024576001928523938,0.017910516737472237,0.01124503154642098,0.0045795463553691684,-0.0020859388356821995,-0.0087514240267339005,-0.015416909217785379,-0.022082394408836969,-0.028747879599888226,-0.035413364790940038],[0.054792846550935637,0.055447219625357813,0.056101592699780434,0.056755965774202721,0.057410338848625009,0.058064711923047407,0.058719084997469806,0.059373458071891871,0.060027831146314381,0.060682204220737002,0.061332375208750678,0.061871067799197155,0.062084229639693067,0.061752865639195798,0.060657980706660619,0.058650253899685367,0.055872892766891002,0.052545864863356662,0.048889138051971037,0.045118065848263944,0.041342442518929379,0.037566819189594924,0.033791195860260803,0.030015572530926238,0.026239949201592228,0.022464325872257995,0.018688702542923541,0.014913079213589198,0.011137455884254965,0.0073618325549208441],[0.10338607135467048,0.10040727319363119,0.097428475032591111,0.094449676871551813,0.091470878710511738,0.088492080549472441,0.085513282388432588,0.082534484227392846,0.079555686066353548,0.076576887905313473,0.073600953355880128,0.070700988408703314,0.068022864124253068,0.065715819909676121,0.063929095172121864,0.062764448214995383,0.062124286852310751,0.061858708968201559,0.061817812237037528,0.061854838889812247,0.061894967015260205,0.061935095140708385,0.061975223266156454,0.062015351391604523,0.062055479517052592,0.062095607642500661,0.06213573576794873,0.06217586389339691,0.062215992018844979,0.062256120144292937],[0.15408501631722804,0.14725217333424279,0.1404193303512572,0.13358648736827228,0.12675364438528647,0.11992080140230132,0.11308795841931596,0.10625511543633059,0.09942227245334545,0.092589429470360085,0.085767218505564458,0.079227067507061877,0.073510566600069716,0.069171811908551484,0.066764899556472357,0.066667637808930369,0.068517686363065544,0.071758488589747982,0.075833487081037432,0.080197799539942882,0.084573627147693453,0.088949454755444357,0.093325282363195039,0.097701109970945943,0.10207693757869651,0.10645276518644708,0.11082859279419832,0.115204420401949,0.11958024800969935,0.12395607561745048],[0.20534758331940661,0.19460157596775385,0.18385556861610108,0.17310956126444854,0.16236355391279533,0.15161754656114301,0.14087153920949003,0.13012553185783726,0.1193795245061845,0.10863351715453173,0.097906098316291268,0.087671819667688666,0.078877571547567582,0.072492109186389264,0.06948418781461474,0.070514349379927288,0.074949095381934172,0.081815368760322138,0.090140111093135245,0.098970676165456517,0.10782137377469792,0.1166720713839392,0.12552276899318093,0.13437346660242178,0.14322416421166428,0.15207486182090546,0.16092555943014664,0.16977625703938848,0.17862695464862932,0.18747765225787105],[0.25661621979242766,0.2419564114320546,0.22729660307168131,0.2126367947113087,0.19797698635093586,0.18331717799056324,0.16865736963018996,0.1539975612698169,0.13933775290944428,0.124677944549071,0.11004468322280048,0.096115696043761023,0.084243276740838535,0.075810945209884628,0.072202221346751805,0.074360452756621287,0.081380913713401393,0.091873940697957401,0.1044498682465379,0.11774818242269858,0.13107524871756449,0.14440231501243017,0.15772938130729708,0.17105644760216343,0.18438351389702923,0.19771058019189514,0.21103764648676127,0.22436471278162806,0.23769177907649286,0.25101884537135943],[0.30788485626544915,0.28931124689635579,0.27073763752726288,0.25216402815816952,0.23359041878907683,0.21501680941998302,0.19644320005088989,0.17786959068179697,0.15929598131270362,0.14072237194361048,0.12218326812931013,0.10455957241983371,0.089608981934109599,0.079129781233380991,0.074920254878889203,0.078206556133315619,0.087812732044868169,0.10193251263559322,0.11875962539994001,0.13652568867994042,0.15432912366043117,0.17213255864092225,0.18993599362141289,0.20773942860190375,0.22554286358239561,0.24334629856288614,0.26114973354337689,0.27895316852386787,0.29675660350435795,0.31456003848484915],[0.35915349273847041,0.33666608236065698,0.31417867198284311,0.29169126160503012,0.26920385122721713,0.24671644084940325,0.22422903047159026,0.20174162009377683,0.1792542097159634,0.15676679933815041,0.13432185303581956,0.1130034487959054,0.094974687127380553,0.082448617256876577,0.07763828841102649,0.082052659510009285,0.094244550376336167,0.11199108457322959,0.13306938255334244,0.15530319493718192,0.1775829986032974,0.199862802269413,0.22214260593552837,0.24442240960164441,0.26670221326775978,0.28898201693387582,0.31126182059999141,0.33354162426610678,0.35582142793222127,0.37810123159833753],[0.41042212921149124,0.38402091782495773,0.35761970643842422,0.33121849505189072,0.30481728366535721,0.2784160722788237,0.25201486089229064,0.22561364950575669,0.19921243811922318,0.17281122673268923,0.14646043794232888,0.12144732517197765,0.10034039232065151,0.08576745328037183,0.080356321943163556,0.085898762886703617,0.1006763687078035,0.1220496565108653,0.14737913970674577,0.17408070119442398,0.20083687354616397,0.22759304589790419,0.25434921824964507,0.28110539060138506,0.30786156295312594,0.33461773530486594,0.36137390765660637,0.38813008000834659,0.41488625236008569,0.44164242471182702],[0.46169076568451251,0.43137575328925848,0.40106074089400445,0.37074572849875176,0.34043071610349729,0.31011570370824371,0.27980069131299012,0.24948567891773654,0.21917066652248296,0.18885565412722849,0.15859902284883898,0.12989120154805001,0.10570609751392102,0.08908628930386786,0.083074355475300177,0.089744866263397838,0.10710818703926972,0.13210822844850079,0.16168889686014787,0.19285820745166526,0.22409074848902988,0.25532328952639538,0.28655583056376022,0.3177883716011255,0.34902091263849078,0.38025345367585561,0.41148599471322045,0.44271853575058573,0.47395107678794879,0.50518361782531451],[0.51295940215753377,0.47873058875355967,0.44450177534958557,0.41027296194561191,0.37604414854163826,0.34181533513766371,0.30758652173368961,0.2733577083297164,0.2391288949257423,0.20490008152176864,0.17073760775534819,0.1383350779241227,0.11107180270719286,0.092405125327363113,0.085792389007437353,0.093590969640092503,0.11354000537073661,0.14216680038613594,0.17599865401354975,0.21163571370890766,0.24734462343189667,0.28305353315488679,0.31876244287787669,0.35447135260086615,0.39018026232385672,0.42588917204684551,0.46159808176983563,0.49730699149282553,0.53301590121581321,0.56872481093880445],[0.56422803863055548,0.52608542421786131,0.48794280980516713,0.4498001953924734,0.41165758097977923,0.37351496656708505,0.33537235215439087,0.29722973774169537,0.25908712332900297,0.22094450891630746,0.18287619266185828,0.14677895430019461,0.11643750790046425,0.095723961350859255,0.088510422539575417,0.097437073016786946,0.11997182370220461,0.15222537232377242,0.19030841116695352,0.23041321996615005,0.27059849837476346,0.31078377678337932,0.35096905519199206,0.39115433360060836,0.43133961200922244,0.47152489041783718,0.51171016882645237,0.55189544723506578,0.59208072564367853,0.63226600405229438],[0.61549667510357764,0.57344025968216294,0.53138384426074825,0.48932742883933356,0.44727101341791975,0.4052145979965055,0.36315818257509169,0.32110176715367611,0.2790453517322623,0.23698893631084716,0.19501477756836649,0.15522283067626785,0.12180321309373598,0.099042797374354952,0.09122845607171215,0.10128317639348183,0.12640364203367205,0.16228394426140846,0.20461816832035573,0.24919072622339244,0.2938523733176307,0.33851402041187117,0.38317566750610921,0.42783731460034968,0.47249896169458949,0.51716060878882752,0.56182225588306822,0.60648390297730781,0.65114555007154384,0.69580719716578521],[0.66676531157659757,0.62079509514646236,0.57482487871632892,0.52885466228619504,0.48288444585606027,0.43691422942592462,0.39094401299579118,0.34497379656565641,0.29900358013552164,0.25303336370538732,0.20715336247487726,0.16366670705233965,0.12716891828700516,0.10236163339784965,0.093946489603848549,0.1051292797701745,0.13283546036513894,0.17234251619904339,0.21892792547375872,0.26796823248063351,0.3171062482604966,0.36624426404036148,0.41538227982022569,0.46452029560008945,0.51365831137995344,0.56279632715981653,0.61193434293968119,0.66107235871954628,0.71021037449940694,0.75934839027927248]],"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"surface"}},"layout":{"margin":{"b":40,"l":60,"t":25,"r":10},"scene":{"xaxis":{"title":"factor1"},"yaxis":{"title":"factor2"},"zaxis":{"title":"target"}},"hovermode":"closest","showlegend":false,"legend":{"yanchor":"top","y":0.5}},"source":"A","config":{"modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"data":[{"colorbar":{"title":"","ticklen":2,"len":0.5,"lenmode":"fraction","y":1,"yanchor":"top"},"colorscale":[["0","rgba(68,1,84,1)"],["0.0416666666666667","rgba(70,19,97,1)"],["0.0833333333333333","rgba(72,32,111,1)"],["0.125","rgba(71,45,122,1)"],["0.166666666666667","rgba(68,58,128,1)"],["0.208333333333333","rgba(64,70,135,1)"],["0.25","rgba(60,82,138,1)"],["0.291666666666667","rgba(56,93,140,1)"],["0.333333333333333","rgba(49,104,142,1)"],["0.375","rgba(46,114,142,1)"],["0.416666666666667","rgba(42,123,142,1)"],["0.458333333333333","rgba(38,133,141,1)"],["0.5","rgba(37,144,140,1)"],["0.541666666666667","rgba(33,154,138,1)"],["0.583333333333333","rgba(39,164,133,1)"],["0.625","rgba(47,174,127,1)"],["0.666666666666667","rgba(53,183,121,1)"],["0.708333333333333","rgba(79,191,110,1)"],["0.75","rgba(98,199,98,1)"],["0.791666666666667","rgba(119,207,85,1)"],["0.833333333333333","rgba(147,214,70,1)"],["0.875","rgba(172,220,52,1)"],["0.916666666666667","rgba(199,225,42,1)"],["0.958333333333333","rgba(226,228,40,1)"],["1","rgba(253,231,37,1)"]],"showscale":true,"x":[0.7978361825269199,0.81226957498091046,0.82670296743490101,0.84113635988889146,0.85556975234288202,0.87000314479687257,0.88443653725086313,0.89886992970485369,0.91330332215884424,0.9277367146128348,0.94217010706682525,0.9566034995208158,0.97103689197480636,0.98547028442879692,0.99990367688278736,1.0143370693367779,1.0287704617907685,1.043203854244759,1.0576372466987496,1.0720706391527401,1.0865040316067307,1.1009374240607213,1.1153708165147118,1.1298042089687024,1.1442376014226929,1.1586709938766833,1.1731043863306738,1.1875377787846644,1.2019711712386549,1.2164045636926455],"y":[-3.9025450879528085,-3.6343082186495845,-3.3660713493463601,-3.0978344800431357,-2.8295976107399117,-2.5613607414366877,-2.2931238721334632,-2.0248870028302388,-1.7566501335270148,-1.4884132642237908,-1.2201763949205664,-0.95193952561734196,-0.68370265631411797,-0.41546578701089398,-0.14722891770766955,0.12100795159555489,0.38924482089877888,0.65748169020200287,0.92571855950522686,1.1939554288084517,1.4621922981116757,1.7304291674148997,1.9986660367181246,2.2669029060213486,2.5351397753245726,2.8033766446277966,3.0716135139310206,3.3398503832342454,3.6080872525374694,3.8763241218406934],"z":[[-0.28459620487309267,-0.27322850311029789,-0.26186080134750311,-0.25049309958470845,-0.23912539782191367,-0.22775769605911889,-0.21638999429632411,-0.20502229253352933,-0.19365459077073455,-0.18228688900793977,-0.17088108894259146,-0.15846456770068473,-0.14309661524184036,-0.1227917080842944,-0.095564322746282127,-0.060060637842821452,-0.017579047548868743,0.029886107788708371,0.080340485032271092,0.13183157711909355,0.1833639320796997,0.23489628704030607,0.28642864200091267,0.33796099696151882,0.38949335192212542,0.44102570688273091,0.49255806184333772,0.54409041680394388,0.59562277176454936,0.64715512672515618],[-0.27014104994338339,-0.25882570270728,-0.24751035547117661,-0.23619500823507333,-0.22487966099896994,-0.21356431376286644,-0.20224896652676305,-0.19093361929065966,-0.17961827205455616,-0.16830292481845288,-0.15695380593804598,-0.14470874908201148,-0.12984744219477495,-0.11060984904669069,-0.085235933408112752,-0.052525621490983276,-0.013629854500114558,0.029683515762704338,0.075646635021848874,0.12252873393583363,0.16944740967878336,0.21636608542173341,0.26328476116468325,0.3102034369076333,0.35712211265058336,0.40404078839353275,0.45095946413648258,0.49787813987943264,0.54479681562238158,0.59171549136533208],[-0.25568589501367411,-0.244422902304262,-0.23315990959485,-0.221896916885438,-0.210633924176026,-0.199370931466614,-0.188107938757202,-0.17684494604779,-0.16558195333837789,-0.15431896062896588,-0.14302652293350049,-0.13095293046333822,-0.11659826914770954,-0.098427990009087085,-0.074907544069943377,-0.044990605139145101,-0.0096806614513603728,0.029480923736700193,0.070952785011426545,0.1132258907525735,0.15553088727786679,0.19783588380316053,0.24014088032845404,0.28244587685374756,0.32475087337904129,0.36705586990433392,0.40936086642962788,0.45166586295492139,0.49397085948021402,0.53627585600550776],[-0.24123074008396483,-0.23002010190124422,-0.2188094637185235,-0.207598825535803,-0.19638818735308239,-0.18517754917036167,-0.17396691098764094,-0.16275627280492033,-0.15154563462219961,-0.140334996439479,-0.129099239928955,-0.11719711184466497,-0.10334909610064413,-0.086246130971483481,-0.064579154731773891,-0.037455588787306926,-0.005731468402606299,0.02927833171069627,0.066258935001004327,0.10392304756931348,0.14161436487695045,0.17930568218458764,0.21699699949222484,0.25468831679986181,0.29237963410749901,0.33007095141513554,0.36776226872277273,0.40545358603040993,0.44314490333804624,0.48083622064568365],[-0.22677558515425567,-0.21561730149822644,-0.20445901784219722,-0.19330073418616789,-0.18214245053013867,-0.17098416687410922,-0.15982588321808,-0.14866759956205067,-0.13750931590602133,-0.126351032249992,-0.11517195692440962,-0.10344129322599172,-0.090099923053578612,-0.074064271933879766,-0.054250765393604516,-0.029920572435468751,-0.0017822753538521141,0.029075739684692237,0.061565084990582108,0.094620204386053564,0.127697842476034,0.16077548056601476,0.19385311865599542,0.22693075674597629,0.26000839483595695,0.29308603292593716,0.32616367101591803,0.35924130910589847,0.39231894719587868,0.42539658528585933],[-0.2123204302245465,-0.20121450109520855,-0.19010857196587072,-0.17900264283653278,-0.16789671370719483,-0.15679078457785678,-0.14568485544851895,-0.134578926319181,-0.12347299718984295,-0.11236706806050512,-0.10124467391986414,-0.089685474607318461,-0.076850750006513091,-0.061882412896276273,-0.043922376055435031,-0.022385556083630576,0.0021669176949020708,0.028873147658688203,0.05687123498015989,0.08531736120279354,0.11378132007511754,0.14224527894744188,0.17070923781976621,0.19917319669209033,0.22763715556441488,0.25610111443673855,0.28456507330906311,0.31302903218138722,0.34149299105371089,0.36995694992603523],[-0.19786527529483733,-0.18681170069219077,-0.17575812608954422,-0.16470455148689767,-0.15365097688425111,-0.14259740228160445,-0.13154382767895789,-0.12049025307631134,-0.10943667847366467,-0.098383103871018118,-0.087317390915318649,-0.075929655988645206,-0.063601576959447681,-0.049700553858672558,-0.033593986717265545,-0.014850539731792511,0.0061161107436562556,0.028670555632684058,0.052177384969737561,0.076014518019533406,0.0998647976742012,0.12371507732886899,0.14756535698353679,0.17141563663820458,0.1952659162928726,0.21911619594753995,0.24296647560220797,0.26681675525687598,0.29066703491154333,0.31451731456621113],[-0.18341012036512794,-0.17240890028917277,-0.16140768021321761,-0.15040646013726244,-0.13940524006130717,-0.128404019985352,-0.11740279990939684,-0.10640157983344167,-0.095400359757486286,-0.084399139681531121,-0.073390107910773161,-0.062173837369971841,-0.05035240391238216,-0.037518694821068843,-0.02326559737909617,-0.0073155233799542252,0.010065303792410663,0.028467963606680136,0.047483534959315454,0.066711674836273382,0.085948275273284636,0.10518487571029611,0.12442147614730759,0.14365807658431906,0.16289467702133054,0.18213127745834157,0.20136787789535304,0.2206044783323643,0.23984107876937555,0.25907767920638702],[-0.16895496543541877,-0.15800609988615488,-0.14705723433689111,-0.13610836878762733,-0.12515950323836356,-0.11421063768909956,-0.10326177213983567,-0.092312906590571897,-0.081364041041308011,-0.070415175492044235,-0.059462824906227674,-0.048418018751298697,-0.037103230865316861,-0.02533683578346535,-0.012937208040926684,0.00021949297188383898,0.014014496841164625,0.028265371580676213,0.042789684948893014,0.057408831653013581,0.072031752872368404,0.086654674091723449,0.10127759531107849,0.11590051653043332,0.13052343774978836,0.14514635896914296,0.15976928018849812,0.17439220140785305,0.18901512262720777,0.20363804384656292],[-0.15449981050570949,-0.14360329948313699,-0.13270678846056461,-0.12181027743799211,-0.11091376641541972,-0.10001725539284723,-0.089120744370274729,-0.078224233347702343,-0.067327722325129624,-0.056431211302557238,-0.045535541901682297,-0.03466220013262522,-0.02385405781825134,-0.013154976745861635,-0.0026088187027574206,0.0077545093237221252,0.017963689889918921,0.028062779554672068,0.038095834938471018,0.048105988469753336,0.058115230471451951,0.068124472473150455,0.07813371447484907,0.088142956476547574,0.0981521984782463,0.10816144047994447,0.1181706824816432,0.12817992448334159,0.1381891664850402,0.14819840848673882],[-0.14004215908161244,-0.12919826444848015,-0.11835436981534797,-0.10751047518221568,-0.096666580549083392,-0.085822685915950991,-0.074978791282818813,-0.064134896649686635,-0.053291002016554234,-0.042447107383421945,-0.031608380197121755,-0.020906741741603341,-0.010605419386017889,-0.00097371875130769503,0.0077190545415856127,0.015289275513115652,0.021913051297106745,0.027860883386195745,0.033403273651544607,0.038805049546342008,0.044201228769053302,0.049597407991764819,0.054993587214476336,0.060389766437187742,0.065785945659899148,0.071182124882610331,0.076578304105321959,0.081974483328033365,0.087370662550744993,0.092766841773456066],[-0.12510987537681673,-0.11436838234967384,-0.10362688932253095,-0.092885396295387945,-0.082143903268245055,-0.071402410241102054,-0.060660917213959165,-0.049919424186816275,-0.039177931159673274,-0.028436438132530384,-0.017704279985855886,-0.0072197696538556233,0.0025415783382356505,0.011093269235338488,0.017948808282373885,0.022776480942013455,0.025894420926572015,0.027791283307372017,0.028955723839533043,0.02986614756535011,0.030766461024600233,0.031666774483850357,0.032567087943100814,0.033467401402350938,0.034367714861601284,0.035268028320851408,0.036168341780101643,0.037068655239351878,0.037968968698602445,0.038869282157852458],[-0.1082108623581014,-0.097778065470144893,-0.087345268582188385,-0.076912471694231987,-0.066479674806275479,-0.05604687791831886,-0.045614081030362463,-0.035181284142405955,-0.024748487254449336,-0.014315690366492828,-0.0038957394672808343,0.0061834163836628209,0.015267408417555806,0.022686757665815738,0.027771985159860901,0.030066609295324898,0.030008422725403161,0.028269877289967993,0.025523425769884533,0.022427414654886513,0.019317490520759928,0.016207566386633454,0.013097642252507091,0.0099877181183803954,0.0068777939842540325,0.0037678698501273367,0.00065794571600108487,-0.0024519784181255,-0.0055619025522516408,-0.0086718266863784477],[-0.087719233495244908,-0.077971970068865026,-0.068224706642485144,-0.058477443216105263,-0.04873017978972527,-0.038982916363345277,-0.029235652936965284,-0.019488389510585402,-0.0097411260842054093,6.1373421744725931e-06,0.0097382425782027715,0.019068211683226255,0.027223893400161847,0.033415306525988786,0.036852469857686976,0.036996737815787717,0.034364703128479834,0.029749854969232992,0.023945683621883318,0.017729034030589452,0.011495967159434017,0.0052629002882785825,-0.00097016658287651936,-0.0072032334540320653,-0.013436300325187389,-0.019669367196342824,-0.025902434067497926,-0.032135500938653472,-0.038368567809808352,-0.044601634680964009],[-0.062009102258025806,-0.053494752404805612,-0.044980402551585308,-0.036466052698365115,-0.027951702845144921,-0.019437352991924617,-0.010923003138704312,-0.0024086532854841192,0.0061056965677360742,0.014620046420956379,0.023118667370193746,0.031200011557109053,0.038062855834273268,0.042887475801722585,0.044854147059493088,0.043403943746140428,0.039072908570682463,0.032684405980416775,0.025061801574812415,0.017011188908097385,0.0089435408425243557,0.00087589277695154788,-0.0071917552886212599,-0.01525940335419429,-0.023327051419767098,-0.031394699485339905,-0.039462347550912824,-0.047529995616485854,-0.055597643682058218,-0.063665291747631358],[-0.029500405802327823,-0.022932085876526176,-0.016363765950724307,-0.0097954460249225495,-0.0032271260991206807,0.003341193826680966,0.0099095137524829457,0.016477833678284592,0.023046153604086794,0.02961447352988833,0.036168762615348626,0.042350823372836688,0.047445931237196648,0.05072285775204699,0.05145037446100631,0.049129896114064042,0.044239595231992834,0.037513947345670373,0.029687429013759759,0.021479109408025776,0.013255593513866848,0.0050320776197079198,-0.0031914382744508973,-0.011414954168609714,-0.019638470062768532,-0.02786198595692746,-0.036085501851086055,-0.044309017745245316,-0.0525325336394038,-0.060756049533562728],[0.0099493511004579149,0.013843578108835075,0.017737805117212457,0.021632032125589618,0.025526259133967,0.029420486142344271,0.033314713150721542,0.037208940159098813,0.041103167167476085,0.044997394175853578,0.048881604737474382,0.052500086010066838,0.055342604794603401,0.056887145964266805,0.056611694392239009,0.054160316115280938,0.04987437269679007,0.044278197310688272,0.037896123864618381,0.031241487119575528,0.024576001928523938,0.017910516737472237,0.01124503154642098,0.0045795463553691684,-0.0020859388356821995,-0.0087514240267339005,-0.015416909217785379,-0.022082394408836969,-0.028747879599888226,-0.035413364790940038],[0.054792846550935637,0.055447219625357813,0.056101592699780434,0.056755965774202721,0.057410338848625009,0.058064711923047407,0.058719084997469806,0.059373458071891871,0.060027831146314381,0.060682204220737002,0.061332375208750678,0.061871067799197155,0.062084229639693067,0.061752865639195798,0.060657980706660619,0.058650253899685367,0.055872892766891002,0.052545864863356662,0.048889138051971037,0.045118065848263944,0.041342442518929379,0.037566819189594924,0.033791195860260803,0.030015572530926238,0.026239949201592228,0.022464325872257995,0.018688702542923541,0.014913079213589198,0.011137455884254965,0.0073618325549208441],[0.10338607135467048,0.10040727319363119,0.097428475032591111,0.094449676871551813,0.091470878710511738,0.088492080549472441,0.085513282388432588,0.082534484227392846,0.079555686066353548,0.076576887905313473,0.073600953355880128,0.070700988408703314,0.068022864124253068,0.065715819909676121,0.063929095172121864,0.062764448214995383,0.062124286852310751,0.061858708968201559,0.061817812237037528,0.061854838889812247,0.061894967015260205,0.061935095140708385,0.061975223266156454,0.062015351391604523,0.062055479517052592,0.062095607642500661,0.06213573576794873,0.06217586389339691,0.062215992018844979,0.062256120144292937],[0.15408501631722804,0.14725217333424279,0.1404193303512572,0.13358648736827228,0.12675364438528647,0.11992080140230132,0.11308795841931596,0.10625511543633059,0.09942227245334545,0.092589429470360085,0.085767218505564458,0.079227067507061877,0.073510566600069716,0.069171811908551484,0.066764899556472357,0.066667637808930369,0.068517686363065544,0.071758488589747982,0.075833487081037432,0.080197799539942882,0.084573627147693453,0.088949454755444357,0.093325282363195039,0.097701109970945943,0.10207693757869651,0.10645276518644708,0.11082859279419832,0.115204420401949,0.11958024800969935,0.12395607561745048],[0.20534758331940661,0.19460157596775385,0.18385556861610108,0.17310956126444854,0.16236355391279533,0.15161754656114301,0.14087153920949003,0.13012553185783726,0.1193795245061845,0.10863351715453173,0.097906098316291268,0.087671819667688666,0.078877571547567582,0.072492109186389264,0.06948418781461474,0.070514349379927288,0.074949095381934172,0.081815368760322138,0.090140111093135245,0.098970676165456517,0.10782137377469792,0.1166720713839392,0.12552276899318093,0.13437346660242178,0.14322416421166428,0.15207486182090546,0.16092555943014664,0.16977625703938848,0.17862695464862932,0.18747765225787105],[0.25661621979242766,0.2419564114320546,0.22729660307168131,0.2126367947113087,0.19797698635093586,0.18331717799056324,0.16865736963018996,0.1539975612698169,0.13933775290944428,0.124677944549071,0.11004468322280048,0.096115696043761023,0.084243276740838535,0.075810945209884628,0.072202221346751805,0.074360452756621287,0.081380913713401393,0.091873940697957401,0.1044498682465379,0.11774818242269858,0.13107524871756449,0.14440231501243017,0.15772938130729708,0.17105644760216343,0.18438351389702923,0.19771058019189514,0.21103764648676127,0.22436471278162806,0.23769177907649286,0.25101884537135943],[0.30788485626544915,0.28931124689635579,0.27073763752726288,0.25216402815816952,0.23359041878907683,0.21501680941998302,0.19644320005088989,0.17786959068179697,0.15929598131270362,0.14072237194361048,0.12218326812931013,0.10455957241983371,0.089608981934109599,0.079129781233380991,0.074920254878889203,0.078206556133315619,0.087812732044868169,0.10193251263559322,0.11875962539994001,0.13652568867994042,0.15432912366043117,0.17213255864092225,0.18993599362141289,0.20773942860190375,0.22554286358239561,0.24334629856288614,0.26114973354337689,0.27895316852386787,0.29675660350435795,0.31456003848484915],[0.35915349273847041,0.33666608236065698,0.31417867198284311,0.29169126160503012,0.26920385122721713,0.24671644084940325,0.22422903047159026,0.20174162009377683,0.1792542097159634,0.15676679933815041,0.13432185303581956,0.1130034487959054,0.094974687127380553,0.082448617256876577,0.07763828841102649,0.082052659510009285,0.094244550376336167,0.11199108457322959,0.13306938255334244,0.15530319493718192,0.1775829986032974,0.199862802269413,0.22214260593552837,0.24442240960164441,0.26670221326775978,0.28898201693387582,0.31126182059999141,0.33354162426610678,0.35582142793222127,0.37810123159833753],[0.41042212921149124,0.38402091782495773,0.35761970643842422,0.33121849505189072,0.30481728366535721,0.2784160722788237,0.25201486089229064,0.22561364950575669,0.19921243811922318,0.17281122673268923,0.14646043794232888,0.12144732517197765,0.10034039232065151,0.08576745328037183,0.080356321943163556,0.085898762886703617,0.1006763687078035,0.1220496565108653,0.14737913970674577,0.17408070119442398,0.20083687354616397,0.22759304589790419,0.25434921824964507,0.28110539060138506,0.30786156295312594,0.33461773530486594,0.36137390765660637,0.38813008000834659,0.41488625236008569,0.44164242471182702],[0.46169076568451251,0.43137575328925848,0.40106074089400445,0.37074572849875176,0.34043071610349729,0.31011570370824371,0.27980069131299012,0.24948567891773654,0.21917066652248296,0.18885565412722849,0.15859902284883898,0.12989120154805001,0.10570609751392102,0.08908628930386786,0.083074355475300177,0.089744866263397838,0.10710818703926972,0.13210822844850079,0.16168889686014787,0.19285820745166526,0.22409074848902988,0.25532328952639538,0.28655583056376022,0.3177883716011255,0.34902091263849078,0.38025345367585561,0.41148599471322045,0.44271853575058573,0.47395107678794879,0.50518361782531451],[0.51295940215753377,0.47873058875355967,0.44450177534958557,0.41027296194561191,0.37604414854163826,0.34181533513766371,0.30758652173368961,0.2733577083297164,0.2391288949257423,0.20490008152176864,0.17073760775534819,0.1383350779241227,0.11107180270719286,0.092405125327363113,0.085792389007437353,0.093590969640092503,0.11354000537073661,0.14216680038613594,0.17599865401354975,0.21163571370890766,0.24734462343189667,0.28305353315488679,0.31876244287787669,0.35447135260086615,0.39018026232385672,0.42588917204684551,0.46159808176983563,0.49730699149282553,0.53301590121581321,0.56872481093880445],[0.56422803863055548,0.52608542421786131,0.48794280980516713,0.4498001953924734,0.41165758097977923,0.37351496656708505,0.33537235215439087,0.29722973774169537,0.25908712332900297,0.22094450891630746,0.18287619266185828,0.14677895430019461,0.11643750790046425,0.095723961350859255,0.088510422539575417,0.097437073016786946,0.11997182370220461,0.15222537232377242,0.19030841116695352,0.23041321996615005,0.27059849837476346,0.31078377678337932,0.35096905519199206,0.39115433360060836,0.43133961200922244,0.47152489041783718,0.51171016882645237,0.55189544723506578,0.59208072564367853,0.63226600405229438],[0.61549667510357764,0.57344025968216294,0.53138384426074825,0.48932742883933356,0.44727101341791975,0.4052145979965055,0.36315818257509169,0.32110176715367611,0.2790453517322623,0.23698893631084716,0.19501477756836649,0.15522283067626785,0.12180321309373598,0.099042797374354952,0.09122845607171215,0.10128317639348183,0.12640364203367205,0.16228394426140846,0.20461816832035573,0.24919072622339244,0.2938523733176307,0.33851402041187117,0.38317566750610921,0.42783731460034968,0.47249896169458949,0.51716060878882752,0.56182225588306822,0.60648390297730781,0.65114555007154384,0.69580719716578521],[0.66676531157659757,0.62079509514646236,0.57482487871632892,0.52885466228619504,0.48288444585606027,0.43691422942592462,0.39094401299579118,0.34497379656565641,0.29900358013552164,0.25303336370538732,0.20715336247487726,0.16366670705233965,0.12716891828700516,0.10236163339784965,0.093946489603848549,0.1051292797701745,0.13283546036513894,0.17234251619904339,0.21892792547375872,0.26796823248063351,0.3171062482604966,0.36624426404036148,0.41538227982022569,0.46452029560008945,0.51365831137995344,0.56279632715981653,0.61193434293968119,0.66107235871954628,0.71021037449940694,0.75934839027927248]],"type":"surface","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.20000000000000001,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
</div>
  </main><aside class="col-md-3"><nav id="toc" aria-label="Table of contents"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p>Developed by Yihao Li, Gissette Reyes-Soffer, Badri Vardarajan.</p>
</div>

<div class="pkgdown-footer-right">
  <p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.2.0.</p>
</div>

    </footer>
</div>





  </body>
</html>
